\begin{abstract}

Alzheimer's disease is a devastating neurodegenerative disorder characterised by progressive intracellular accumulation of hyperphosphorylated tau and extracellular deposition of amyloid-beta. It affects over 50 million people worldwide with numbers expecting to triple by 2050. Despite recent success in identifying candidate AD-risk genes, the mechanisms underpinning disease progression remain unknown. An increasing number of transcriptome profiling studies implicate altered transcriptional regulation and RNA splicing in the development of AD pathology. However, such studies are methodologically-flawed due to inherent limitations of standard short-read RNA-sequencing approaches, which fail to capture full-length transcripts critical for transcriptome assembly.  

The primary aim of this thesis was to utilise two long-read sequencing approaches, Pacific Biosciences isoform sequencing and Oxford Nanopore Technologies nanopore cDNA sequencing, to assess progressive transcriptional and splicing variation associated with AD in a mouse model of tau pathology, rTg4510. By generating long reads that span full-length transcripts, our studies revealed widespread RNA isoform diversity with unprecedented detection of novel transcripts not presents in genome annotations. By performing targeted sequencing of 20 AD-risk genes, we identified robust expression changes at the transcript level associated with tau accumulation. Our analyses further provide a systematic evaluation of transcript usage, even in the absence of gene expression alterations, and highlight the importance of alternative RNA splicing as a mechanism underpinning gene regulation in the development of tau pathology. 

In summary, this thesis presents the first comprehensive study leveraging the power of long-read sequencing to assess AD transcriptional and splicing variation in a tau mouse model.       

\end{abstract}