\chapter{Splicing signatures of progressive tau pathology in AD mouse model}\label{ch: transcriptional_global_differences}
%Classification of AS events, which most commonly observed/dominant? Isoforms derived from transcriptional regulation (alternative promoters) vs post-transcriptional regulation? 
%Impact of AS events on protein domains. Non-sense mediated decay? 
%%Furthermore changes in gene/transcript expression can be due to differences in cellular composition (i.e. neuronal loss/reactive gliosis) rather than indicative of disease-associated transcriptional regulation. 
%%Gene expression and mRNA isoforms vary widely across tissues (\cite{Wang2008}), thus sequencing the disease-relevant tissue (in this case entorhinal cortex) is important for understanding the pathology of AD. However, it is consequently important to note that other tissues may have to be considered to fully grasp the whole picture of AD development. 

\section{Introduction}
There is increasing evidence for the role of transcriptional dysregulation and aberrant splicing in AD development and pathogenesis. Recent studies have identified changes in splicing and transcript expression in human AD-postmortem brain tissues and AD transgenic mouse models. However to date, these studies rely on short-read RNA-Seq data, which is constrained for accurate detection and characterisation of RNA isoforms (as discussed in \cref{rnaseq_intro}). In contrast, we have illustrated the power of long-read sequencing to identify full-length transcripts and improve our annotations of alternatively spliced isoforms in the cortex of a mouse model of AD tauopathy, rTg4510 (\cref{ch: whole_transcriptome}). Following on from this work, this chapter aimed to exploit these long-read sequencing datasets generated from rTg4510 mice to identify transcriptional and splicing changes associated with progressive tau pathology.  

%% METYHYLATION %%


%Genome-wide RNAseq study of the molecular mechanisms underlying microglia activation in response to pathological tau perturbation in the rTg4510 tau transgenic animal model - PubMed (nih.gov)
%Integrating human brain proteomes with genome-wide association data implicates new proteins in Alzheimer's disease pathogenesis - PubMed (nih.gov)
%The role of microglia in processing and spreading of bioactive tau seeds in Alzheimer’s disease | Journal of Neuroinflammation | Full Text (biomedcentral.com) 
%%https://actaneurocomms.biomedcentral.com/articles/10.1186/s40478-018-0574-5 - rTg4510


\section{Methods}

\subsection{Datasets}
All analyses pertaining to this chapter follow on from \cref{ch: whole_transcriptome} using the same Iso-Seq datasets generated from 12 female mice (wild-type controls = 6, rTg4510 transgenic = 6, aged 2 and 8 months, \cref{tab:whole_phenotype}). Briefly, RNA was prepared for Iso-Seq library preparation and PacBio SMRT sequencing on the Sequel (\cref{ch4_methods: isoseq_library}), followed by QC and data processing (\cref{ch4_methods: isoseq_data}); reads from individual samples were processed separately with \textit{IsoSeq3} and merged for transcript collapse with \textit{Cupcake}, genome alignment with \textit{Minimap2} and re-annotation with \textit{SQANTI3}. 

\subsection{Quantification of human MAPT transgene expression} 
The presence of human- and mouse-specific \textit{Mapt/MAPT} sequences was determined in FL transcripts as QC check of sample identity. Species-specific \textit{Mapt/MAPT} sequence, located in a 2kb region present in the 3'UTR, was identified after using BLAT\cite{Kent2002} to compare human and mouse MAPT/Mapt sequence for divergent transcript sequences \cite{Castanho2020}).  

\subsection{Characterisation of Alternative Splicing Events} 
Alternative splicing events were examined using a range of packages and custom scripts, as described and implemented in \cref{sec:AS_methods}, to assess whether there was a change in splicing patterns associated with rTg4510 pathology and across age. 


\subsection{Gene and Isoform Quantification}
Gene and isoform expression were estimated using two approaches: i) alignment of short-read RNA-Seq reads to the Iso-Seq-defined transcriptome (hybrid approach) using \textit{Kallisto}, and ii) normalised Iso-Seq full-length (FL) read counts (\cref{sec: gene_isoform_quant_explained}). FL read counts for each sample were extracted from \textit{Cupcake read\_stat.txt} file, using the sequencing run ID as identifier. 

To ease downstream quantitative analyses, ISM and shorter FSM transcripts annotated to the same known isoform (exemplified in \cref{fig:redudant_sncatranscripts}) were assumed as partial products of the longest detected FSM transcript, and thus removed with their associated counts aggregated (as shown in \cref{fig:ism_collapse}). Such products are likely to be technical artefacts from varying degrees of 5'degradation and truncation, generating shorter FSM transcripts with slightly different 5' and 3' ends while preserving internal exonic structure (small degree) and shorter ISM transcripts with matching but smaller overlap in exonic structure (large degree). Notably, significantly shorter ISM transcripts could in theory be mapped to multiple isoforms.  

\begin{figure}[htp]
	\begin{center}
		\includegraphics[page=10,trim={0cm 0cm 0 0cm},clip, scale = 0.7]{Figures/ProjectDevelopment_Figures.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Redundant ISM and FSM associated with same known isoform]%
	{\textbf{Redundant ISM and FSM associated with known transcript of \textit{Snca}}: \textbf{A)} Visualising transcripts using \textit{Swan}, shown are \textbf{B)} transcripts detected after \textit{SQANTI} annotation of the mouse transcriptome using the targeted approach. An example of detecting multiple redundant transcripts annotated to the same isoform, all the transcripts depicted were annotated to ENMUST0000014268.4 which typically differed at the 5' and 3' end.}
	\label{fig:redudant_sncatranscripts}
\end{figure}

\begin{figure}[htp]
	\centering
	\includegraphics[page=9,trim={0cm 12cm 0cm 0cm},clip,scale = 0.8]{Figures/ProjectDevelopment_Figures.pdf}
	\captionsetup{width=0.95\textwidth,singlelinecheck=off}
	\caption[Merging of ISM and FSM]%
	{\textbf{Merging of ISM and FSM associated with the same known isoform}. All illustration of handling with multiple ISM and FSM associated with the same known transcript. In this example, 6 isoforms were detected for "Gene 1", however, only 4 known isoforms were identified (denoted as ENMUST1-4). For ease of annotation and quantification, ISM and shorter FSM were assumed partial products of the longer FSM. Consequently, in the case where: 
		\begin{itemize}
			\item both ISM (PB.1.2) and FSM (PB.1.1) are annotated to the same transcript (shaded green), the FSM takes precedence as the associated transcript and counts from respective ISM are aggregated 
			\item there is one ISM (PB.1.3) but no corresponding FSM for the associated transcript (shaded blue), the ISM is retained
			\item there are multiple ISM (PB.1.4, PB.1.5) annotated to the same transcript (shaded orange), the longest ISMs (PB.1.5) takes precedence and counts from respective ISM are similarly aggregated 
			\item there are multiple FSM (PB.1.6, PB.1.7) annotated to the same transcript (shaded purple), the longest FSM (PB.1.7) takes precedence and counts are similarly aggregated
			\\
		\end{itemize} 
		PB - PacBio. FSM - Full Splice Match, ISM - Incomplete Splice Match, FL - Full-length read counts.  
	}
	\label{fig:ism_collapse}
\end{figure}

\newpage
\subsection{Differential analysis}
Iso-Seq and aligned RNA-Seq read counts were analysed for differential expression using \textit{tappAS} (described in \cref{ch3_tappas_explained}). Briefly, \textit{tappAS} filters out lowly-expressed isoforms, normalises read counts using TMM approach and performs differential expression analysis using \textit{maSigPro}\cite{Conesa2006,Nueda2014,Conesa2017} to elucidate effects for both genotype and age with the following adapted model\cite{Conesa2006}: 

\vspace{1cm}
\begin{myequation}[h]
Let \textit{I} denote the genotype groups (wild-type - WT, transgenic - TG) and \textit{J} as the age (2 and 8 months) for each particular group, and assuming that gene or transcript expression in measured in replicated samples (\textit{R}).  
\begin{align}
	y_{ijr} =  \:&\beta_{0} + \beta_{1}D_{ijr} \nonumber
	\\ &+ \delta_{0}T_{ijr} + \delta_{1}T_{ijr}D_{ijr}   \nonumber
\end{align}
where
\begin{conditions*}
	y_{ijr} & normalised expression value for each gene or transcript in the situation \textit{ijr} (genotype group \textit{i} at age \textit{j} of replicate \textit{r}) \\
	D  &  dummy binary variable to distinguish between the genotype groups, whereby 0 refers to reference group (WT) and 1 refers to experimental group (TG) \\
	T  &  age at 2 8 months described using a polynomial model with a degree of 1 \\
	\beta_{0}, \delta_{0}, \gamma_{0}, \lambda _{0} & regression coefficients for reference group (WT) relating to the age \\ 
	\beta_{1}, \delta_{1}, \gamma_{1}, \lambda _{1} & regression coefficients for the difference between the experimental group (TG) and reference group (WT) at each age  
\end{conditions*}
therefore, if:
\begin{conditions*}
	FDR(\beta_{1}) < 0.05 & significant expression difference between WT and TG at 2 months \\ 
	FDR(\delta_{0}) < 0.05 & significant expression difference in WT across 2 and 8 months \\
\end{conditions*}
\label{eq:dea_lm_masigpro}
\captionsetup{width=1\textwidth}
\caption[Linear regression model to determine differential gene and transcript expression]%
{\textbf{Linear regression model to determine differential gene and transcript expression}. The model, adapted from \textit{MaSigPro} and implemented as part of \textit{tappAS}, describes gene or transcript expression between two groups (WT - wild-type, TG - transgenic) at four different time points (age in months). FDR - False discovery rate}    
\end{myequation}

\begin{figure}[!htp]
	\centering
	\includegraphics[page=2,trim={0 5cm 0 4cm},scale = 0.45]{Figures/Tg4510_diff_figures.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Different conditions modelled for rTg4510 genotype and age effects]%
	{\textbf{Different conditions modelled for rTg4510 genotype and age effects.} Statistical models generated to dissect genotype and age effects with \textit{maSigPro} using Equation \ref{eq:dea_lm_masigpro} for 2 experimental groups (WT - wild-type/Control, TG - Transgenic/Case) across two time points/age (T1, T2). 
	\\\\
	The regression coefficients from Equation \ref{eq:dea_lm_masigpro} - $\beta_{1}$, $\delta_{0}$, $\delta_{1}$ - refer to the different variables modelled, the significance of which can be used to infer whether there is a genotype, age or interaction effect. The significance is symbolised by the tick and cross, which refers to adjusted P-value (FDR) < 0.05 and > 0.05 respectively. A significance of  $\beta_{1}$ denotes to a statistically significant difference between WT and TG at T1 (Genotype effect),$\delta_{0}$ to a difference in WT over time (Age effect), and $\delta_{1}$ to a difference between WT and TG across age (Interaction effect).}   
	\label{fig:dea_model}
\end{figure}

Under this model, a differentially expressed gene or transcript between WT and TG mice was defined with a statistically significant coefficient (adjusted P-value < 0.05) and a regression model with R\textsuperscript{2} > 0.5  (\cref{fig:dea_model}) .


\clearpage 
\section{Results}
\subsection{PacBio Iso-Seq run performance and sequencing metrics}
No significant difference in sequencing run yield was identified between WT and TG (n = 12 animals, two-tailed unpaired t-test, t(10) = -0.636, P = 0.539, \cref{ig:rTg4510_sequencing_metrics}\textbf{A}), and no significant correlation was observed between run yield and RIN across samples (n = 12 animals, Pearson's correlation, t = -0.98, df = 10, P = 0.350, \cref{ig:rTg4510_sequencing_metrics}\textbf{B}). No difference was observed in the number of reads (\cref{fig:rTg4510_sequencing_metrics}\textbf{C}) and transcripts generated between WT and TG (n = 12, two-tailed unpaired t-test, t = -0.005, df = 10, P = 0.996, \cref{fig:rTg4510_sequencing_metrics}\textbf{E}) or by age (n = 12, t = -1.58, df = 10, P = 0.15). ERCC alignment and annotations similarly revealed no difference in number of ERCC detected between WT and TG (mean number of ERCC: WT = 32.4 (35\%), TG = 32.2 (35.22\%)). 

\begin{figure}[htp]
	\begin{center}
		\includegraphics[page=1,trim={0 0 0 0},clip,scale = 0.55]{Figures/rTg4510WholeTranscriptome.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Sequencing and data processing metrics of rTg4510 mice]%
	{\textbf{No significant difference in sequencing metrics, number of transcripts and read length observed between WT and rTg4501 TG mice}: \textbf{A)} No significant difference in sequencing reads observed between WT and TG mice, despite \textbf{B)} TG samples having distinctly lower RIN values. Of note, two samples with significantly lower throughput ($<$25Gb) were generated with a lower chemistry. \textbf{C)} Processing of Iso-Seq reads generated a similar number of reads across all samples throughout Iso-Seq3 bioinformatics pipeline, except the two lower-throughput samples. No significant difference in \textbf{D)} number of FL transcripts and \textbf{E)} length of CCS reads were observed between WT and TG}
	\label{fig:rTg4510_sequencing_metrics}
\end{figure}


\subsection{Change in endogenous expression}
%Check whether overexpression of human MAPT result in any unwanted, compensatory effects on equivalent mouse genes,as expression levels of mouse APP and MAPT should be slightly reduced, thereby suggesting no evidence that human transgene expression increase expression of directly-related mouse genes.
As expected, human-specific \textit{MAPT} sequences were only detected in reads from TG mice, confirming stable activation of human \textit{MAPT} transgene (\cref{fig:isoseq_humanmapt}\textbf{A}) and supporting findings from Castanho et al.\cite{Castanho2020}. Alignment of these human-specific transcripts to the mouse genome were mapped either to the mouse prion protein gene (\textit{Prnp}) with high identity but short overlap (\cref{fig:isoseq_humanmapt}\textbf{b,c}), given that the transgene contains exons 2-3 of mouse \textit{Prnp}\cite{Ramsden2005}, and to the mouse \textit{Mapt} gene with low identity but long overlap (\cref{fig:isoseq_humanmapt}\textbf{b,d}). Applying filter thresholds for downstream analysis removed these human-specific \textit{MAPT} transcripts (\cref{fig:isoseq_humanmapt}\textbf{B}). 

\begin{figure}[htp]
	\begin{center}
		\includegraphics[page=1,trim={1cm 20cm 0cm 0cm},clip,scale = 0.55]{Figures/AltFigures_Diff.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Quantifying human-specific and mouse-specific \textit{MAPT}/\textit{Mapt} sequences in Iso-Seq Whole Transcriptome]%
	{\textbf{Human-specific \textit{MAPT} sequences only present in transgenic mice with poor alignment to mouse \textit{Prnp} and \textit{Mapt} gene}: Presence of human- and mouse-specific \textit{MAPT}/\textit{Mapt} sequences was determined in full-length transcripts generated from Iso-Seq merged dataset. \textbf{A)} Ratio of full-length transcripts that were mapped to human-specific \textit{MAPT} and mouse-specific \textit{Mapt} sequences. Dotted lines represent the mean paths across ages. \textbf{B)} As expected, human-specific \textit{MAPT} transcripts were poorly aligned to mouse genome. Transcripts were either aligned to mouse \textit{Prnp} gene (boxed yellow) or  mouse \textit{Mapt} gene (boxed blue). Alignment to mouse \textit{Prnp} gene was near 100\% within a short region, given that the transgene contains exon 2 and 3 of mouse \textit{Prnp} gene\cite{Ramsden2005}. Conversely, while human-specific \textit{MAPT} gene was sufficiently divergent from mouse \textit{Mapt} gene for transgene quantification, it still mapped to the mouse \textit{Mapt} gene 3'UTR albeit poorly. \textbf{C)} USCS genome browser tracks of human-specific (black) \textit{MAPT} transcripts (transgene) and mouse \textit{Prnp} gene and \textbf{D)} mouse \textit{Mapt} gene. Blue tracks represent known transcripts from reference mouse genome (mm10). Tracks were cropped and modified to remove irrelevant genes within the same locus.  UTR - Untranslated region}
	\label{fig:isoseq_humanmapt}
\end{figure}


\subsection{Transcriptome Annotation of rTg4510 mice}
While identifying widespread RNA isoform diversity amongst genes expressed in the mouse entorhinal cortex (described in \cref{ch: whole_transcriptome}), no difference was observed in the number of genes (mean number = 13,302 genes) or isoforms (mean number = 35,157 isoforms) detected between wild-type and transgenic mouse aged 2 and 8 months. Further characterisation of the transcriptome revealed similar profile of isoform diversity between the two genotypes and ages (\cref{tab:isoseq_whole_subsqantioutput}), with half of the isoforms annotated as known (mean number = 18,567 isoforms (52.8\%), as also shown in \cref{sec:whole_novelIso}) and with a similar distribution in isoform length and number of exons (median: 9, range = 1-89). 

\begin{landscape}
	\begin{table}[]
		\centering
		\captionsetup{width=1\linewidth}
		\caption[Overview of the whole transcriptome Iso-Seq datasets generated from mouse rTg4510, subsected by phenotype and age]%
		{\textbf{Overview of the whole transcriptome Iso-Seq datasets generated from mouse rTg4510, subsected by phenotype and age}. Annotations from wild-type (n = 6) and transgenic mouse (n = 6) were generated from merging Iso-Seq datasets from mouse aged 2 and 8 months of the respective phenotype. Novel genes refer to genes that were not currently present in existing genome annotations (mm10). Isoform can be further classified as known (FSM, ISM) or novel (ISM, NIC, NNC, Genic Genomic, Antisense, Fusion, Intergenic, Genic Intron), as described in \cref{sec:sq_exp}. FSM – Full Splice Match, ISM – Incomplete Splice Match, NIC – Novel In Catalogue, NNC – Novel Not in Catalogue.}
		\label{tab:isoseq_whole_subsqantioutput}
		\resizebox{1.5\textwidth}{!}{%
	\begin{tabular}{@{}ccccccc@{}}
		\toprule
		& \begin{tabular}[c]{@{}c@{}}wild-type \\ (n = 6)\end{tabular}              & \begin{tabular}[c]{@{}c@{}}Transgenic\\  (n = 6)\end{tabular}            & \begin{tabular}[c]{@{}c@{}}wild-type, 2 months \\ (n = 3)\end{tabular}    & \begin{tabular}[c]{@{}c@{}}wild-type, 8 months\\  (n = 3)\end{tabular}   & \begin{tabular}[c]{@{}c@{}}Transgenic, 2 months \\ (n = 3)\end{tabular}  & \begin{tabular}[c]{@{}c@{}}Transgenic, 8 months \\ (n = 3)\end{tabular}  \\ \midrule
		Total Number of Genes               & 14083                                                                    & 14183                                                                    & 12798                                                                    & 12947                                                                   & 12384                                                                    & 13418                                                                    \\
		Annotated Genes                     & 13911 (98.78\%)                                                          & 14018 (98.84\%)                                                          & 12696 (99.2\%)                                                           & 12816 (98.99\%)                                                         & 12286 (99.21\%)                                                          & 13289 (99.04\%)                                                          \\
		Novel Genes                         & 172 (1.22\%)                                                             & 165 (1.16\%)                                                             & 102 (0.8\%)                                                              & 131 (1.01\%)                                                            & 98 (0.79\%)                                                              & 129 (0.96\%)                                                             \\
		Total Number of Isoforms            & 41081                                                                    & 41671                                                                    & 31407                                                                    & 32630                                                                   & 28982                                                                    & 35169                                                                    \\
		FSM                                 & 18287 (44.51\%)                                                          & 18574 (44.57\%)                                                          & 15503 (49.36\%)                                                          & 15870 (48.64\%)                                                         & 14675 (50.63\%)                                                          & 16892 (48.03\%)                                                          \\
		ISM                                 & 3164 (7.7\%)                                                             & 3242 (7.78\%)                                                            & 2243 (7.14\%)                                                            & 2368 (7.26\%)                                                           & 2066 (7.13\%)                                                            & 2590 (7.36\%)                                                            \\
		NIC                                 & 11781 (28.68\%)                                                          & 12033 (28.88\%)                                                          & 8356 (26.61\%)                                                           & 8856 (27.14\%)                                                          & 7518 (25.94\%)                                                           & 9805 (27.88\%)                                                           \\
		NNC                                 & 7354 (17.9\%)                                                            & 7343 (17.62\%)                                                           & 4980 (15.86\%)                                                           & 5175 (15.86\%)                                                          & 4427 (15.27\%)                                                           & 5520 (15.7\%)                                                            \\
		Genic Genomic                       & 55 (0.13\%)                                                              & 54 (0.13\%)                                                              & 37 (0.12\%)                                                              & 41 (0.13\%)                                                             & 28 (0.1\%)                                                               & 43 (0.12\%)                                                              \\
		Antisense                           & 93 (0.23\%)                                                              & 100 (0.24\%)                                                             & 49 (0.16\%)                                                              & 74 (0.23\%)                                                             & 65 (0.22\%)                                                              & 74 (0.21\%)                                                              \\
		Fusion                              & 253 (0.62\%)                                                             & 242 (0.58\%)                                                             & 180 (0.57\%)                                                             & 176 (0.54\%)                                                            & 155 (0.53\%)                                                             & 177 (0.5\%)                                                              \\
		Intergenic                          & 94 (0.23\%)                                                              & 83 (0.2\%)                                                               & 59 (0.19\%)                                                              & 70 (0.21\%)                                                             & 48 (0.17\%)                                                              & 68 (0.19\%)                                                              \\
		Genic Intron                        & 0 (0\%)                                                                  & 0 (0\%)                                                                  & 0 (0\%)                                                                  & 0 (0\%)                                                                 & 0 (0\%)                                                                  & 0 (0\%)                                                                  \\
		Isoform Length (bp)                 & \begin{tabular}[c]{@{}c@{}}Median: 2946, \\ Range: 83-15016\end{tabular} & \begin{tabular}[c]{@{}c@{}}Median: 2955, \\ Range: 83-15913\end{tabular} & \begin{tabular}[c]{@{}c@{}}Median: 2987, \\ Range: 88-15016\end{tabular} & \begin{tabular}[c]{@{}c@{}}Median: 2890,\\ Range: 83-14850\end{tabular} & \begin{tabular}[c]{@{}c@{}}Median: 2798, \\ Range: 88-14302\end{tabular} & \begin{tabular}[c]{@{}c@{}}Median: 3013, \\ Range: 83-15913\end{tabular} \\
		Number of Exons                     & \begin{tabular}[c]{@{}c@{}}Median: 9, \\ Range: 1-89\end{tabular}        & \begin{tabular}[c]{@{}c@{}}Median: 9, \\ Range: 1-89\end{tabular}        & \begin{tabular}[c]{@{}c@{}}Median: 9, \\ Range: 1-89\end{tabular}        & \begin{tabular}[c]{@{}c@{}}Median: 9, \\ Range: 1-89\end{tabular}       & \begin{tabular}[c]{@{}c@{}}Median: 9, \\ Range: 1-77\end{tabular}        & \begin{tabular}[c]{@{}c@{}}Median: 9, \\ Range: 1-89\end{tabular}        \\
		Number of Isoforms within 50bp CAGE & 34574 (84.16\%)                                                          & 35097 (84.22\%)                                                          & 26539 (84.5\%)                                                           & 27689 (84.86\%)                                                         & 24398 (84.18\%)                                                          & 29911 (85.05\%)                                                          \\ \bottomrule
			\end{tabular}%
		}
		\end{table}
\end{landscape}

\subsection{rTg4510 mice display similar global AS patterns}
Transcriptome profiling of the mouse cortex previously identified widespread isoform diversity, driven predominantly by the usage of alternative first exons (AF) and exon skipping (ES) (described in \cref{sec:whole_novelIso}). Upon further examination, there was no significant difference in splicing patterns associated with rTg510 pathology or across age with AF (mean n = 20,400, 40.2\%) as the most prevalent event across all datasets \cref{AS_WholeTranscriptome_diff}. 
%stats?
%XX of known transcripts were identified to have intron retention; XX of known transcripts were identified to be fusion genes. XX of know transcripts identified to have non-sense-mediated decay. 

\vspace{0.5cm}
\begin{table}[!htp]
	\centering
	\captionsetup{width=1\textwidth}
	\caption[Alternative Splicing Events associated with tau pathology and age]%
	{\textbf{Alternative Splicing Events associated with tau pathology and age}. Tabulated is the number of splicing events detected for wild-type and transgenic Tg4510 mice aged 2 and 8 months (n = 12, 3 biological replicates per group)}
	\begin{tabular}{@{}ccccc@{}}
		\toprule
		\multirow{2}{*}{Splicing Events} & \multicolumn{2}{c}{wild-type} & \multicolumn{2}{c}{Transgenic} \\ \cmidrule(l){2-5} 
		& 2 months        & 8 months        & 2 months        & 8 months        \\ \midrule
		A3 & 3394 (6.8\%)    & 3526 (6.82\%)   & 2969 (6.41\%)   & 3825 (6.93\%)   \\
		A5 & 1388 (2.78\%)   & 1456 (2.82\%)   & 1231 (2.66\%)   & 1568 (2.84\%)   \\
		AF & 20185 (40.44\%) & 20597 (39.87\%) & 19182 (41.41\%) & 21781 (39.44\%) \\
		AL & 15620 (31.3\%)  & 16079 (31.12\%) & 14796 (31.94\%) & 16932 (30.66\%) \\
		IR & 3669 (7.35\%)   & 4169 (8.07\%)   & 3017 (6.51\%)   & 4794 (8.68\%)   \\
		MX & 324 (0.65\%)    & 315 (0.61\%)    & 285 (0.62\%)    & 378 (0.68\%)    \\
		SE & 5329 (10.68\%)  & 5524 (10.69\%)  & 4847 (10.46\%)  & 5953 (10.78\%)  \\ \bottomrule
	\end{tabular}
	\label{AS_WholeTranscriptome_diff}
\end{table}

\subsection{rTg4510-associated isoforms enriched for 3'UTR motif}
We next investigated whether the inclusion of the specific functional elements differed between wild-type and transgenic mice, suggesting a disease-specific regulation. Features characterised with DFI were observed across all functional categories (\cref{fig:DFI}\textbf{A}), with significant enrichment for 3'UTR motif (Fisher's Exact Test (FET) P = 1.51 x 10\textsuperscript{-4}), post-transcriptional,  modifications (PTM, FET P = 7.36 x 10\textsuperscript{-152}) and protein-binding sites (FET P = 6.13 x 10\textsuperscript{-17}. For transcript features, we observed that feature gain (i.e. feature is upregulated or favoured) was more apparent in in Wild-type whereas the the protein features, with the exception of transmembranes, were more frequent in the transgenic (\cref{fig:DFI}\textbf{B}). 


\begin{figure}[!htp]
	\centering
	\includegraphics[page=3,trim={0cm 20cm 0cm 0cm},scale = 0.6]{Figures/WholeFDA.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[DFI]%
	{\textbf{DFI} \textbf{A)} Enrichment DFI analysis. The Lines represent the proportion of total features per category across all annotated features, whereas the bars represent the proportion of DFI features observed per category across all features that were included in the DFI analysis. A functional category is considered significantly differentially included if the proportion of DFI feature is greater than proportion of annotated feature (i.e. the bar is higher than the line)}   
	\label{fig:DFI}
\end{figure}

\clearpage 
   
\subsection{Iso-Seq confirms widespread gene expression differences associated with tau pathology in rTg4510 mice}
%\boldheader{Usage of Iso-Seq reads alone detected robust changes in gene expression}
While Iso-Seq reads are commonly used for accurately characterising RNA diversity, we found that Iso-Seq reads were also able to accurately quantify the abundance of highly-expressed transcripts - as illustrated by the strong correlation of gene-level expression quantified using Iso-Seq and RNA-Seq reads, and near-perfect correlation Iso-Seq reads and actual amount of ERCC controls (described in XX). Strikingly, despite the lower long-read sequencing coverage and smaller sample size (n = 6 TG, n = 6 WT) sequenced on the long-read platform, a significant number of genes (n = 446 genes) were identified as differentially expressed when we used the Iso-Seq long reads for both annotation and expression. Using \textit{EnrichR}, differentially expressed genes (n = 466) were found to be highly enriched in lysosome (KEGG 2021 Human: odds ratio = 5.99, adjusted P = 4.41 x 10\textsuperscript{-6}) and in particularly the TGF-\textbeta pathway (WikiPathway 2021 Human: odds ratio = 58.98, adjusted P = 2.69 x 10\textsuperscript{-3} ). Classification of the differentially expressed genes by effects (n = 448 genes, depicted in \cref{fig:dea_model}\textbf{A} and showcased in \cref{fig:dea_model_genexp}), further identified the majority (n = 372 genes, \cref{fig:dea_model_num}\textbf{A}) to be associated with tau pathology (n = 55, \cref{fig:dea_model_genexp}\textbf{a, b}) and progressive with age (n = 317, \cref{fig:dea_model_genexp}\textbf{d,e,f,g}). Further examination of these genes revealed enrichment in the "turquoise" co-expression module (n = 134 genes, 36\%) identified from previous WGCNA analyses, which was previously found to become increasing upregulated with development of tau pathology and was highly enriched in pathways involved in immune system activation\cite{Castanho2020} (\cref{fig:dea_model_num}\textbf{B}).  
 
\begin{figure}[h]
	\centering
	\includegraphics[page=5,trim={0 20cm 0 0},clip,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differentially expressed genes classified by conditions]%
	{\textbf{Differentially expressed genes were identified across all the different conditions, with most genes exhibiting an interaction effect of rTg4510 genotype and age.} \textbf{A)} A bar plot of the number of differentially expressed genes (n = 446) classified by the different conditions, as modelled in \cref{fig:dea_model}. The differentially expressed genes were identified from the whole transcriptome datasets (WT = 6, TG = 6, across age 2 and 8 months) using Iso-Seq read counts as abundance. WT - wild-type, TG - Transgenic. \textbf{B)} A pie chart of the number and proportion of these differentially expressed genes associated with progressive tau pathology co-identified by the "Red","Turquoise", and "Yellow" significant co-expression module identified from WGCNA. All three modules were significantly associated with progressive tau pathology: "Red" module was downregulated in TG and is enriched in synaptic transmission, "Turquoise" module was upregulated in TG and is enriched for immune system activation, and "Yellow" module was downregulated in TG and is enriched in mitochondria and synpatic processes.}    
	\label{fig:dea_model_num}
\end{figure}

\begin{figure}[h]
	\centering
	\includegraphics[page=6,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Examples of gene expression differing across conditions]%
	{\textbf{Differential expressed genes exhibiting genotype, age and interaction effects} Shown are examples of differentially expressed genes classified under the different models, using the whole transcriptome dataset (WT = 6, TG = 6, across age 2 and 8 months) using Iso-Seq read counts as abundance: \textbf{A} \textit{Herc2} with a genotype effect, \textbf{B} \textit{Kcng1} with a genotype and age effect, \textbf{C} \textit{Clnc3} with an age effect, and \textbf{E} \textit{Cd34}, \textbf{E} \textit{Unc93b1}, \textbf{f} \textit{Csf1r} and \textbf{g} \textit{Gjb2} with an interaction effect. Dotted lines represent the mean paths across ages.}   
	\label{fig:dea_model_genexp}
\end{figure}

Usage of Iso-Seq reads alone identified differentially-expressed genes previously reported to be highly associated with AD. The top ranked tau-associated differentially-expressed genes that was found to be progressive with age in our previous analyse\cite{Castanho2020} was also recapitulated (\cref{fig:whole_dea}\textbf{a,b}): \textit{Gfap}, which encodes for glial fibrillary acidic protein (GFAP\nomenclature{GFAP}{Glial Fibrillary Acidic Protein}), a cytoskeletal protein that acts as a marker for astrocyte activation and proliferation - its increased expression was reported to correlate with increased neurofibrillary tangles density in Alzheimer's disease entorhinal cortex tissues \cite{Muramori1998}. Higher GFAP levels have also be documented in cerebrospinal fluid (CSF) in patients with AD compared to healthy control \cite{Ishiki2016}, and more recently, in plasma of cognitively intact older adults at risk of AD \cite{Chatterjee2021}. 


\begin{figure}[h]
	\begin{center}
		\includegraphics[page=2,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Comparison of Known and Novel Isoforms from Iso-Seq Whole Transcriptome runs]%
	{\textbf{Novel isoforms were less expressed, longer and had more exons than known isoforms}: Shown is the \textbf{A)} Iso-Seq transcript expression, the \textbf{C)} transcript length, and the \textbf{E)} the number of exons of novel and known isoforms. The known and novel isoforms can be further subdivided and classified, with the \textbf{B)} Iso-Seq expression \textbf{D)} transcript length and \textbf{f)} number of exons for each category. According to SQANTI, known isoforms are subdivided into FSM and ISM, and novel isoforms are subdivided into NIC, NNC, and fusion. FSM – Full Splice Match, ISM – Incomplete Splice Match, NIC – Novel In Catalogue, NNC – Novel Not in Catalogue.}   
	\label{fig:whole_dea}
\end{figure}


Other top-ranked differentially-expressed genes have been reported to be involved in AD development and pathology, notably \textit{C4b} - a member of the complement immune system (\cref{fig:whole_dea}\textbf{c,d}), \textit{Slc14a1} encoding the urea transporter 1, \textit{Tgfbr1} encoding the TGF-\textbeta receptor protein (\cref{fig:whole_dea}\textbf{e,f}) and \textit{Unc93b1}, a transmembrane protein required for toll (\cref{tab:dea_wholemouse}). Up-regulated expression of these genes have been observed previously in AD patients and other AD mouse models \cite{Zorzetto2016,Castillo2017,Wirz2013}. 

%log2fc calculated by mean expression at 8 case/mean expression at 8 control
\begin{table}[!htp]
	\centering
	\caption[Top-ranked differentially expressed genes associated with rTg4510]%
	{Tabulated are the top-ranked genes identified as differentially expressed in rTg4510 using \textit{maSigPro} with Iso-Seq defined transcriptome for annotation and Iso-Seq FL read count as expression. Gene expression is determined from the sum of normalised expression of associated transcripts. FDR - False Discovery Rate. }
	\begin{tabularx}{0.96\textwidth}{cccccccc}
		\toprule
		\multirow{3}{*}{Gene} & \multirow{3}{*}{FDR} & \multirow{3}{*}{R\textsuperscript{2}} & \multirow{3}{*}{\begin{tabular}[c]{@{}c@{}}log2-fold \\ change \\ (8 months)\end{tabular}} & \multicolumn{4}{c}{Mean Gene Expression}                      \\ \cmidrule(l){5-8} 
		&                      &                    &                                                                                            & \multicolumn{2}{c}{wild-type} & \multicolumn{2}{c}{Transgenic} \\ \cmidrule(l){5-8} 
		&                      &                    &                                                                                            & 2 months      & 8 months     & 2 months       & 8 months      \\ \midrule
		\textit{C4b}          & 3.9E-39              & 0.941              & 4.44                                                                                       & 3.57          & 1.79         & 4.03           & 87.4          \\
		\textit{Gfap}         & 8.88E-36             & 0.935              & 3.09                                                                                       & 75.6          & 62.3         & 106            & 900           \\
		\textit{Tgfbr1}       & 1.06E-20             & 0.880              & 3.48                                                                                       & 0.673         & 3.39         & 1.28           & 14.4          \\
		\textit{Slc14a1}      & 1.94E-16             & 0.872              & 2.92                                                                                       & 8.56          & 12.3         & 5.2            & 39.4          \\
		\textit{Unc93b1}      & 1.69E-15             & 0.853              & 1.5                                                                                        & 3.68          & 5.04         & 6.68           & 18.9          \\ \bottomrule
	\end{tabularx}
	\label{tab:dea_wholemouse}
\end{table}

\subsection{rTg4510 mice characterised by expression differences in novel, antisense genes}
Our group previously characterised widespread transcriptional differences in rTg4510 entorhinal cortex, identifying extensive differences in gene expression associated with development of tau pathology in TG mice (n = 154 differentially expressed genes)\cite{Castanho2020}. Mapping the same RNA-Seq reads (n = 29 TG, n = 30 WT) to Iso-Seq reads (n = 6 TG, n = 6 WT) derived from the same animals rather to the reference genome annotation, we similarly identify extensive differences in gene expression (n = XX differentially expressed genes) with a large overlap. 

The hybrid approach also enabled identification of tau-pathology associated changes in genes not previously known in reference. Using the Iso-Seq reads as annotation and RNA-Seq reads for expression, we identified robust expression changes in a few genes (n = 3) that we would have otherwise not detected if we used the reference genome for annotation. These "novel" genes, not present in existing genome annotations (mm10), have been previously characterised as more lowly-expressed and often were antisense to known genes, with a large proportion sharing exonic regions either at the 5'UTR, 3'UTR or within the gene body (\cref{sec:whole_novelgenes}). The most significant differentially-expressed novel gene was identified in Chromosome 10 (\cref{fig:whole_novelgene_difftracks}\textbf{A}), with progressive down-regulation in TG over time (\cref{fig:whole_novelgene_diffexp}\textbf{A}). The other differentially-expressed novel genes were found antisense to \textit{Fgfr1op} (\cref{fig:whole_novelgene_difftracks}\textbf{B}), within the gene-body, and to \textit{Htra1}  (\cref{fig:whole_novelgene_difftracks}\textbf{C}), at the 5'UTR. Both genes were associated with increased expression in TG compared to the WT over time (\cref{fig:whole_novelgene_diffexp}\textbf{b,d}). Notably, while \textit{Fgfr1op} was not identified as differentially-expressed (\cref{fig:whole_novelgene_diffexp}\textbf{C}), \textit{Htra1} was also found to have higher expression in TG compared to WT (\cref{fig:whole_novelgene_diffexp}\textbf{E}).     
%Htra1-AS shared exonic regions with Htra1 so misalignment of RNA-Seq reads

\begin{landscape}
	\begin{figure}[!htp]
		\centering
		\includegraphics[page=1,trim={0 4cm 0 0}, scale = 0.45]{Figures/TracksFigures_Diff.pdf}
		\captionsetup{width=1.4\textwidth}
		\caption[Tracks of novel genes that were differentially expressed in rTg4510 mice]%
		{\textbf{Tracks of novel genes that were differentially expressed in rTg4510 mice}: Shown are UCSC genome browser Iso-Seq tracks of three novel genes - \textbf{A)}novel gene in Chromosome 10, \textbf{B)} novel gene antisense to \textit{Fgfr1op}, and \textbf{C)} novel gene antisense to \textit{Htra1} - that were identified as differentially expressed with Iso-Seq reads (n = 12 samples) as annotation and RNA-Seq reads (n = 59 samples) as expression. The isoforms were coloured based on \textit{SQANTI} classification, with the novel gene coloured grey for genic/intergenic in panel a) and pink for antisense in panels b) and c). Shown are also the reference genome annotations (mm10) and RNA-Seq reads.}   
		\label{fig:whole_novelgene_difftracks}
	\end{figure}
\end{landscape}

\begin{figure}[!htp]
	\begin{center}
		\includegraphics[page=11,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Comparison of Known and Novel Isoforms from Iso-Seq Whole Transcriptome runs]%
	{\textbf{Novel isoforms were less expressed, longer and had more exons than known isoforms}: Shown is the \textbf{A)}}   
	\label{fig:whole_novelgene_diffexp}
\end{figure}



\clearpage
\subsection{rTg4510 characterised by widespread expression changes in transcripts associated to genes implicated in AD}
One of the added advantages of long reads over short-reads is the power to accurately identify isoforms and to explore isoform expression changes across conditions and over time. Using \textit{MaSigPro} with Iso-Seq reads as both reference annotation and expression, we identified hundreds of differentially expressed isoforms (n = 582 isoforms), of which 36 isoforms (6.19\%) were associated with rTg4510 genotype, and 378 isoforms (64.9\%) associated with progressive tau pathology. 

Other isoforms that were significantly unregulated with progressive tau pathology were annotated to genes that have been previously strongly implicated in AD pathology and development (\cref{fig:DEI_ADgenes_isoseq}). This included ENSMUST00000151120.8 associated with \textit{Ctsd}, which encodes for Cathepsin D, a lysosomal protease that is involved in degradation of A$\beta$ \cite{JR1996}, tau \cite{A1997}, and has recently been identified as a key regulator of A$\beta$42/40 ratio \cite{Suire2020}. Other isoforms include ENSMUST00000172785.7 associated with \textit{H2-D1} - that encodes for major histocompatibility complex (MHC) class 1, an immune-related gene that has been found to be upregulated in microglial cells of a different mouse model of neurodegeneration with AD-like phenotypes \cite{Mathys2017} - ENSMUST00000028624.8 associated with \textit{Gatm}, a mitochondrial protein that has been recently revealed as a key protein signature of AD from a large proteomic analysis of human cortex and CSF \cite{Wang2020}, - and ENSMUST00000030765.6 associated with \textit{Padi2}/\textit{Pad2}, an enzyme that has been found abnormally activated in astrocytes of patients with AD \cite{A2005}.

%\boldheader{Iso-Seq reads can accurately quantify highly-expressed isoforms and identify significant expression changes in isoforms annotated to highly-expressed genes}
Observations of differential isoform expression using Iso-Seq reads as expression were recapitulated with RNA-Seq reads, highlighting the power to accurately quantify isoforms at high expression (\cref{fig:DEI_ADgenes_rnaseq}).
However, on closer examination, the majority of isoforms identified as differentially expressed with progressive tau pathology (n = 321, 84.9\%) were not similarly identified as differentially expressed when RNA-Seq reads were used as expression. Given that the wide majority of Iso-Seq-identified-differentially-expressed isoforms were very-lowly expressed (295 isoforms,91.9\%, with mean full-length counts < 24 FL, n = 12 samples) with low read count, a seemingly significant expression change may result in calling the isoform differentially expressed (\cref{fig:dei_lowisoexp}). However, even for more highly-expressed isoforms, while a change in mean expression was observed, the variance was substantial due to a small sample size (n = 3 replicates). Conversely, with a greater sample size (n = 6 replicates) and higher sequencing coverage of RNA-Seq reads, the usage of RNA-Seq reads as expression reduced the probability of calling an isoform as differentially expressed due to chance (\cref{fig:dei_highisoexp}).      

\subsection{Gene expression differences in rTg4510 are primarily driven by differential expression of dominant isoform}
%\boldheader{Robust changes in isoform expression associated with progressive tau pathology were identified with Iso-Seq annotation and expression}
Strikingly, the two most significant progressive-tau-associated differentially expressed isoforms were annotated to \textit{Gfap} (\cref{fig:DEI_gfap}) and \textit{C4b} (\cref{fig:DEI_c4b}), the top two most differentially-expressed genes (\cref{tab:dea_wholemouse}). Both genes were characterised by a dominant known isoform in rTg4510 mice, which was significantly up-regulated with progressive tau pathology (\cref{fig:DEI_gfap}\textbf{A}, \cref{fig:DEI_c4b}\textbf{A}), indicating that increased \textit{Gfap} (\cref{fig:whole_dea}\textbf{A}) and \textit{C4b} (\cref{fig:whole_dea}\textbf{C}) gene expression in rTg4510 mice were primarily driven by one associated isoform. Notably, expression of other minor novel isoforms was also higher in rTg4510 mice aged 8 months (\cref{fig:DEI_gfap}\textbf{B}, \cref{fig:DEI_c4b}\textbf{B}). A similar finding was also observed when using RNA-Seq reads as abundance, though tau-pathology associated expression changes in minor isoforms were significantly more pronounced - a reflection of the comparably higher sequencing coverage of RNA-Seq reads to Iso-Seq reads. 


\begin{figure}[!htp]
	\centering
	\includegraphics[page=4,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential Isoform Expression: Changes in transcript expression of isoforms associated with \textit{Gfap}]%
	{\textbf{Significant upregulation of known isoform of \textit{Gfap} with progressive tau pathology}. Shown are \textbf{A)} normalised counts of \textit{Gfap} with associated isoforms in WT and TG across 2 time points (aged 2 and 8 months) for emphasis of dominant isoform (PB.2972.16, ENSMUST00000067444.9), and \textbf{B)} alternative plot of \textit{Gfap} with y-axis log-transformed for better visualisation of minor isoform expression. Novel minor isoforms, PB.2972.13 and PB.2972.14, were also identified as differentially expressed. Normalised counts of isoforms are derived directly from Iso-Seq full-length reads. FSM - Full Splice Match, ISM - Incomplete Splice Match, NIC - Novel In Catalogue, NNC - Novel Not in Catalogue. WT - Wild-type, TG - Transgenic. Dotted lines represent the mean paths across ages.} 
	\label{fig:DEI_gfap}
\end{figure}

\begin{figure}[!htp]
	\centering
	\includegraphics[page=12,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential Isoform Expression: Changes in transcript expression of isoforms associated with \textit{C4b}]%
	{\textbf{Significant upregulation of known isoform of \textit{C4b} with progressive tau pathology}. Shown are \textbf{A)} normalised counts of \textit{C4b} with associated isoforms in WT and TG across 2 time points (aged 2 and 8 months) for emphasis of dominant isoform (PB.7004.8, ENSMUST00000069507.8), and \textbf{B)} alternative plot of \textit{C4b} with y-axis log-transformed for better visualisation of minor isoform expression. Normalised counts of isoforms are derived directly from Iso-Seq full-length reads. FSM - Full Splice Match, ISM - Incomplete Splice Match, NIC - Novel In Catalogue, NNC - Novel Not in Catalogue. WT - Wild-type, TG - Transgenic. Dotted lines represent the mean paths across ages.}   
	\label{fig:DEI_c4b}
\end{figure}

\begin{figure}[!htp]
	\centering
	\includegraphics[page=18,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Robust changes in transcript expression of isoforms annotated to genes that are strongly implicated in AD]%
	{\textbf{Robust changes in transcript expression of isoforms annotated to genes that are strongly implicated in AD}. Using Iso-Seq reads (n = 6 WT, n = 6 TG, across 2 and 8 months) as annotation and expression, differential isoform expression was observed for \textbf{A)} ENSMUST00000151120.8 (PB.15108.6) annotated to \textit{Ctsd}, \textbf{B)} ENSMUST00000172785.7 (PB.7039.1) annotated to \textit{H2-D1}, \textbf{C)} ENSMUST00000028624.8 (PB.9298.1) annotated to \textit{Gatm}, and \textbf{D)} ENSMUST00000030765.6 (PB.11607.2) associated with \textit{Padi2}/\textit{Pad2}.  WT - wild-type, TG - Transgenic. Dotted lines represent the mean paths across ages.}    
	\label{fig:DEI_ADgenes_isoseq}
\end{figure}

\begin{figure}[!htp]
	\centering
	\includegraphics[page=19,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Changes in transcript expression of genes strongly implicated in AD were similarly detected using RNA-Seq reads]%
	{\textbf{Changes in transcript expression of genes strongly implicated in AD were similarly detected using RNA-Seq reads}. Usage of RNA-Seq reads (n = 30 WT, n = 29 TG, across 2, 4, 6 and 8 months) as expression similarly identified isoforms as differentially expressed (\cref{fig:DEI_ADgenes_isoseq}). \textbf{A)} ENSMUST00000151120.8 (PB.15108.6) annotated to \textit{Ctsd}, \textbf{B)} ENSMUST00000172785.7 (PB.7039.1) annotated to \textit{H2-D1}, \textbf{C)} ENSMUST00000028624.8 (PB.9298.1) annotated to \textit{Gatm}, and \textbf{D)} ENSMUST00000030765.6 (PB.11607.2) associated with \textit{Padi2}/\textit{Pad2}.  WT - Wild-type, TG - Transgenic. Dotted lines represent the mean paths across ages.}   
	\label{fig:DEI_ADgenes_rnaseq}
\end{figure}

\begin{figure}[!htp]
	\centering
	\includegraphics[page=15,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential isoform expressed observed with Iso-Seq reads as expression were not recapitulated using RNA-Seq reads]%
	{\textbf{Differential isoform expressed observed with Iso-Seq reads as expression were not recapitulated using RNA-Seq reads}. Shown are normalised counts \textbf{A)} \textit{Cd34} and \textbf{C)} \textit{Ubqln1} with associated isoforms identified from using Iso-Seq reads as expression, and of the same two genes \textbf{B)} \textit{Cd34} and \textbf{D)} \textit{Ubqln1} identified from using RNA-Seq reads as expression. Significant changes in isoform expression identified using Iso-Seq reads as expression - PB.1063.2 associated with \textit{Cd34} and PB.4255.13 and PB.4255.4 associated with \textit{Ubqln1} - was not recapitulated when using RNA-Seq reads as expression, due to lower sequencing coverage. Notably, minor isoforms had a higher expression with RNA-Seq alignment than direct detection with Iso-Seq reads.
	FSM - Full Splice Match, ISM - Incomplete Splice Match, NIC - Novel In Catalogue, NNC - Novel Not in Catalogue. Dotted lines represent the mean paths across ages.
	}   
	\label{fig:dei_lowisoexp}
\end{figure}

\begin{figure}[!htp]
	\centering
	\includegraphics[page=17,trim={0cm 20cm 0cm 0cm},clip,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential Isoform Expression observed in isoforms with high expression but large variance]%
	{\textbf{Usage of RNA-Seq reads as expression, with larger sample size, reduce probability of calling isoforms differentially expressed due to chance} Shown are \textbf{A)} \textit{Slc1a3} with associated isoforms identified as differentially expressed when Iso-Seq reads were used as expression (n = 6 WT, n = 6 TG, across 2 time points), and \textbf{B)} \textit{Slc1a3} when RNA-Seq reads were used as expression (n = 30 WT, n = 29 TG, across 4 time points). 
	\\
	\\	
	Using Iso-Seq reads as expression, the more-highly expressed isoforms PB.527.1 (red) in Figure a) was identified as differentially expressed. The expression change, however, of the same isoform was less prominent when RNA-Seq reads were used due to greater sample size and higher sequencing coverage. Notably, all the minor isoforms had a higher expression when RNA-Seq reads were used, resulting in some novel isoforms not being pre-filtered (as in the case with Iso-Seq reads as expression due to low full-length read count)
	\\
	\\
	FSM - Full Splice Match, ISM - Incomplete Splice Match, NIC - Novel In Catalogue, NNC - Novel Not in Catalogue. Dotted lines represent the mean paths across ages.
}   
	\label{fig:dei_highisoexp}
\end{figure}


\clearpage
\subsection{Iso-Seq read depth from global transcriptome profiling insufficient for differential transcript usage analysis}
Contributing to the complexity of transcript regulation through alternative splicing, while the expression of a gene may be constant between conditions, the relative expression of the isoforms (and thus isoform proportion) can change. This phenomenon is known as differential transcript/isoform usage (DIU), and was also assessed using \textit{tappAS} for genes with more than one isoform. Of note, a major isoform switching event occurs when the dominant (major) isoform in one condition becomes a minor isoform in another.    

Using Iso-Seq reads for annotation and expression and after filtering lowly-expressed isoforms, we identified 400 genes that were characterised with differential isoform usage. However upon further examination, the majority of these genes were lowly expressed and the associated isoforms that were observed to undergo differential usage had only 1-2 full-length long-read counts. We were therefore not confident in any of observed DIU changes detected with Iso-Seq abundance from the whole transcriptome sequencing due to low sequencing depth. Indeed, none of the DTU genes were significant after applying a gene expression threshold (described in \cref{ch:diu_method}).


\iffalse
\begin{figure}[htp]
	\begin{center}
		\includegraphics[page=10,trim={0cm 18cm 0cm 0cm},clip,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Genes characterised with differential isoform usage had low Iso-Seq full-length read counts]%
	{\textbf{Genes characterised with differential isoform usage had low Iso-Seq full-length read counts}: Shown are density plots of \textbf{A)} median full-length vs normalised counts and \textbf{B)} mean full-length vs normalised counts of DIU genes that were identified using Iso-Seq reads as expression. }   
	\label{fig:DIU_lowdepth}
\end{figure}
\fi

% DIU genes with major switching	
In further support of this conclusion, there was a low overlap of DIU genes that were identified when we used RNA-Seq reads as expression. This was likely to be a reflection of lower sequencing depth of Iso-Seq reads, resulting in i) a different pool of isoforms after filtering lowly-expressed isoforms - a minor isoform was more likely to be filtered when using RNA-Seq reads as abundance rather than Iso-Seq reads, particularly if the overall gene expression was low, due to relatively greater expression difference, and ii) smaller differences in isoform expression using Iso-Seq reads are translated to misleading significant changes in isoform proportions, resulting in false detection of genes as DIU. These implications can be seen in the gene \textit{Esyt2}, whereby the low Iso-Seq isoform counts resulted in misleading representation of isoform fraction, which was not recapitulated when RNA-Seq reads were used as expression (\cref{fig:DIU_esyt2}). 

\begin{figure}[htp]
	\begin{center}
		\includegraphics[page=9,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[\textit{Esyt2} was misidentified with differential isoform usage due to low Iso-Seq read counts]%
	{\textbf{\textit{Esyt2} was misidentified with differential isoform usage due to low Iso-Seq read counts}: \textbf{A)} Isoform expression (normalised counts) and \textbf{C)} subsequent deduction of isoform fraction of \textit{Esyt2} using Iso-Seq reads as expression, and the equivalent \textbf{B)} isoform expression and \textbf{D)} isoform fraction of the same gene, \textit{Esyt2} with RNA-Seq reads as expression. Iso-Seq reads were used as annotation in both analyses. 
		\\
		As can be observed, the Iso-Seq normalised counts are significantly lower than RNA-Seq normalised counts for the associated isoforms, resulting in a misleading representation of isoform fraction with major isoform switching events (Figure c, PB.3907.1 becomes the dominant isoform in transgenic mice), which is not recapitulated with RNA-Seq reads as expression (Figure d)}   
	\label{fig:DIU_esyt2}
\end{figure}

\subsection{Hybrid approach identifies tau-pathology associated differential isoform usage with major isoform switching events}
Differential isoform usage was performed with Iso-Seq reads as annotation and RNA-Seq reads as expression, given higher sequencing RNA-Seq read depth and greater sample size. 570 genes were identified with differential isoform usage (\cref{tab:DIU_DEA_nums}). 

\vspace{1cm}
\begin{table}[!htp]
	\centering
	\begin{tabularx}{0.85\textwidth}{cccc}
		\toprule
		\multicolumn{3}{c}{Conditions}                                                                                                                                                                                       & \multirow{2}{*}{Number of Genes} \\ \cmidrule(r){1-3}
		\begin{tabular}[c]{@{}c@{}}Differential Gene\\  Expression\end{tabular} & \begin{tabular}[c]{@{}c@{}}Differential Isoform \\ Usage\end{tabular} & \begin{tabular}[c]{@{}c@{}}Isoform Major\\  Switching\end{tabular} &                                  \\ \midrule
		\checkmark                                                                      & \checkmark                                                                    & \checkmark                                                                 & 13                               \\
		\checkmark                                                                      & \checkmark                                                                    & x                                                                  & 59                               \\
		x                                                                       & \checkmark                                                                    & \checkmark                                                                 & 39                               \\
		x                                                                       & \checkmark                                                                    & x                                                                  & 459                              \\ \midrule
		\multicolumn{3}{c}{Total Number of Genes}                                                                                                                                                                            & 570                              \\ \bottomrule
	\end{tabularx}
	\caption[Number of Genes identified with differential gene expression and isoform usage]%
	{\textbf{Number of Genes identified with differential gene expression and isoform usage}. Tabulated are the number of genes identified from differential gene expression and isoform usage analysis from \textit{tappAS} with Iso-Seq reads as annotation and RNA-Seq reads as expression. The models for each condition are depicted in \cref{fig:DIU_DEA_model}. Isoform major switching refers to the event of an isoform being predominantly expressed in one condition (major) but lowly-expressed in another condition (minor)}
	\label{tab:DIU_DEA_nums}
\end{table}

The majority of these genes (n = 498, 87.3\%), while observed with DIU, were not differentially expressed between wild-type and transgenic mice - a scenario whereby the differential upregulation of one isoform is compensated by the differential downregulation of another isoform, resulting in no net gene expression change. This indicates that a significant degree of the post-transcriptional regulation was independent of gene expression regulation. An example of this was \textit{Ptprz1}, a receptor protein tyrosine phosphatase, which is involved in oligodendrocyte development and function. While there was no change in overall gene expression, closer examination revealed progressive upregulation of two isoforms (PB.13000,10, PB.13000.11) which was offset by progressive downregulation of two other isoforms (PB.13000.14, PB.13000.7) in transgenic mice. In transgenic mice, the known isoform PB.13000.10 (ENSMUST00000090568.6) continued to dominate in both wild-type and transgenic mice across all ages with a slight increase. This was complemented by a slight decrease in the another known shorter isoform PB.13000.14 (ENSMUST00000202579.3). Interestingly, the two other differentially expressed isoforms were novel, with skipping of exon 16. 

The most significant gene characterised with differential isoform usage with a dominant isoform switch but no change in gene expression was \textit{Cisd3}, a mitochondrial iron-sulphur domain-containing protein involved in regulating electron transport and iron homeostasis essential for normal mitochondrial function. While there was little change in overall gene expression, there is a major isoform shift between wild-type and transgenic mice that was generally consistent across all ages. The two isoforms only differ at the 5'end with the first exon of the upregulated isoform, ENSMUST00000107583.2 (PB.2833.2), spanning across exons 1 and 2 of the downregulated isoform, ENSMUST00000107584.7 (PB.2833.1). 
% Predicted protein structure

%GFAP not significant?   
%Interestingly, the was a lower consensus when comparing the number of DIU genes identified when using RNA-Seq reads as abundance, suggesting that the choice of filtering strategy becomes redundant when dealing with low sequencing depth.  


\begin{figure}[htp]
	\begin{center}
		\includegraphics[page=1,scale = 0.55]{Figures/DIU_notDEG_nomajor.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential isoform expression and usage of \textit{Ptprz1}]%
	{\textbf{Differential isoform expression and usage of \textit{Ptprz1}}: \textbf{A)} \textit{Cisd3}'s RNA-Seq gene expression in wild-type (grey) and transgenic (red) mice, with no significant overall change. \textbf{B)} RNA-Seq expression of the two known and two novel isoforms in wild-type and mice across age. \textbf{C)} Overall isoform fraction of two isoforms across wild-type and transgenic, \textbf{D)} and by age.}    
	\label{fig:DIU_ptprz1}
\end{figure}


\begin{figure}[htp]
	\begin{center}
		\includegraphics[page=1,scale = 0.55]{Figures/DIU_notDEG_major.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential isoform expression and usage of \textit{Cisd3}]%
	{\textbf{Differential isoform expression and usage of \textit{Cisd3}}: \textbf{A)} \textit{Cisd3}'s RNA-Seq gene expression in wild-type (grey) and transgenic (red) mice, with no significant overall change. \textbf{B)} RNA-Seq expression of the two known isoforms in wild-type and mice across age. \textbf{C)} Overall isoform fraction of two isoforms across wild-type and transgenic, \textbf{D)} and by age.}    
	\label{fig:DIU_Cisd3}
\end{figure}


\clearpage
\subsection{Integrative analysis reveals co-localisation of genotype-associated differentially expressed transcripts and differentially-methylated positions}
Epigenetic modifications such as DNA methylation have been widely acknowledged to heavily influence gene regulation and more specifically, alternative splicing. Using genome-wide DNA methylation data on the same mouse samples, we wanted to use an integrative approach to identify genes with both splicing and methylation changes between wild-type and transgenic mice. Genes defined to undergo differential splicing were classified by either genes with a change in transcript expression (differential isoform expression - DIE), a change in usage of isoform proportions (differential isoform usage - DIU) or a change in feature element (differential feature inclusion - DFI). Genes with changes in methylation were classified as either with a change in a specific position (differential methylated position - DMP) associated with tau pathology (genotype effect) and progressively over time (interaction effect), or associated with a cluster of differentially methylated cytosines (differentially methylated region - DMR). 

Using a stringent approach to identify biologically meaningful splicing changes, we identified nine genes with an altered splicing and methylation profile associated with progressive tau pathology. The most significant co-differentially-methylated position and differentially-spliced gene was \textit{Spata13} - a gene involved in cell migration\cite{Bourbia2019} and has been reported to be upregulated in entorhinal cortex of AD patients\cite{Yan2019} - which was characterised with a change in transcript expression, usage and feature inclusion. There was a significant progressive increase in expression of the shorter known isoform (PB.4966.2, ENSMUST00000162945.1) in the transgenic mice, whereas there was no change in expression of the longer, novel NIC isoform (PB.4966.1) which harboured a 5' miRNA-binding site (miR-446h-5p) and an alternative transcription start site with an upstream open reading frame (\cref{fig:IntMeth_Spata13}\textbf{a, d}). A DMP was identified upstream of \textit{Spata13} (8.9Kb and 107Kb from known and novel isoform respectively, \cref{fig:IntMeth_Spata13}\textbf{b, d}) and was hypermethylated in transgenic mice compared to wild-type ($\Delta$$\beta$ = 0.777, FDR\textsubscript{interaction} = 0.012), \cref{fig:IntMeth_Spata13}\textbf{C}), suggesting a distal suppressed expression of both isoforms in the wild-type. Conversely, the decreased methylation in the transgenic is only associated with suppressed expression of the proximal, novel isoform. 

\begin{figure}[ht]
	\includegraphics[page=2,scale = 0.4]{Figures/WholeDifferentialAnalysis_DMPDMR.pdf}
	\\
	\hspace*{0.2cm}\vspace{0.5cm}\large d
	\\
	\includegraphics[page=1,trim={1.5cm 0 0 0},scale = 0.9]{Figures/SPATA13_DMP.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential splicing and methylation of \textit{Spata13}]%
	{\textbf{Differential splicing and methylation of \textit{Spata13}}: \textbf{A}) Upregulation of known isoform (PB.4966.2, ENSMUST00000162945.1) associated with \textit{Spata13} in TG (P = 9.42 x 10 \textsuperscript{-33}, R\textsuperscript{2} = 0.73) \textbf{B)} Hypermethylated DMP identified upstream of promoter ($\Delta$$\beta$ = 0.777, FDR\textsubscript{interaction} = 0.012). \textbf{C)} Correlation between DNA methylation at this site and normalised isoform expression of the \textit{Spata13} gene. \textbf{D)} UCSC browser track showing relative position of DMP and two isoforms. }    
	\label{fig:IntMeth_Spata13}
\end{figure}

%Another gene with co-differentially-methylated position and differentially-spliced gene was \textit{Tmem237}, characterised by an isoform switch of a known (PB.261.4) and novel NNC (PB.261.10) isoform, which had an alternative first exon and a shorter 3'UTR resulting in absence of miRNA-binding sites. 
% check methylation levels across first exon.

\newpage
Other genes were characterised with a change in transcript expression correlated with a change in methylation. This included \textit{Ncf2} (\cref{fig:IntMeth_Ncf2}), \textit{Osmr}, and \textit{Cebpa} where there was a progressive increase in expression of the respective known isoform and an associated hypomethylated DMP (located in the promoter) in transgenic; whereas in \textit{Rnf165} and \textit{Susd5} (\cref{fig:IntMeth_Susd5}), there was a progressive downregulation of known isoform and an associated hypermethylated DMP in transgenic. Interestingly, we also identified altered splicing and methylation changes in \textit{Irf8} (\cref{fig:IntMeth_Irf8}) - a transcription factor of the IRF family that is involved in microglial activation in AD\cite{Zeng2017}, where a hypomethylated DMP (located 130kb downstream) in TG was associated with progressive increase in transcript expression. The associations described here between DNA methylation and transcript expression support well-established theories of the relationship between DNA methylation and gene expression, whereby increased methylation at the promoter is associated with decreased gene expression. Of note, only one isoform was detected for each of these genes.  
% Note other example where this was not the case 

\begin{figure}[h]
	\includegraphics[page=3,scale = 0.4]{Figures/WholeDifferentialAnalysis_DMPDMR.pdf}
	\\
	\hspace*{0.2cm}\vspace{0.5cm}\large d
	\\
	\includegraphics[page=1,trim={1.5cm 0 0 0},scale = 0.9]{Figures/NCF2_DMP.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential splicing and methylation of \textit{Ncf2}]%
	{\textbf{Differential splicing and methylation of \textit{Ncf2}}. \textbf{A}) Upregulation of known isoform associated with \textit{Ncf2} (PB.700.1,ENSMUST00000027754.6) in TG coincided with two \textbf{B)} hypomethylated DMP located upstream of promoter. Only the most significant DMP is shown ($\Delta$$\beta$ = -0.293, FDR\textsubscript{interaction} = 0.039). \textbf{C)} Correlation between DNA methylation at this site and normalised isoform expression of the \textit{Ncf2} gene. \textbf{D)} UCSC browser track showing relative position of DMP and isoform.}    
	\label{fig:IntMeth_Ncf2}
\end{figure}

\begin{figure}[]
	\includegraphics[page=4,scale = 0.4]{Figures/WholeDifferentialAnalysis_DMPDMR.pdf}
	\\
	\hspace*{0.2cm}\vspace{0.5cm}\large d
	\\
	\includegraphics[page=1,trim={1.5cm 0 0 0},scale = 0.9]{Figures/SUSD5_DMP.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential splicing and methylation of \textit{Susd5}]%
	{\textbf{Differential splicing and methylation of \textit{Susd5}}: \textbf{A}) Downregulation of known isoform associated with \textit{Susd5} (PB.16983.1,ENSMUST00000135338.2) in TG (P = 5.93 x 10\textsuperscript{-52}, R\textsuperscript{2} = 0.84) \textbf{B)} Five hypermethylated DMPs were identified in the intronic region after the first exon. Only the most significant DMP is shown ($\Delta$$\beta$ = 0.344, P\textsubscript{Genotype} = 1.63). \textbf{C)} Correlation between DNA methylation at this site and normalised isoform expression of the \textit{Susd5} gene. \textbf{D)} UCSC browser track showing relative position of DMPs and isoform}    
	\label{fig:IntMeth_Susd5}
\end{figure}

\begin{figure}[]
	\includegraphics[page=5,scale = 0.4]{Figures/WholeDifferentialAnalysis_DMPDMR.pdf}
	\\
	\hspace*{0.2cm}\vspace{0.5cm}\large d
	\\
	\includegraphics[page=1,trim={1.5cm 0 0 0},scale = 0.9]{Figures/IRF8_DMP.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential splicing and methylation of \textit{Irf8}]%
	{\textbf{Differential splicing and methylation of \textit{Irf8}}: \textbf{A}) Upregulation of known isoform associated with \textit{Irf8} (PB.15969.1, ENSMUST00000047737.9) in TG (P = 8.17 x 10\textsuperscript{-78}, R\textsuperscript{2} = 0.87)\textbf{B)} Two hypermethylated DMPs were identified downstream in the distal intergenic region. Only the most significant DMP is shown ($\Delta$$\beta$ = -0.464, FDR\textsubscript{Interaction} = 0.0368) \textbf{C)} Correlation between DNA methylation at this site and normalised isoform expression of the \textit{Irf8} gene. \textbf{D)} UCSC browser track showing relative position of DMPs and isoform}    
	\label{fig:IntMeth_Irf8}
\end{figure}

Focusing on genotype-associated differentially methylated regions, we identified splicing changes annotated to \textit{As3mt}  (\cref{fig:IntMeth_As3mt}) and \textit{Prnp}. Three isoforms were detected for \textit{As3mt}, a schizophrenia-associated GWAS gene, whereby there was a progressive upregulation in the known isoform (PB.8363.2, ENSMUST00000003655.8) in TG (P = 4.98 x 10\textsuperscript{-18}, R\textsuperscript{2} = 0.56) while there was no detected change in expression of the two other novel NNC isoforms, both of which were characterised by a novel splice junction resulting in a novel exon that was absent in the known isoform. A 485bp DMR consisting of nine differentially methylated cytosines was identified \textasciitilde{}8kb upstream of this novel exon within the intron, and was hypermethylated in transgenic mice compared to wild-type ($\Delta$ = 0.19, P = 1.22 x 10\textsuperscript{-8}). We hypothesise that an increase in methylation in this region would hinder recruitment and assembly of the spliceosome, resulting in skipping of the novel exon and subsequent increased expression of the known isoform. 

There was an increase in expression of a novel isoform of \textit{Prnp} in transgenic mice. However, given that \textit{Mapt} transgene includes exons 2 and 3 of the \textit{Prnp}, we were uncertain whether the transcript expression changes attributed to \textit{Prnp} from short-read RNA-Seq reads was due to the mouse gene or the human transgene. No change in transcript expression was identified from Iso-Seq full-length read counts. 

\begin{figure}[htp]
	\includegraphics[page=1,scale = 0.4]{Figures/WholeDifferentialAnalysis_DMPDMR.pdf}
	\\
	\hspace*{0.2cm}\vspace{0.5cm}\large c
	\\
	\includegraphics[page=1,trim={1.5cm 0 0 0},scale = 0.9]{Figures/AS3MT_DMP.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential splicing and methylation of \textit{As3mt}]%
	{\textbf{Differential splicing and methylation of \textit{As3mt}}: \textbf{A}) Upregulation of known isoform associated with \textit{As3mt} (PB.8363.2, ENSMUST00000003655.8) in TG (P = 4.98 x 10\textsuperscript{-18}, R\textsuperscript{2} = 0.56) \textbf{B)} A DMR of nine hypermethylated sites was identified upstream of 11th novel exon. \textbf{D)} UCSC browser track showing relative position of DMR and isoform}    
	\label{fig:IntMeth_As3mt}
\end{figure}	

\newpage
\section{Conclusions}






