\chapter{Splicing signatures of progressive tau pathology in AD mouse model}\label{ch: transcriptional_global_differences}

\section{Introduction}
There is increasing evidence for the role of transcriptional dysregulation and aberrant splicing in AD development and pathogenesis (described in \cref{intro:AD_alteredsplicing}). Recent transcriptome profiling studies have identified changes in splicing and transcript expression in human AD post-mortem brain tissues and AD mouse models (reviewed in \cref{tab: AS_ADHuman_studies} and \cref{tab: AS_ADMouse_studies}, respectively). However to date, these studies have relied on short-read RNA-Sequencing approaches, which cannot reliably detect RNA isoforms (discussed in \cref{rnaseq_intro}) and thereby renders any attempts to perform downstream transcript-based analysis futile. In contrast, we have illustrated the power of long-read sequencing to identify full-length transcripts and improve our annotations of alternatively spliced isoforms in the cortex of a mouse model of AD tauopathy, rTg4510 (\cref{ch: whole_transcriptome}). 

While long-read sequencing approaches are currently considered semi-quantitative, recent studies have delivered promising strategies for transcript-based analysis by using a hybrid approach\cite{Tseng2021}: the alignment of short-read data to improved transcriptome annotations derived from long-read data. This has enabled identification of differentially expressed isoforms and evaluation of differential isoform usage between experimental groups\cite{Tseng2021}. 

Following on from \textbf{Chapter 4}, this chapter aimed to exploit the long-read sequencing datasets generated from rTg4510 transgenic (TG) and wild-type (WT) control mice to identify transcriptional and splicing changes associated with progressive tau pathology. The following objectives are: 
\begin{enumerate}
	\item To assess global variation in splicing patterns between rTg4510 TG and WT mice
	\item To perform differential gene expression analysis and validate changes in gene expression associated with tau pathology from previous RNA-Seq studies 
	\item To perform differential transcript expression analysis to identify changes in transcript expression associated with tau pathology   
	\item To perform differential transcript usage analysis to identify genes with significant altering of isoform proportions between rTg4510 transgenic and control mice 
	\item To integrate differential splicing changes with epigenetic data (DNA methylation) available from the same samples
\end{enumerate} 

\newpage
\section{Methods}

\subsection{Datasets}
All analyses pertaining to this chapter follow on from \cref{ch: whole_transcriptome} using the same Iso-Seq long-read datasets generated from 12 female mice (WT = 6, rTg4510 TG = 6, aged 2 and 8 months, \cref{tab:whole_phenotype}). Briefly, RNA was prepared for Iso-Seq library preparation and PacBio SMRT sequencing on the Sequel (\cref{ch4_methods: isoseq_library}), followed by QC and data processing (\cref{ch4_methods: isoseq_data}); reads from individual samples were processed separately with \textit{IsoSeq3} and merged for transcript collapse with \textit{Cupcake}, genome alignment with \textit{Minimap2}(v2.17) and re-annotation with \textit{SQANTI3} with no splice junction filtering from short-read RNA-Seq data. ISM transcripts with only the 3' fragment matching reference transcript (3'ISM) were considered technical artefacts from 5'degradation and thus removed.  

\subsection{Quantification of human MAPT transgene expression} 
The presence of human- and mouse-specific \textit{Mapt/MAPT} sequences was determined in FL transcripts as QC check of sample identity. Species-specific \textit{Mapt/MAPT} sequence, located in a 2kb region present in the 3'UTR, was identified after using BLAT\cite{Kent2002} to compare human and mouse MAPT/Mapt sequence for divergent transcript sequences \cite{Castanho2020}).  

\subsection{Characterisation of Alternative Splicing Events} 
Alternative splicing events were examined using a range of packages and custom scripts (as described and implemented in \cref{ch4: transcriptome_annotation}), to assess whether there was a change in splicing patterns associated with progressive tau pathology in rTg4510 mouse model. 

\subsection{Gene and Isoform Quantification}
Gene and isoform expression were estimated using two approaches (described in \cref{sec: gene_isoform_quant_explained}): i) alignment of short-read RNA-Seq reads to the Iso-Seq-defined transcriptome (hybrid approach) using \textit{Kallisto}\cite{Bray2016} (v0.46.0), and ii) normalised Iso-Seq FL read counts as proxy for expression. FL read counts for each sample were taken from \textit{read\_stat.txt} file generated from \textit{collapse\_isoforms\_by\_sam.py} script (\textit{Cupcake}), using the sequencing run ID as identifier. 

\subsection{Differential analysis}
Differential expression analysis was performed with \textit{tappAS} (fully described in \cref{ch3_tappas_explained}). Briefly, \textit{tappAS} filters out lowly-expressed isoforms, normalises read counts using TMM approach and performs differential expression analysis using \textit{maSigPro}\cite{Conesa2006,Nueda2014,Conesa2017} to elucidate effects for both genotype and age with the following adapted model\cite{Conesa2006}: 

\vspace{1cm}
\begin{myequation}[h]
Let \textit{I} denote the genotype groups (wild-type - WT, transgenic - TG) and \textit{J} as the age (2 and 8 months) for each particular group, and assuming that gene or transcript expression in measured in replicated samples (\textit{R}).  
\begin{align}
	y_{ijr} =  \:&\beta_{0} + \beta_{1}D_{ijr} \nonumber
	\\ &+ \delta_{0}T_{ijr} + \delta_{1}T_{ijr}D_{ijr}   \nonumber
\end{align}
where
\begin{conditions*}
	y_{ijr} & normalised expression value for each gene or transcript in the situation \textit{ijr} (genotype group \textit{i} at age \textit{j} of replicate \textit{r}) \\
	D  &  dummy binary variable to distinguish between the genotype groups, whereby 0 refers to reference group (WT) and 1 refers to experimental group (TG) \\
	T  &  age at 2 8 months described using a polynomial model with a degree of 1 \\
	\beta_{0}, \delta_{0}, \gamma_{0}, \lambda _{0} & regression coefficients for reference group (WT) relating to the age \\ 
	\beta_{1}, \delta_{1}, \gamma_{1}, \lambda _{1} & regression coefficients for the difference between the experimental group (TG) and reference group (WT) at each age  
\end{conditions*}
therefore, if:
\begin{conditions*}
	FDR(\beta_{1}) < 0.05 & significant expression difference between WT and TG at 2 months \\ 
	FDR(\delta_{0}) < 0.05 & significant expression difference in WT across 2 and 8 months \\
\end{conditions*}
\captionsetup{width=1\textwidth}
\caption[Linear regression model to determine differential gene and transcript expression]%
{\textbf{Linear regression model to determine differential gene and transcript expression}. The model, adapted from \textit{MaSigPro} and implemented as part of \textit{tappAS}, describes gene or transcript expression between two groups (WT - wild-type, TG - transgenic) at four different time points (age in months). FDR - False discovery rate}    
\end{myequation}

\begin{figure}[!htp]
	\centering
	\includegraphics[page=2,trim={0 5cm 0 4cm},scale = 0.45]{Figures/Tg4510_diff_figures.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Different conditions modelled for rTg4510 genotype and age effects]%
	{\textbf{Different conditions modelled for rTg4510 genotype and age effects.} Statistical models generated to dissect genotype and age effects with \textit{maSigPro} using \textbf{Equation 5.1} for 2 experimental groups (WT - wild-type/Control, TG - Transgenic/Case) across two time points/age (T1, T2). 
	\\\\
	The regression coefficients from \textbf{Equation 5.1} - $\beta_{1}$, $\delta_{0}$, $\delta_{1}$ - refer to the different variables modelled, the significance of which can be used to infer whether there is a genotype, age or interaction effect. The significance is symbolised by the tick and cross, which refers to adjusted P-value (FDR) < 0.05 and > 0.05 respectively. A significance of  $\beta_{1}$ denotes to a statistically significant difference between WT and TG at T1 (Genotype effect),$\delta_{0}$ to a difference in WT over time (Age effect), and $\delta_{1}$ to a difference between WT and TG across age (Interaction effect).}   
	\label{fig:dea_model}
\end{figure}

Under this model, a differentially expressed gene or transcript between WT and TG mice across age was defined with a statistically significant coefficient (adjusted P-value < 0.05) and a regression model with R\textsuperscript{2} > 0.5  (\cref{fig:dea_model}) .


\clearpage 
\section{Results}
\subsection{PacBio Iso-Seq run performance and sequencing metrics}
No significant difference in sequencing run yield was identified between WT and TG (n = 12 animals, two-tailed unpaired t-test, t(10) = -0.636, P = 0.539, \cref{fig:rTg4510_sequencing_metrics}\textbf{A}), and no significant correlation was observed between run yield and RIN across samples (n = 12 animals, Pearson's correlation, t = -0.98, df = 10, P = 0.350, \cref{fig:rTg4510_sequencing_metrics}\textbf{B})- although interestingly, WT samples generally had a higher RIN. No difference was observed in the number of reads (\cref{fig:rTg4510_sequencing_metrics}\textbf{C}) and transcripts generated between WT and TG (n = 12, two-tailed unpaired t-test, t = -0.005, df = 10, P = 0.996, \cref{fig:rTg4510_sequencing_metrics}\textbf{E}) or by age (n = 12, t = -1.58, df = 10, P = 0.15). Notably, a similar read profile was attained for all the samples bar the first two samples, which were sequenced using an older chemistry and had a relatively lower throughput. Nonetheless, all the samples were successfully sequenced with optimal runs, as indicated by the high throughput and the similar number of FL, FLNC and Poly-A FLNC reads recovered. ERCC alignment and annotations similarly revealed no difference in number of ERCC detected between WT and TG (mean number of ERCC: WT = 32.4 (35\%), TG = 32.2 (35.22\%)). 

\begin{figure}[htp]
	\begin{center}
		\includegraphics[page=1,trim={0 0 0 0},clip,scale = 0.55]{Figures/rTg4510WholeTranscriptome.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Sequencing and data processing metrics of rTg4510 mice]%
	{\textbf{No significant difference in sequencing metrics, number of transcripts and read length were observed between WT and rTg4501 TG mice}: \textit{Caption continues on the following page.}}
	\label{fig:rTg4510_sequencing_metrics}
\end{figure}
\begin{figure}[p]
	\captionsetup{width=0.95\textwidth}
	\contcaption{\textbf{(A)} Shown is a box-plot of the total yield generated from Iso-Seq sequencing of rTg4510 WT (n = 6) and TG mice (n = 6). Full details of all runs are provided in \cref{tab:isoseq_wholerun_result}. \textbf{(B)} A scatter plot of the total yield generated and the RIN attained for each sample (RIN refers to the quality of RNA used for library preparation). \textbf{(C)} The number of reads generated through the Iso-Seq bioinformatics pipeline from initial generation of CCS reads, to FL reads with primer removal, and poly-A FLNC reads with removal of artificial concatemers and trimming of poly(A) tails. Note, the first two samples with lower throughput were sequenced with an older chemistry. \textbf{(D)} A box-plot of the total number of FL transcripts generated for WT and TG samples. \textbf{(E)} Distribution of CCS read length. CCS - Circular Consensus Sequence, FL - Full-length, FLNC - Full-length non-chimeric, Gb - Gigabases, K - Thousand, kb - Kilobases, TG - Transgenic, WT - Wild-type}%
\end{figure}


\subsection{\textit{MAPT} transgene only expressed in rTg4510 TG mice}
%Check whether overexpression of human MAPT result in any unwanted, compensatory effects on equivalent mouse genes,as expression levels of mouse APP and MAPT should be slightly reduced, thereby suggesting no evidence that human transgene expression increase expression of directly-related mouse genes.
As expected, human-specific \textit{MAPT} sequences were only detected in reads from TG mice, confirming stable activation of human \textit{MAPT} transgene (\cref{fig:isoseq_humanmapt}\textbf{A}) and supporting findings from Castanho et al.(2020)\cite{Castanho2020}. Alignment of these human-specific transcripts to the mouse genome were either mapped to the mouse prion protein gene (\textit{Prnp}) with high identity but low coverage/alignment length (\cref{fig:isoseq_humanmapt}\textbf{B,C}) or to the mouse \textit{Mapt} gene with low identity but high alignment length (\cref{fig:isoseq_humanmapt}\textbf{B,D}); this is reflective of the transgene sequence in rTg4510 mouse model, given it contains exons 2 and 3 of mouse \textit{Prnp}\cite{Ramsden2005} and is homologous to the mouse \textit{Mapt} gene. Notably, applying filter thresholds (85\% alignment identity and 95\% alignment length) for downstream analysis removed these human-specific \textit{MAPT} transcripts (\cref{fig:isoseq_humanmapt}\textbf{B}). 

\begin{figure}[htp]
	\begin{center}
		\includegraphics[page=1,trim={0cm 11cm 0cm 0cm},clip,scale = 0.80]{Figures/AltFigures_Diff.pdf}	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Quantifying human-specific and mouse-specific \textit{MAPT}/\textit{Mapt} sequences in Iso-Seq Whole Transcriptome]%
	{\textbf{Human-specific \textit{MAPT} sequences only present in transgenic mice with poor alignment to mouse \textit{Prnp} and \textit{Mapt} gene}: Presence of human- and mouse-specific \textit{MAPT}/\textit{Mapt} sequences was determined in full-length transcripts generated from Iso-Seq merged dataset. \textbf{(A)} Ratio of full-length transcripts that were mapped to human-specific \textit{MAPT} and mouse-specific \textit{Mapt} sequences. Dotted lines represent the mean paths across ages. \textbf{(B)} Human-specific \textit{MAPT} transcripts were poorly aligned to mouse genome, as expected. Transcripts were either aligned to mouse \textit{Prnp} gene with high identity but low length, given that the transgenic contains only exon 2 and 3 of mouse \textit{Prnp} gene\cite{Ramsden2005} (boxed yellow) or  mouse \textit{Mapt} gene with low alignment identity and length (boxed blue). Green box refers to the transcripts retained after applying identity and length threshold. \textbf{(C)} USCS genome browser tracks of human-specific (black) \textit{MAPT} transcripts (transgene) and mouse \textit{Prnp} gene and \textbf{(D)} mouse \textit{Mapt} gene. Blue tracks represent known transcripts from reference mouse genome (mm10). The double horizontal lines indicate unalignable sequences and the red lines indicate bases that deter between the genome and read. Tracks were cropped and modified to remove irrelevant genes within the same locus.  UTR - Untranslated region}
	\label{fig:isoseq_humanmapt}
\end{figure}

\clearpage
\subsection{rTg4510 mice exhibited similar global transcriptomic profile and splicing patterns}
Despite identifying widespread RNA isoform diversity amongst genes expressed in the mouse entorhinal cortex (\cref{ch: whole_transcriptome}), the global transcriptomic profile between rTg4510 WT and TG mice were very similar; no difference was observed in the number of genes (mean number = 13,572 genes) or isoforms (mean number = 53,833 isoforms). Further characterisation of the transcriptome revealed similar profile of isoform diversity across genotype and age (\cref{tab:isoseq_whole_subsqantioutput}), with half of the isoforms annotated as known and FSM (mean number = 30,018 isoforms (55.8\%), as also shown in \cref{sec:whole_novelIso}) and with a similar distribution in isoform length and number of exons (median: 8, range = 1-89). Splicing patterns were also very similar across genotype and age with usage of alternative first exons (AF) (mean n = 12,564, 35\%) (\cref{AS_WholeTranscriptome_diff}) as the most prevalent across all datasets, in line with previous findings (\cref{sec:whole_novelIso}).

\vspace{2cm}
\begin{table}[!htp]
	\centering
	\captionsetup{width=1\textwidth}
	\caption[Alternative Splicing Events associated with tau pathology and age]%
	{\textbf{Alternative Splicing Events associated with tau pathology and age}. Tabulated is the number of splicing events detected for wild-type and transgenic Tg4510 mice aged 2 and 8 months (n = 12, 3 biological replicates per group)}
	\begin{tabular}{@{}ccccc@{}}
		\toprule
		\multirow{2}{*}{Splicing  Events} & \multicolumn{2}{c}{Wildtype} & \multicolumn{2}{c}{Transgenic} \\ \cmidrule(l){2-5} 
		& 2 months        & 8 months        & 2 months        & 8 months        \\ \midrule
		A3 & 2164 (6.58\%)   & 2571 (6.61\%)   & 2388 (6.77\%)   & 2388 (6.5\%)    \\
		A5 & 1369 (4.16\%)   & 1589 (4.09\%)   & 1473 (4.18\%)   & 1488 (4.05\%)   \\
		AF & 12048 (36.61\%) & 13073 (33.61\%) & 12514 (35.48\%) & 12622 (34.36\%) \\
		AL & 8140 (24.73\%)  & 9688 (24.91\%)  & 8641 (24.5\%)   & 9287 (25.28\%)  \\
		IR & 3611 (10.97\%)  & 5404 (13.9\%)   & 4293 (12.17\%)  & 4774 (13\%)     \\
		MX & 299 (0.91\%)    & 392 (1.01\%)    & 331 (0.94\%)    & 329 (0.9\%)     \\
		SE & 5278 (16.04\%)  & 6174 (15.88\%)  & 5632 (15.97\%)  & 5846 (15.91\%)  \\ \bottomrule
	\end{tabular}
	\label{AS_WholeTranscriptome_diff}
\end{table}

%stats?
%XX of known transcripts were identified to have intron retention; XX of known transcripts were identified to be fusion genes. XX of know transcripts identified to have non-sense-mediated decay. 

\begin{landscape}
	\begin{table}[]
		\centering
		\captionsetup{width=1\linewidth}
		\caption[Overview of the whole transcriptome Iso-Seq datasets generated from mouse rTg4510, subsected by phenotype and age]%
		{\textbf{Overview of the whole transcriptome Iso-Seq datasets generated from mouse rTg4510, subsected by phenotype and age}. Annotations from wild-type (n = 6) and transgenic mouse (n = 6) were generated from merging Iso-Seq datasets from mouse aged 2 and 8 months of the respective phenotype. Novel genes refer to genes that were not currently present in existing genome annotations (mm10). Isoform can be further classified as known (FSM, ISM) or novel (ISM, NIC, NNC, Genic Genomic, Antisense, Fusion, Intergenic, Genic Intron), as described in \cref{sec:sq_exp}. FSM – Full Splice Match, ISM – Incomplete Splice Match, NIC – Novel In Catalogue, NNC – Novel Not in Catalogue.}
		\label{tab:isoseq_whole_subsqantioutput}
		\resizebox{1.5\textwidth}{!}{%
		\begin{tabular}{@{}ccccccc@{}}
		\toprule
		\multicolumn{1}{l}{} &
		\begin{tabular}[c]{@{}c@{}}Wildtype \\ (n = 6)\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Transgenic \\ (n = 6)\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Wildtype, 2 months \\ ( n = 3)\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Wildtype, 8 months \\ ( n = 3)\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Transgenic, 2 months\\ ( n = 3)\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Transgenic, 8 months \\ ( n = 3)\end{tabular} \\ \midrule
		Total Number of Genes             & 14118           & 14213           & 13191           & 13312           & 12985           & 13616           \\
		Annotated Genes                   & 13932 (98.68\%) & 14031 (98.72\%) & 13081 (99.17\%) & 13168 (98.92\%) & 12874 (99.15\%) & 13474 (98.96\%) \\
		Novel Genes                       & 186 (1.32\%)    & 182 (1.28\%)    & 110 (0.83\%)    & 144 (1.08\%)    & 111 (0.85\%)    & 142 (1.04\%)    \\
		Total Number of Isoforms          & 62533           & 63038           & 48516           & 50278           & 45903           & 52730           \\
		FSM                               & 33239 (53.15\%) & 33563 (53.24\%) & 27878 (57.46\%) & 28689 (57.06\%) & 26825 (58.44\%) & 29916 (56.73\%) \\
		ISM                               & 4927 (7.88\%)   & 4864 (7.72\%)   & 3426 (7.06\%)   & 3841 (7.64\%)   & 3279 (7.14\%)   & 3764 (7.14\%)   \\
		NIC                               & 15305 (24.48\%) & 15595 (24.74\%) & 11012 (22.7\%)  & 11407 (22.69\%) & 10214 (22.25\%) & 12369 (23.46\%) \\
		NNC                               & 8518 (13.62\%)  & 8484 (13.46\%)  & 5838 (12.03\%)  & 5953 (11.84\%)  & 5259 (11.46\%)  & 6282 (11.91\%)  \\
		Genic Genomic                     & 63 (0.1\%)      & 61 (0.1\%)      & 44 (0.09\%)     & 44 (0.09\%)     & 32 (0.07\%)     & 47 (0.09\%)     \\
		Antisense                         & 97 (0.16\%)     & 104 (0.16\%)    & 52 (0.11\%)     & 77 (0.15\%)     & 68 (0.15\%)     & 75 (0.14\%)     \\
		Fusion                            & 276 (0.44\%)    & 268 (0.43\%)    & 200 (0.41\%)    & 186 (0.37\%)    & 167 (0.36\%)    & 196 (0.37\%)    \\
		Intergenic                        & 108 (0.17\%)    & 99 (0.16\%)     & 66 (0.14\%)     & 81 (0.16\%)     & 59 (0.13\%)     & 81 (0.15\%)     \\
		Genic Intron                      & 0 (0\%)         & 0 (0\%)         & 0 (0\%)         & 0 (0\%)         & 0 (0\%)         & 0 (0\%)         \\
		Isoform Length (bp) &
		\begin{tabular}[c]{@{}c@{}}Median: 2691, \\ Range: 82-15016\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Median: 2698, \\ Range: 82-15913\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Median: 2740, \\ Range: 88-15016\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Median: 2614, \\ Range: 82-14850\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Median: 2548, \\ Range: 88-14302\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Median: 2754, \\ Range: 82-15913\end{tabular} \\
		Number of Exons &
		\begin{tabular}[c]{@{}c@{}}Median: 8, \\ Range: 1-89\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Median: 8, \\ Range: 1-89\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Median: 9, \\ Range: 1-89\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Median: 8, \\ Range: 1-89\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Median: 8, \\ Range: 1-77\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Median: 9, \\ Range: 1-89\end{tabular} \\
		Number of Isoforms with 50bp CAGE & 52096 (83.31\%) & 52633 (83.49\%) & 40589 (83.66\%) & 42378 (84.29\%) & 38227 (83.28\%) & 44729 (84.83\%) \\ \bottomrule
	\end{tabular}%
		}
		\end{table}
\end{landscape}

 
\subsection{Iso-Seq confirms widespread gene expression differences associated with tau pathology in rTg4510 mice}
%\boldheader{Usage of Iso-Seq reads alone detected robust changes in gene expression}
Although long-reads are considered less quantitative than traditional short-read RNA sequencing approaches, we previously demonstrated the power of Iso-Seq to accurately quantify the abundance of highly-expressed transcripts (described in \cref{sec: whole_isoseqvsrnaseq}). Subsequently, we sought to evaluate the utility of full-length read count from Iso-Seq reads as a proxy of abundance to identify changes in gene expression associated with progressive tau pathology in TG mice; a recent RNA-Seq study in our group  identified extensive gene expression differences in the same mouse model from mapping short-reads to the mouse reference genome annotation\cite{Castanho2020}.

Using Iso-Seq reads for annotation and expression, we identified 483 genes differentially expressed at a stringent FDR < 0.05. Using \textit{MasigPro} to differentiate genotype and age effects (illustrated in \cref{fig:dea_model}), we observed evidence for transcriptional consequences of the rTg4510 genotype (\cref{fig:dea_model_genexp}\textbf{A}), age (\cref{fig:dea_model_genexp}\textbf{B}), and interaction between genotype and age (\cref{fig:dea_model_genexp}\textbf{D,E,F,G}). Classifying differentially expressed genes by effects, we identified 18 differentially expressed genes that were associated by genotype effect and 356 genes (73.7\%) genes whose expression significantly changed with tau pathology progression in rTg4510 mice (\cref{fig:dea_model_num}). Among these, there was a significant (exact bionomial test, n = 356 genes, P = 1.91 x 10\textsuperscript{-44}) enrichment of upregulated genes (n = 304 genes (85.3\%) with upregulated expression in TG compared to WT; n = 52 (14.6\%) genes with downregulated expression in TG). Using \textit{EnrichR}, the differentially expressed genes were found to be highly enriched in the lysosome (KEGG 2021 Human: odds ratio = 7.01, adjusted P = 1.70 x 10\textsuperscript{-5}) and in particularly the TGF-\textbeta signalling pathway (WikiPathway 2021 Human: odds ratio = 17.16, adjusted P = 2.92 x 10\textsuperscript{-2} ). Further in line with previous findings, a third of genes identified with expression changes from Iso-Seq reads were enriched in pathways involved in immune system activation (n = 140 genes, 34.4\%, "turquoise" co-expression module\cite{Castanho2020}, \cref{fig:dea_model_num}\textbf{B}). 

While our previous RNA-Seq study\cite{Castanho2020} was more powered to identify gene expression changes (n = 1916 differentially expressed genes) with a bigger sample size (RNA-Seq: n = 29 rTg4510 TG, n = WT; Iso-Seq: n = 6 TG, n = 6 WT) and deeper sequencing coverage (RNA-Seq: mean number of reads = 18.8M; Iso-Seq: mean number of CCS reads = 5.7M reads), the top XX genes were also detected as differentially expressed using Iso-Seq reads. The top differentially expressed gene associated with progressive tau pathology rTg4510 mice was also \textit{Gfap} - which encodes for glial fibrillary acidic protein (GFAP\nomenclature{GFAP}{Glial Fibrillary Acidic Protein}), a cytoskeletal protein that acts as a marker for astrocyte activation and its upregulation in human AD post-mortem brain tissues and other AD mouse  models has been well reported\cite{Muramori1998,Ishiki2016, Chatterjee2021}. Other top-ranked tau-associated differentially-expressed genes (\cref{tab:dea_wholemouse}) have been previously reported to play a role in AD development and pathology, notably \textit{C4b}\cite{Zorzetto2016} - a member of the complement immune system (\cref{fig:whole_dea}\textbf{C,D}), \textit{Slc14a1}\cite{Castillo2017} encoding the urea transporter 1, \textit{Tgfbr1} encoding the TGF-\textbeta receptor protein (\cref{fig:whole_dea}\textbf{E,F}) and \textit{Unc93b1}\cite{Wirz2013}, a transmembrane protein required for toll (\cref{tab:dea_wholemouse}). 

Finally as a means of validation, we also performed differential expression analysis using expression determined from mapping RNA-Seq reads (n = 29 TG, n = 30 WT) to the improved Iso-Seq-derived transcriptome annotation. In summary, our results implicate the utility of long-reads for accurate quantification at a gene-level in identifying robust tau-associated gene expression differences in the entorhinal cortex of rTg4510 mice. 
%% What is the overlap? 
 
% downregulation of micrologia genes in rTg4510 ==> cross reference gene list: https://actaneurocomms.biomedcentral.com/articles/10.1186/s40478-020-01099-x
  
\begin{figure}[h]
	\centering
	\includegraphics[page=5,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Examples of gene expression differing across conditions]%
	{\textbf{Differential expressed genes exhibiting genotype, age and interaction effects} Shown are examples of differentially expressed genes classified under the different models, using the whole transcriptome dataset (WT = 6, TG = 6, across age 2 and 8 months) using Iso-Seq read counts as abundance: \textbf{(A)} \textit{Tigd2} with a genotype effect, \textbf{(B)} \textit{Mobp} with a genotype and age effect, \textbf{(C)} \textit{Cik1} with an age effect, and \textbf{(D)} \textit{Cd34}, \textbf{(E)} \textit{Unc93b1}, \textbf{(F)} \textit{Csf1r} and \textbf{(G)} \textit{Tgfbr2} with an interaction effect. Dashed lines represent mean paths across age groups. rTg4510 wild-type and transgenic mice are denoted by red and grey, respectively. }   
	\label{fig:dea_model_genexp}
\end{figure}
 
\begin{figure}[h]
	\centering
	\includegraphics[page=4,trim={0 19cm 0 0},clip,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differentially expressed genes classified by conditions]%
	{\textbf{Differentially expressed genes were identified across all the different conditions with a number of differentially expressed genes exhibiting an interaction effect of rTg4510 genotype and age.} \textbf{(A)} A bar plot of the number of differentially expressed genes (n = 483, determined from Iso-Seq FL read count as proxy of expression) classified by rTg4510 genotype, age, and interaction effect (n = 6 WT, n = 6 TH, across 2 and 8 months). \textbf{(B)} A pie chart of the number and proportion of differentially expressed genes with genotype and interaction effect (n = 407 genes) identified in discrete co-expression network modules from a previous RNA-Seq study\cite{Castanho2020}; all three modules are significantly associated with progressive tau pathology: "Red" module is downregulated in TG and enriched in synaptic transmission, "Turquoise" module is upregulated in TG and enriched for immune system activation, and "Yellow" module is downregulated in TG and enriched in mitochondria and synpatic processes.}    
	\label{fig:dea_model_num}
\end{figure}


\begin{figure}[h]
	\begin{center}
		\includegraphics[page=1,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Top ranked differentially expressed genes associated with progressive tau pathology in rTg4510]%
	{\textbf{Top ranked differentially expressed genes associated with progressive tau pathology in rTg4510}: Shown are the the expression plots for \textbf{(A, B)} \textit{Gfap}, \textbf{(C, D)} \textit{C4b} and \textbf{(E, F)} \textit{Tgfbr1} using either Iso-Seq FL read count or RNA-Seq reads as expression. Dashed lines represent mean paths across age groups. rTg4510 wild-type and transgenic mice are denoted by red and grey, respectively.}   
	\label{fig:whole_dea}
\end{figure}

\vspace{2cm}
%log2fc calculated by mean expression at 8 case/mean expression at 8 control
\begin{table}[!htp]
	\centering
	\caption[Top-ranked differentially expressed genes associated with rTg4510]%
	{Tabulated are the top-ranked genes identified as differentially expressed in rTg4510 using \textit{maSigPro} with Iso-Seq defined transcriptome for annotation and Iso-Seq FL read count as expression. Gene expression is determined from the sum of normalised expression of associated transcripts. FDR - False Discovery Rate. }
	\begin{tabularx}{0.85\textwidth}{cccccccc}
	\toprule
	\multirow{3}{*}{Gene} &
	\multirow{3}{*}{FDR} &
	\multirow{3}{*}{R\textsuperscript{2}} &
	\multirow{3}{*}{\begin{tabular}[c]{@{}c@{}}log2-fold\\  change \\ (8 months)\end{tabular}} &
	\multicolumn{4}{c}{Mean Gene Expression} \\ \cmidrule(l){5-8} 
	&          &       &      & \multicolumn{2}{c}{Wild-type} & \multicolumn{2}{c}{Transgenic} \\ \cmidrule(l){5-8} 
	&          &       &      & 2 moss      & 8 mos      & 2 mos       & 8 mos      \\ \midrule
	\textit{C4b}    & 1.6 x 10\textsuperscript{-41}  & 0.945 & 4.38 & 4.94          & 2.73          & 4.97           & 103           \\
	\textit{Gfap}     & 6.04 x 10\textsuperscript{-36} & 0.933 & 3.12  & 82.8          & 70.5          & 118            & 1030           \\
	\textit{Tgfbr1}  & 7.9 x 10\textsuperscript{-24}  & 0.892 & 2.95 & 0.663         & 3.38          & 2.03           & 15.7          \\
	\textit{Slc14a1} & 4.31 x 10\textsuperscript{-22} & 0.899 & 2.95 & 9.55          & 14.7          & 6.16           & 47.7            \\
	\textit{Pros1} & 1.05 x 10\textsuperscript{-17} & 0.894 & 2.08 & 8.17          & 9.32         & 6.26           & 26.4          \\
	\textit{Unc93b1} & 1.46 x 10\textsuperscript{-16} & 0.863 & 1.61  & 3.59          & 5.04          & 6.47          & 19.8          \\ \bottomrule
	\end{tabularx}
	\label{tab:dea_wholemouse}
\end{table}

\clearpage
\subsection{rTg4510 mice characterised by expression differences in novel, antisense genes}
Highlighting the power of long-reads to comprehensively annotate the transcriptome, we previously detected novel genes in our Iso-Seq dataset that were not present in existing genome annotations(\cref{sec:whole_novelgenes}). These genes were lowly-expressed and typically antisense to known genes with overlap at the UTR or gene body. Given the improved annotation, we next sought to explore expression changes in novel genes associated with rTg4510 genotype. 

While we were unable to detect significant differentially-expressed novel genes using Iso-Seq FL reads as expression due to low sequencing coverage, we identified 3 novel genes with significant expression change from mapping RNA-Seq reads to Iso-Seq-derived transcriptome. The most significant differentially-expressed novel gene was detected in chromosome 10 (\cref{fig:whole_novelgene_difftracks}\textbf{A}) with progressive down-regulation in TG (\cref{fig:whole_novelgene_diffexp}\textbf{A}). The other two differentially-expressed novel genes were found antisense to known genes: \textit{Htra1} at the 5'UTR (\cref{fig:whole_novelgene_difftracks}\textbf{B}) and \textit{Fgfr1op} (\cref{fig:whole_novelgene_difftracks}\textbf{C}) within the gene-body. Both genes were upregulated with progressive tau pathology in TG mice (\cref{fig:whole_novelgene_diffexp}\textbf{B,D}). Notably, while \textit{Fgfr1op} was not identified as differentially-expressed (\cref{fig:whole_novelgene_diffexp}\textbf{C}), \textit{Htra1} was also found to have higher expression in rTg4510 TG compared to WT (\cref{fig:whole_novelgene_diffexp}\textbf{E}).     
%Htra1-AS shared exonic regions with Htra1 so misalignment of RNA-Seq reads

\begin{landscape}
	\begin{figure}[!htp]
		\centering
		\includegraphics[page=1,trim={0 3cm 0 0}, scale = 0.45]{Figures/TracksFigures_Diff.pdf}
		\captionsetup{width=1.4\textwidth}
		\caption[Visualisation of differentially expressed novel genes in rTg4510 TG mice]%
		{\textbf{Visualisation of novel genes that were differentially expressed in rTg4510 mice.} Shown are UCSC genome browser tracks of three novel genes whose expression changed with rTg4510 genotype: \textbf{(A)} novel gene in chromosome 10, \textbf{(B)} novel gene antisense to \textit{Fgfr1op}, and \textbf{(C)} novel gene antisense to \textit{Htra1}. Isoforms are coloured based on \textit{SQANTI} classification, with the novel gene coloured grey for genic/intergenic (Figure A) and pink for antisense (Figures B and C). Shown are also reference genome annotations (mm10) and RNA-Seq data from matched samples.}   
		\label{fig:whole_novelgene_difftracks}
	\end{figure}
\end{landscape}

\begin{figure}[!htp]
	\begin{center}
		\includegraphics[page=8,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Three novel genes were found differentially expressed in rTg4510 TG mice]%
	{\textbf{Three novel genes were found differentially expressed in rTg4510 TG mice}: Shown are gene expression of three novel genes: \textbf{(A)} intergenic in chromosome 10 (\cref{fig:whole_novelgene_difftracks}\textbf{A}), \textbf{(B)} antisense to \textit{Fgfr1op} and \textbf{(D)} to \textit{Htra1}. Gene expression for the two known genes, \textbf{(C)} \textit{Fgfr1op} and \textbf{(E)} \textit{Htra1} are also shown. Gene expression was determined from mapping RNA-Seq reads to Iso-Seq-derived annotations.}   
	\label{fig:whole_novelgene_diffexp}
\end{figure}



\clearpage
\subsection{Gene expression differences in rTg4510 are primarily driven by differential expression of dominant isoform}
One of the added advantages of long-reads from accurate isoform annotation is the improved confidence to reliably detect isoforms whose expression significantly changed across experimental conditions and over time. Given that we were able to reliably detect tau-associated differentially-expressed genes in rTg4510 TG mice using normalised FL long-read read counts, we subsequently sought to extend this in detecting differentially expressed transcripts using the same approach. 

Using Iso-Seq reads for annotation and expression, we identified 582 differentially expressed transcripts, 378 (64.9\%) of which were associated with progressive tau pathology. Strikingly, the top significant progressive-tau-associated differentially expressed transcript in rTg4510 TG mice was the known canonical transcript (\textit{Gfap}-$\alpha$, ENSMUST00000077902.5) annotated to \textit{Gfap} (\cref{fig:DEI_gfap}), the top tau-associated differentially-expressed genes previously identified (\cref{tab:dea_wholemouse}). \textit{Gfap}-$\alpha$ was dramatically upregulated with progressive tau pathology, suggesting that increased \textit{Gfap} gene expression in aged rTg4510 mice is primarily driven by one isoform. This is in line with previous findings that reported a differential increase of \textit{Gfap}-$\alpha$ in AD temporal cortex. Conversely, a study using \textit{Gfap}-isoform specific qPCR assays in 3xTgAD and APPswePS1dE9 mouse model reported significant increase of transcript levels of all \textit{Gfap} isoforms with no differential change in the expression of the various isoforms. Interestingly, the transcriptomic profile was more in line with this finding when we mapped RNA-Seq to Iso-Seq-derived annotation; this included upregulation of \textit{Gfap}-$\beta$ (PB.2972.6), \textit{Gfap}-$\delta$ (PB.2972.12). 
%hough tau-pathology associated expression changes in minor isoforms were significantly more pronounced - a reflection of the comparably higher sequencing coverage of RNA-Seq reads to Iso-Seq reads.

The second top significant progressive-tau-associated differentially expressed transcript was the only known transcript (ENSMUST00000069507.8) annotated to \textit{C4b}, the second top tau- associated differentially expressed gene(\cref{tab:dea_wholemouse}). Similar to the \textit{Gfap}-$\alpha$, the canonical \textit{C4b}-isoform was dramatically upregulated with progressive tau pathology in rTg4510 using expression whereas expression of the novel isoforms were all lowly expressed.      

\begin{figure}[!htp]
	\centering
	\includegraphics[page=1,trim={0.5cm 4.8cm 2cm 1cm}, scale = 0.85]{Figures/Ch5_DiffPlots.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential Isoform Expression: Changes in transcript expression of isoforms associated with \textit{Gfap}]%
	{\textbf{Significant upregulation of known isoform of \textit{Gfap} with progressive tau pathology}. Shown are \textbf{(A)} UCSC genome browser tracks of isoforms associated with \textit{Gfap}, \textbf{(B)} hierarchal clustering of each \textit{Gfap}-associated isoform based on abundance (Iso-Seq FL read count, log2), \textbf{(C)} Normalised Iso-Seq FL read count of the top 15 most abundant isoforms, and \textbf{(D)} differentially expressed transcripts identified using normalised Iso-Seq read and \textbf{(E)} RNA-Seq read counts. Grey dots denote to differentially expressed transcripts identified from RNA-Seq but not Iso-Seq reads. FSM - Full Splice Match, ISM - Incomplete Splice Match, NIC - Novel In Catalogue, NNC - Novel Not in Catalogue. WT - Wild-type, TG - Transgenic. Dotted lines represent the mean paths across ages (months).} 
	\label{fig:DEI_gfap}
\end{figure}

\begin{figure}[!htp]
	\centering
	\includegraphics[page=2,trim={1.5cm 4.8cm 2cm 1cm}, scale = 0.85]{Figures/Ch5_DiffPlots.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential Isoform Expression: Changes in transcript expression of isoforms associated with \textit{C4b}]%
	{\textbf{Significant upregulation of known isoform of \textit{C4b} with progressive tau pathology}. Shown are \textbf{(A)} UCSC genome browser tracks of isoforms associated with \textit{C4bp}, \textbf{(B)} hierarchal clustering of each \textit{C4b}-associated isoform based on abundance (Iso-Seq FL read count, log2), \textbf{(C)} Normalised Iso-Seq FL read count of the top 15 most abundant isoforms, and \textbf{(D)} differentially expressed transcripts identified using normalised Iso-Seq read and \textbf{(E)} RNA-Seq read counts. Grey dots denote to differentially expressed transcripts identified from RNA-Seq but not Iso-Seq reads. FSM - Full Splice Match, ISM - Incomplete Splice Match, NIC - Novel In Catalogue, NNC - Novel Not in Catalogue. WT - Wild-type, TG - Transgenic. Dotted lines represent the mean paths across ages (months).}   
	\label{fig:DEI_c4b}
\end{figure}


\subsection{rTg4510 characterised by widespread expression changes in transcripts associated to genes implicated in AD}

Other isoforms that were significantly unregulated with progressive tau pathology were annotated to genes that have been previously strongly implicated in AD pathology and development (\cref{fig:DEI_ADgenes_isoseq}). This included ENSMUST00000151120.8 associated with \textit{Ctsd}, which encodes for Cathepsin D, a lysosomal protease that is involved in degradation of A$\beta$ \cite{JR1996}, tau \cite{A1997}, and has recently been identified as a key regulator of A$\beta$42/40 ratio \cite{Suire2020}. Other isoforms include ENSMUST00000172785.7 associated with \textit{H2-D1} - that encodes for major histocompatibility complex (MHC) class 1, an immune-related gene that has been found to be upregulated in microglial cells of a different mouse model of neurodegeneration with AD-like phenotypes \cite{Mathys2017} - ENSMUST00000028624.8 associated with \textit{Gatm}, a mitochondrial protein that has been recently revealed as a key protein signature of AD from a large proteomic analysis of human cortex and CSF \cite{Wang2020}, - and ENSMUST00000030765.6 associated with \textit{Padi2}/\textit{Pad2}, an enzyme that has been found abnormally activated in astrocytes of patients with AD \cite{A2005}.


Observations of differential isoform expression using Iso-Seq reads as expression were recapitulated with RNA-Seq reads, highlighting the power to accurately quantify isoforms at high expression (\cref{fig:DEI_ADgenes_rnaseq}).
However, on closer examination, the majority of isoforms identified as differentially expressed with progressive tau pathology (n = 321, 84.9\%) were not similarly identified as differentially expressed when RNA-Seq reads were used as expression. Given that the wide majority of Iso-Seq-identified-differentially-expressed isoforms were very-lowly expressed (295 isoforms,91.9\%, with mean full-length counts < 24 FL, n = 12 samples) with low read count, a seemingly significant expression change may result in calling the isoform differentially expressed (\cref{fig:dei_lowisoexp}). However, even for more highly-expressed isoforms, while a change in mean expression was observed, the variance was substantial due to a small sample size (n = 3 replicates). Conversely, with a greater sample size (n = 6 replicates) and higher sequencing coverage of RNA-Seq reads, the usage of RNA-Seq reads as expression reduced the probability of calling an isoform as differentially expressed due to chance (\cref{fig:dei_highisoexp}).      


\begin{figure}[!htp]
	\centering
	\includegraphics[page=15,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Robust changes in transcript expression of isoforms annotated to genes that are strongly implicated in AD]%
	{\textbf{Robust changes in transcript expression of isoforms annotated to genes that are strongly implicated in AD}. Using Iso-Seq reads (n = 6 WT, n = 6 TG, across 2 and 8 months) as annotation and expression, differential isoform expression was observed for \textbf{(A)} ENSMUST00000151120.8 (PB.15108.6) annotated to \textit{Ctsd}, \textbf{(B)} ENSMUST00000172785.7 (PB.7039.1) annotated to \textit{H2-D1}, \textbf{(C)} ENSMUST00000028624.8 (PB.9298.1) annotated to \textit{Gatm}, and \textbf{(D)} ENSMUST00000030765.6 (PB.11607.2) associated with \textit{Padi2}/\textit{Pad2}.  WT - wild-type, TG - Transgenic. Dotted lines represent the mean paths across ages.}    
	\label{fig:DEI_ADgenes_isoseq}
\end{figure}

\begin{figure}[!htp]
	\centering
	\includegraphics[page=16,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Changes in transcript expression of genes strongly implicated in AD were similarly detected using RNA-Seq reads]%
	{\textbf{Changes in transcript expression of genes strongly implicated in AD were similarly detected using RNA-Seq reads}. Usage of RNA-Seq reads (n = 30 WT, n = 29 TG, across 2, 4, 6 and 8 months) as expression similarly identified isoforms as differentially expressed (\cref{fig:DEI_ADgenes_isoseq}). \textbf{(A)} ENSMUST00000151120.8 (PB.15108.6) annotated to \textit{Ctsd}, \textbf{(B)} ENSMUST00000172785.7 (PB.7039.1) annotated to \textit{H2-D1}, \textbf{(C)} ENSMUST00000028624.8 (PB.9298.1) annotated to \textit{Gatm}, and \textbf{(D)} ENSMUST00000030765.6 (PB.11607.2) associated with \textit{Padi2}/\textit{Pad2}.  WT - wild-type, TG - Transgenic. Dotted lines represent the mean paths across ages.}   
	\label{fig:DEI_ADgenes_rnaseq}
\end{figure}

\begin{figure}[!htp]
	\centering
	\includegraphics[page=15,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential isoform expressed observed with Iso-Seq reads as expression were not recapitulated using RNA-Seq reads]%
	{\textbf{Differential isoform expressed observed with Iso-Seq reads as expression were not recapitulated using RNA-Seq reads}. Shown are normalised counts \textbf{(A)} \textit{Cd34} and \textbf{(C)} \textit{Ubqln1} with associated isoforms identified from using Iso-Seq reads as expression, and of the same two genes \textbf{(B)} \textit{Cd34} and \textbf{(D)} \textit{Ubqln1} identified from using RNA-Seq reads as expression. Significant changes in isoform expression identified using Iso-Seq reads as expression - PB.1063.2 associated with \textit{Cd34} and PB.4255.13 and PB.4255.4 associated with \textit{Ubqln1} - was not recapitulated when using RNA-Seq reads as expression, due to lower sequencing coverage. Notably, minor isoforms had a higher expression with RNA-Seq alignment than direct detection with Iso-Seq reads.
	FSM - Full Splice Match, ISM - Incomplete Splice Match, NIC - Novel In Catalogue, NNC - Novel Not in Catalogue. Dotted lines represent the mean paths across ages.
	}   
	\label{fig:dei_lowisoexp}
\end{figure}

\begin{figure}[!htp]
	\centering
	\includegraphics[page=14,trim={0cm 20cm 0cm 0cm},clip,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential Isoform Expression observed in isoforms with high expression but large variance]%
	{\textbf{Usage of RNA-Seq reads as expression, with larger sample size, reduce probability of calling isoforms differentially expressed due to chance} Shown are \textbf{(A)} \textit{Slc1a3} with associated isoforms identified as differentially expressed when Iso-Seq reads were used as expression (n = 6 WT, n = 6 TG, across 2 time points), and \textbf{(B)} \textit{Slc1a3} when RNA-Seq reads were used as expression (n = 30 WT, n = 29 TG, across 4 time points). 
	\\
	\\	
	Using Iso-Seq reads as expression, the more-highly expressed isoforms PB.527.1 (red) in Figure a) was identified as differentially expressed. The expression change, however, of the same isoform was less prominent when RNA-Seq reads were used due to greater sample size and higher sequencing coverage. Notably, all the minor isoforms had a higher expression when RNA-Seq reads were used, resulting in some novel isoforms not being pre-filtered (as in the case with Iso-Seq reads as expression due to low full-length read count)
	\\
	\\
	FSM - Full Splice Match, ISM - Incomplete Splice Match, NIC - Novel In Catalogue, NNC - Novel Not in Catalogue. Dotted lines represent the mean paths across ages.
}   
	\label{fig:dei_highisoexp}
\end{figure}


\clearpage
\subsection{Iso-Seq read depth from global transcriptome profiling insufficient for differential transcript usage analysis}
Contributing to the complexity of transcript regulation through alternative splicing, while the expression of a gene may be constant between conditions, the relative expression of the isoforms (and thus isoform proportion) can change. This phenomenon is known as differential transcript/isoform usage (DIU), and was also assessed using \textit{tappAS} for genes with more than one isoform. Of note, a major isoform switching event occurs when the dominant (major) isoform in one condition becomes a minor isoform in another.    

Using Iso-Seq reads for annotation and expression and after filtering lowly-expressed isoforms, we identified 400 genes that were characterised with differential isoform usage. However upon further examination, the majority of these genes were lowly expressed and the associated isoforms that were observed to undergo differential usage had only 1-2 full-length long-read counts. We were therefore not confident in any of observed DIU changes detected with Iso-Seq abundance from the whole transcriptome sequencing due to low sequencing depth. Indeed, none of the DTU genes were significant after applying a gene expression threshold (described in \cref{ch:diu_method}).


\iffalse
\begin{figure}[htp]
	\begin{center}
		\includegraphics[page=10,trim={0cm 18cm 0cm 0cm},clip,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Genes characterised with differential isoform usage had low Iso-Seq full-length read counts]%
	{\textbf{Genes characterised with differential isoform usage had low Iso-Seq full-length read counts}: Shown are density plots of \textbf{(A)} median full-length vs normalised counts and \textbf{(B)} mean full-length vs normalised counts of DIU genes that were identified using Iso-Seq reads as expression. }   
	\label{fig:DIU_lowdepth}
\end{figure}
\fi

% DIU genes with major switching	
In further support of this conclusion, there was a low overlap of DIU genes that were identified when we used RNA-Seq reads as expression. This was likely to be a reflection of lower sequencing depth of Iso-Seq reads, resulting in i) a different pool of isoforms after filtering lowly-expressed isoforms - a minor isoform was more likely to be filtered when using RNA-Seq reads as abundance rather than Iso-Seq reads, particularly if the overall gene expression was low, due to relatively greater expression difference, and ii) smaller differences in isoform expression using Iso-Seq reads are translated to misleading significant changes in isoform proportions, resulting in false detection of genes as DIU. These implications can be seen in the gene \textit{Esyt2}, whereby the low Iso-Seq isoform counts resulted in misleading representation of isoform fraction, which was not recapitulated when RNA-Seq reads were used as expression (\cref{fig:DIU_esyt2}). 

\begin{figure}[htp]
	\begin{center}
		\includegraphics[page=9,scale = 0.55]{Figures/WholeDifferentialAnalysis.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[\textit{Esyt2} was misidentified with differential isoform usage due to low Iso-Seq read counts]%
	{\textbf{\textit{Esyt2} was misidentified with differential isoform usage due to low Iso-Seq read counts}: \textbf{(A)} Isoform expression (normalised counts) and \textbf{(C)} subsequent deduction of isoform fraction of \textit{Esyt2} using Iso-Seq reads as expression, and the equivalent \textbf{(B)} isoform expression and \textbf{(D)} isoform fraction of the same gene, \textit{Esyt2} with RNA-Seq reads as expression. Iso-Seq reads were used as annotation in both analyses. 
		\\
		As can be observed, the Iso-Seq normalised counts are significantly lower than RNA-Seq normalised counts for the associated isoforms, resulting in a misleading representation of isoform fraction with major isoform switching events (Figure c, PB.3907.1 becomes the dominant isoform in transgenic mice), which is not recapitulated with RNA-Seq reads as expression (Figure d)}   
	\label{fig:DIU_esyt2}
\end{figure}

\subsection{Hybrid approach identifies tau-pathology associated differential isoform usage with major isoform switching events}
Differential isoform usage was performed with Iso-Seq reads as annotation and RNA-Seq reads as expression, given higher sequencing RNA-Seq read depth and greater sample size. 570 genes were identified with differential isoform usage (\cref{tab:DIU_DEA_nums}). 

\vspace{1cm}
\begin{table}[!htp]
	\centering
	\begin{tabularx}{0.85\textwidth}{cccc}
		\toprule
		\multicolumn{3}{c}{Conditions}                                                                                                                                                                                       & \multirow{2}{*}{Number of Genes} \\ \cmidrule(r){1-3}
		\begin{tabular}[c]{@{}c@{}}Differential Gene\\  Expression\end{tabular} & \begin{tabular}[c]{@{}c@{}}Differential Isoform \\ Usage\end{tabular} & \begin{tabular}[c]{@{}c@{}}Isoform Major\\  Switching\end{tabular} &                                  \\ \midrule
		\checkmark                                                                      & \checkmark                                                                    & \checkmark                                                                 & 13                               \\
		\checkmark                                                                      & \checkmark                                                                    & x                                                                  & 59                               \\
		x                                                                       & \checkmark                                                                    & \checkmark                                                                 & 39                               \\
		x                                                                       & \checkmark                                                                    & x                                                                  & 459                              \\ \midrule
		\multicolumn{3}{c}{Total Number of Genes}                                                                                                                                                                            & 570                              \\ \bottomrule
	\end{tabularx}
	\caption[Number of Genes identified with differential gene expression and isoform usage]%
	{\textbf{Number of Genes identified with differential gene expression and isoform usage}. Tabulated are the number of genes identified from differential gene expression and isoform usage analysis from \textit{tappAS} with Iso-Seq reads as annotation and RNA-Seq reads as expression. The models for each condition are depicted in \cref{fig:DIU_DEA_model}. Isoform major switching refers to the event of an isoform being predominantly expressed in one condition (major) but lowly-expressed in another condition (minor)}
	\label{tab:DIU_DEA_nums}
\end{table}

The majority of these genes (n = 498, 87.3\%), while observed with DIU, were not differentially expressed between wild-type and transgenic mice - a scenario whereby the differential upregulation of one isoform is compensated by the differential downregulation of another isoform, resulting in no net gene expression change. This indicates that a significant degree of the post-transcriptional regulation was independent of gene expression regulation. An example of this was \textit{Ptprz1}, a receptor protein tyrosine phosphatase, which is involved in oligodendrocyte development and function. While there was no change in overall gene expression, closer examination revealed progressive upregulation of two isoforms (PB.13000,10, PB.13000.11) which was offset by progressive downregulation of two other isoforms (PB.13000.14, PB.13000.7) in transgenic mice. In transgenic mice, the known isoform PB.13000.10 (ENSMUST00000090568.6) continued to dominate in both wild-type and transgenic mice across all ages with a slight increase. This was complemented by a slight decrease in the another known shorter isoform PB.13000.14 (ENSMUST00000202579.3). Interestingly, the two other differentially expressed isoforms were novel, with skipping of exon 16. 

The most significant gene characterised with differential isoform usage with a dominant isoform switch but no change in gene expression was \textit{Cisd3}, a mitochondrial iron-sulphur domain-containing protein involved in regulating electron transport and iron homeostasis essential for normal mitochondrial function. While there was little change in overall gene expression, there is a major isoform shift between wild-type and transgenic mice that was generally consistent across all ages. The two isoforms only differ at the 5'end with the first exon of the upregulated isoform, ENSMUST00000107583.2 (PB.2833.2), spanning across exons 1 and 2 of the downregulated isoform, ENSMUST00000107584.7 (PB.2833.1). 
% Predicted protein structure

%GFAP not significant?   
%Interestingly, the was a lower consensus when comparing the number of DIU genes identified when using RNA-Seq reads as abundance, suggesting that the choice of filtering strategy becomes redundant when dealing with low sequencing depth.  


\begin{figure}[htp]
	\begin{center}
		\includegraphics[page=1,scale = 0.55]{Figures/DIU_notDEG_nomajor.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential isoform expression and usage of \textit{Ptprz1}]%
	{\textbf{Differential isoform expression and usage of \textit{Ptprz1}}: \textbf{(A)} \textit{Cisd3}'s RNA-Seq gene expression in wild-type (grey) and transgenic (red) mice, with no significant overall change. \textbf{(B)} RNA-Seq expression of the two known and two novel isoforms in wild-type and mice across age. \textbf{(C)} Overall isoform fraction of two isoforms across wild-type and transgenic, \textbf{(D)} and by age.}    
	\label{fig:DIU_ptprz1}
\end{figure}


\begin{figure}[htp]
	\begin{center}
		\includegraphics[page=1,scale = 0.55]{Figures/DIU_notDEG_major.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential isoform expression and usage of \textit{Cisd3}]%
	{\textbf{Differential isoform expression and usage of \textit{Cisd3}}: \textbf{(A)} \textit{Cisd3}'s RNA-Seq gene expression in wild-type (grey) and transgenic (red) mice, with no significant overall change. \textbf{(B)} RNA-Seq expression of the two known isoforms in wild-type and mice across age. \textbf{(C)} Overall isoform fraction of two isoforms across wild-type and transgenic, \textbf{(D)} and by age.}    
	\label{fig:DIU_Cisd3}
\end{figure}


\clearpage
\subsection{Integrative analysis reveals co-localisation of genotype-associated differentially expressed transcripts and differentially-methylated positions}
Epigenetic modifications such as DNA methylation have been widely acknowledged to heavily influence gene regulation and more specifically, alternative splicing. Using genome-wide DNA methylation data on the same mouse samples, we wanted to use an integrative approach to identify genes with both splicing and methylation changes between wild-type and transgenic mice. Genes defined to undergo differential splicing were classified by either genes with a change in transcript expression (differential isoform expression - DIE), a change in usage of isoform proportions (differential isoform usage - DIU) or a change in feature element (differential feature inclusion - DFI). Genes with changes in methylation were classified as either with a change in a specific position (differential methylated position - DMP) associated with tau pathology (genotype effect) and progressively over time (interaction effect), or associated with a cluster of differentially methylated cytosines (differentially methylated region - DMR). 

Using a stringent approach to identify biologically meaningful splicing changes, we identified nine genes with an altered splicing and methylation profile associated with progressive tau pathology. The most significant co-differentially-methylated position and differentially-spliced gene was \textit{Spata13} - a gene involved in cell migration\cite{Bourbia2019} and has been reported to be upregulated in entorhinal cortex of AD patients\cite{Yan2019} - which was characterised with a change in transcript expression, usage and feature inclusion. There was a significant progressive increase in expression of the shorter known isoform (PB.4966.2, ENSMUST00000162945.1) in the transgenic mice, whereas there was no change in expression of the longer, novel NIC isoform (PB.4966.1) which harboured a 5' miRNA-binding site (miR-446h-5p) and an alternative transcription start site with an upstream open reading frame (\cref{fig:IntMeth_Spata13}\textbf{a, d}). A DMP was identified upstream of \textit{Spata13} (8.9Kb and 107Kb from known and novel isoform respectively, \cref{fig:IntMeth_Spata13}\textbf{b, d}) and was hypermethylated in transgenic mice compared to wild-type ($\Delta$$\beta$ = 0.777, FDR\textsubscript{interaction} = 0.012), \cref{fig:IntMeth_Spata13}\textbf{C}), suggesting a distal suppressed expression of both isoforms in the wild-type. Conversely, the decreased methylation in the transgenic is only associated with suppressed expression of the proximal, novel isoform. 

\begin{figure}[ht]
	\includegraphics[page=2,scale = 0.4]{Figures/WholeDifferentialAnalysis_DMPDMR.pdf}
	\\
	\hspace*{0.2cm}\vspace{0.5cm}\large d
	\\
	\includegraphics[page=1,trim={1.5cm 0 0 0},scale = 0.9]{Figures/SPATA13_DMP.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential splicing and methylation of \textit{Spata13}]%
	{\textbf{Differential splicing and methylation of \textit{Spata13}}: \textbf{A}) Upregulation of known isoform (PB.4966.2, ENSMUST00000162945.1) associated with \textit{Spata13} in TG (P = 9.42 x 10 \textsuperscript{-33}, R\textsuperscript{2} = 0.73) \textbf{(B)} Hypermethylated DMP identified upstream of promoter ($\Delta$$\beta$ = 0.777, FDR\textsubscript{interaction} = 0.012). \textbf{(C)} Correlation between DNA methylation at this site and normalised isoform expression of the \textit{Spata13} gene. \textbf{(D)} UCSC browser track showing relative position of DMP and two isoforms. }    
	\label{fig:IntMeth_Spata13}
\end{figure}

%Another gene with co-differentially-methylated position and differentially-spliced gene was \textit{Tmem237}, characterised by an isoform switch of a known (PB.261.4) and novel NNC (PB.261.10) isoform, which had an alternative first exon and a shorter 3'UTR resulting in absence of miRNA-binding sites. 
% check methylation levels across first exon.

\newpage
Other genes were characterised with a change in transcript expression correlated with a change in methylation. This included \textit{Ncf2} (\cref{fig:IntMeth_Ncf2}), \textit{Osmr}, and \textit{Cebpa} where there was a progressive increase in expression of the respective known isoform and an associated hypomethylated DMP (located in the promoter) in transgenic; whereas in \textit{Rnf165} and \textit{Susd5} (\cref{fig:IntMeth_Susd5}), there was a progressive downregulation of known isoform and an associated hypermethylated DMP in transgenic. Interestingly, we also identified altered splicing and methylation changes in \textit{Irf8} (\cref{fig:IntMeth_Irf8}) - a transcription factor of the IRF family that is involved in microglial activation in AD\cite{Zeng2017}, where a hypomethylated DMP (located 130kb downstream) in TG was associated with progressive increase in transcript expression. The associations described here between DNA methylation and transcript expression support well-established theories of the relationship between DNA methylation and gene expression, whereby increased methylation at the promoter is associated with decreased gene expression. Of note, only one isoform was detected for each of these genes.  
% Note other example where this was not the case 

\begin{figure}[h]
	\includegraphics[page=3,scale = 0.4]{Figures/WholeDifferentialAnalysis_DMPDMR.pdf}
	\\
	\hspace*{0.2cm}\vspace{0.5cm}\large d
	\\
	\includegraphics[page=1,trim={1.5cm 0 0 0},scale = 0.9]{Figures/NCF2_DMP.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential splicing and methylation of \textit{Ncf2}]%
	{\textbf{Differential splicing and methylation of \textit{Ncf2}}. \textbf{A}) Upregulation of known isoform associated with \textit{Ncf2} (PB.700.1,ENSMUST00000027754.6) in TG coincided with two \textbf{(B)} hypomethylated DMP located upstream of promoter. Only the most significant DMP is shown ($\Delta$$\beta$ = -0.293, FDR\textsubscript{interaction} = 0.039). \textbf{(C)} Correlation between DNA methylation at this site and normalised isoform expression of the \textit{Ncf2} gene. \textbf{(D)} UCSC browser track showing relative position of DMP and isoform.}    
	\label{fig:IntMeth_Ncf2}
\end{figure}

\begin{figure}[]
	\includegraphics[page=4,scale = 0.4]{Figures/WholeDifferentialAnalysis_DMPDMR.pdf}
	\\
	\hspace*{0.2cm}\vspace{0.5cm}\large d
	\\
	\includegraphics[page=1,trim={1.5cm 0 0 0},scale = 0.9]{Figures/SUSD5_DMP.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential splicing and methylation of \textit{Susd5}]%
	{\textbf{Differential splicing and methylation of \textit{Susd5}}: \textbf{A}) Downregulation of known isoform associated with \textit{Susd5} (PB.16983.1,ENSMUST00000135338.2) in TG (P = 5.93 x 10\textsuperscript{-52}, R\textsuperscript{2} = 0.84) \textbf{(B)} Five hypermethylated DMPs were identified in the intronic region after the first exon. Only the most significant DMP is shown ($\Delta$$\beta$ = 0.344, P\textsubscript{Genotype} = 1.63). \textbf{(C)} Correlation between DNA methylation at this site and normalised isoform expression of the \textit{Susd5} gene. \textbf{(D)} UCSC browser track showing relative position of DMPs and isoform}    
	\label{fig:IntMeth_Susd5}
\end{figure}

\begin{figure}[]
	\includegraphics[page=5,scale = 0.4]{Figures/WholeDifferentialAnalysis_DMPDMR.pdf}
	\\
	\hspace*{0.2cm}\vspace{0.5cm}\large d
	\\
	\includegraphics[page=1,trim={1.5cm 0 0 0},scale = 0.9]{Figures/IRF8_DMP.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential splicing and methylation of \textit{Irf8}]%
	{\textbf{Differential splicing and methylation of \textit{Irf8}}: \textbf{A}) Upregulation of known isoform associated with \textit{Irf8} (PB.15969.1, ENSMUST00000047737.9) in TG (P = 8.17 x 10\textsuperscript{-78}, R\textsuperscript{2} = 0.87)\textbf{B)} Two hypermethylated DMPs were identified downstream in the distal intergenic region. Only the most significant DMP is shown ($\Delta$$\beta$ = -0.464, FDR\textsubscript{Interaction} = 0.0368) \textbf{(C)} Correlation between DNA methylation at this site and normalised isoform expression of the \textit{Irf8} gene. \textbf{(D)} UCSC browser track showing relative position of DMPs and isoform}    
	\label{fig:IntMeth_Irf8}
\end{figure}

Focusing on genotype-associated differentially methylated regions, we identified splicing changes annotated to \textit{As3mt}  (\cref{fig:IntMeth_As3mt}) and \textit{Prnp}. Three isoforms were detected for \textit{As3mt}, a schizophrenia-associated GWAS gene, whereby there was a progressive upregulation in the known isoform (PB.8363.2, ENSMUST00000003655.8) in TG (P = 4.98 x 10\textsuperscript{-18}, R\textsuperscript{2} = 0.56) while there was no detected change in expression of the two other novel NNC isoforms, both of which were characterised by a novel splice junction resulting in a novel exon that was absent in the known isoform. A 485bp DMR consisting of nine differentially methylated cytosines was identified \textasciitilde{}8kb upstream of this novel exon within the intron, and was hypermethylated in transgenic mice compared to wild-type ($\Delta$ = 0.19, P = 1.22 x 10\textsuperscript{-8}). We hypothesise that an increase in methylation in this region would hinder recruitment and assembly of the spliceosome, resulting in skipping of the novel exon and subsequent increased expression of the known isoform. 

There was an increase in expression of a novel isoform of \textit{Prnp} in transgenic mice. However, given that \textit{Mapt} transgene includes exons 2 and 3 of the \textit{Prnp}, we were uncertain whether the transcript expression changes attributed to \textit{Prnp} from short-read RNA-Seq reads was due to the mouse gene or the human transgene. No change in transcript expression was identified from Iso-Seq full-length read counts. 

\begin{figure}[htp]
	\includegraphics[page=1,scale = 0.4]{Figures/WholeDifferentialAnalysis_DMPDMR.pdf}
	\\
	\hspace*{0.2cm}\vspace{0.5cm}\large c
	\\
	\includegraphics[page=1,trim={1.5cm 0 0 0},scale = 0.9]{Figures/AS3MT_DMP.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Differential splicing and methylation of \textit{As3mt}]%
	{\textbf{Differential splicing and methylation of \textit{As3mt}}: \textbf{A}) Upregulation of known isoform associated with \textit{As3mt} (PB.8363.2, ENSMUST00000003655.8) in TG (P = 4.98 x 10\textsuperscript{-18}, R\textsuperscript{2} = 0.56) \textbf{(B)} A DMR of nine hypermethylated sites was identified upstream of 11th novel exon. \textbf{(D)} UCSC browser track showing relative position of DMR and isoform}    
	\label{fig:IntMeth_As3mt}
\end{figure}	

\newpage
\section{Conclusions}

%Classification of AS events, which most commonly observed/dominant? Isoforms derived from transcriptional regulation (alternative promoters) vs post-transcriptional regulation? 
%Impact of AS events on protein domains. Non-sense mediated decay? 
%%Furthermore changes in gene/transcript expression can be due to differences in cellular composition (i.e. neuronal loss/reactive gliosis) rather than indicative of disease-associated transcriptional regulation. 
%%Gene expression and mRNA isoforms vary widely across tissues (\cite{Wang2008}), thus sequencing the disease-relevant tissue (in this case entorhinal cortex) is important for understanding the pathology of AD. However, it is consequently important to note that other tissues may have to be considered to fully grasp the whole picture of AD development. 
%Genome-wide RNAseq study of the molecular mechanisms underlying microglia activation in response to pathological tau perturbation in the rTg4510 tau transgenic animal model - PubMed (nih.gov)
%Integrating human brain proteomes with genome-wide association data implicates new proteins in Alzheimer's disease pathogenesis - PubMed (nih.gov)
%The role of microglia in processing and spreading of bioactive tau seeds in Alzheimer’s disease | Journal of Neuroinflammation | Full Text (biomedcentral.com) 
%%https://actaneurocomms.biomedcentral.com/articles/10.1186/s40478-018-0574-5 - rTg4510




