\section{Differential analysis}

This section describes the general differential analyses that were performed downstream after processing long-read sequencing data in \textbf{Chapters 5 and 6}. Used as a global term, differential analysis is the means by which statistically significant differences in expression, splicing and usage of genes and transcripts are identified across experimental groups. Parameters that are specific to individual results chapters can be found in the Method section of the relevant chapter. 

\subsection{Gene and Isoform Quantification}\label{sec: gene_isoform_quant_explained}
Any differential analysis first requires estimation of gene and transcript expression. In handling the short-read nature of RNA-Seq data, previous bioinformatic tools and computational models have determined this primarily from the number of reads that align to each transcript sequence from a reference genome annotation\cite{Conesa2016} (\cref{fig:isoform_quant_strategy}\textbf{A}). While such approaches can accurately determine gene expression, it becomes much more challenging to estimate transcript expression due to the overlapping exonic structure of related transcripts, resulting in ambiguous read alignment (illustrated in \cref{fig:rna_seq_limitations}). Several sophisticated algorithms have been developed, including the Expectation-Maximization (EM) algorithm adopted in \textit{Kallisto}\cite{Bray2016} and \textit{RSEM}\cite{Li2011}, which assigns reads to multiple genomic loci and works without a reference genome\cite{Conesa2016}. 

In the advent of long-read sequencing data and the availability of matched short-read RNA-Seq data, gene and isoform expression can be estimated in two ways: i) a hybrid approach by mapping RNA-Seq data to Iso-Seq-derived or ONT-derived transcripts (\cref{fig:isoform_quant_strategy}\textbf{B}), or ii) directly using the normalised FL read count from long reads as a proxy of gene and transcript expression (\cref{fig:isoform_quant_strategy}\textbf{C}); notably, the gene expression is estimated from the summation of FL read counts from associated transcripts. While the former hybrid approach still suffers from ambiguous alignment to a degree, usage of the long-read-defined transcriptome in place of the reference genome would minimise misalignment, and improve mapping to condition-specific transcripts and other novel transcripts that are otherwise missing in the reference annotations\cite{Au2013}. Conversely, the latter approach does not rely on transcript assembly and is thus not impeded by misalignment. However, long-read sequencing data from whole transcriptome profiling is still considered semi-quantitative due to not achieving the sufficient coverage that is required for reliable differential analysis, despite accurate identification and characterisation of full-length transcripts.  

\begin{figure}[htp]
	\begin{center}
		\includegraphics[page=8,trim={2cm 19cm 2cm 1cm},clip, scale = 0.8]{Figures/ProjectDevelopment_Figures.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Strategies for isoform quantification]%
	{\textbf{Strategies for isoform quantification using long and short reads.}: A schematic diagram of the three strategies adopted for determining isoform abundance, using either \textbf{A)} RNA-Seq reads aligned to the reference genome, \textbf{B)} the hybrid approach with RNA-Seq reads aligned to the long-read defined transcriptome, or \textbf{C)} normalised FL read count directly from long reads. Long reads refer to both Iso-Seq and ONT reads.}
	\label{fig:isoform_quant_strategy}
\end{figure}


\subsection{Differential Gene and Transcript Expression Analysis}
Differential gene expression (DGE\nomenclature{DGE}{Differential Gene Expression Analysis}) or transcript expression analysis (DTE\nomenclature{DTE}{Differential Transcript Expression Analysis}) identifies genes or transcripts that have a statistically significant change in abundance across biological conditions (i.e. "differentially expressed") (\cref{fig:dte_dtu_explanation}\textbf{A}). To facilitate unbiased comparisons across samples and experimental groups, raw read counts are normalised to eliminate feature-length and library-size effects - longer transcripts and samples sequenced at a higher depth would accumulate more reads - to a standard metric, namely TPM (Transcripts per Million\nomenclature{TPM}{Transcripts per Million}). FL reads from long-read sequencing are thus normalised to TPM using the following: 

\begin{myequation}[!h]
	\begin{equation}
		FL\;\:TPM (x_{sample},y_{sample})=\frac{Raw\;\:FL\;\:count (x_{isoform},y_{sample})}{Total\;\:FL\;\:count (y_{sample})} *10^6
	\end{equation}
\end{myequation}
%With a cut-off lower than 0.5 TPM, a 0.5 - 10 TPM refers to low expression, a 11- 1000 refers to medium expression, and > 1000 TPM high expression [literature ref]. 

Between-sample normalisation methods, such as TMM\cite{Robinson2010} (Trimmed Mean of M-values\nomenclature{TMM}{Trimmed Mean of M-values}), are also used to account for differences in sample RNA library composition. This is particularly important when comparing samples from different genotypes. 

While significant computational advances have been made in processing long-read data for transcriptome annotations, methods to harness such data for downstream differential analyses have been limited. Current differential analyses of long-read data typically rely on existing tools developed for short-read RNA-Seq\cite{Amarasinghe2020}, such as \textit{DESeq}, \textit{maSigPro}, \textit{edgeR}, among others. Highlighting the challenges of performing such analyses, various benchmarking studies have demonstrated that the choice of tool can affect the outcome considerably and no single method performs favourably across all datasets. Notably, tools based on negative binomial modelling performed better with higher specificity and sensitivity\cite{Rapaport2013}. Recent methods, such as \textit{FLAIR} and \textit{LIQA}\cite{Hu2021}, have emerged specifically for isoform expression analysis of long-read data. However, such methods have not been systematically assessed and are challenging to use for time-series data analyses; targeted experiments in \textbf{Chapters 5 and 6} include data from two different conditions and across four time points. 

\subsection{Differential Splicing Analysis}\label{intro:dtu}
A change in alternative splicing can be assessed in two ways: i) differential transcript expression (DTE), as described above, defined by a change in \textit{absolute} expression of transcript, and ii) differential transcript (or isoform) usage (DTU\nomenclature{DTU}{Differential Transcript Usage}) defined by a change in the \textit{relative} expression of a transcript, manifesting to a change in the proportions of the transcript (or isoform) of a gene (\cref{fig:dte_dtu_explanation}\textbf{B}). As shown in \cref{fig:dte_dtu_explanation}, DTU always implies DTE whereas the reverse is not necessarily true; e.g. a two-fold increase of two associated isoforms results in a change in absolute but not relative expression (\cref{fig:dte_dtu_explanation}\textbf{A}), indicating a transcription-related mechanism. Conversely, any change in relative abundance of isoforms indicate a splicing-related mechanism. 

One well-known phenomenon characterised in differential splicing analysis is the significant altering of isoform proportions (also known as Isoform Fraction) resulting in detection of a different dominant isoform, known as major isoform switching (\cref{fig:dte_dtu_explanation}\textbf{C}). In this circumstance, the same isoform is predominantly expressed in one condition (major isoform) while lowly expressed in another (minor isoform). Notably, upregulation of one isoform could be compensated by downregulation of another, resulting in no net change at the gene-expression level (\cref{fig:dte_dtu_explanation}\textbf{D}). Transcriptomic profiling studies at the gene level would thus fail to capture such nuances, highlighting the complexity of gene regulation and the importance of performing differential splicing analysis at an isoform-level. 


\begin{figure}[htp]
	\begin{center}
		\includegraphics[page=20,trim={0cm 12cm 0 0cm},clip, scale = 0.8]{Figures/ProjectDevelopment_Figures.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth,singlelinecheck=off}
	\caption[Scenarios of Differential Splicing Analysis]%
	{\textbf{Scenarios of Differential Splicing Analysis.}: A schematic illustration of four different scenarios envisioned under differential splicing of a gene with two isoforms between conditions 1 and 2:
	\begin{enumerate}[label=\textbf{\Alph*})]
		\item Differential transcript expression (DTE) indicates an expression change for at least one transcript between conditions 1 and 2. However, the expression proportion of each transcript (defined as percentage of the total expression of all associated transcripts, in this case 50\%) remains constant).
		\item Conversely in differential transcript usage (DTU), the relative expression of the isoforms is changed between conditions - in this case, isoform B has a relative expression of 33.3\% in condition 1 (5/15), but a relative expression of 41.6\% in condition 2 (10/24).
		\item Differential transcript usage can occur with a switch of the major isoform - in this case, the more abundantly expressed isoform is switched from Isoform A in condition 1 to Isoform B in condition 2. 
		\item Differential transcript usage can result in no change in gene expression if the change of transcripts occur in opposite directions.
		\\
	\end{enumerate} 
	Figures and legends were adapted from Soneson et al. (2016)\cite{Soneson2016} 
   }
	\label{fig:dte_dtu_explanation}
\end{figure}

Despite the limited utility of short-read RNA-Seq data for elucidating differential splicing events, a number of computational methods have been developed (reviewed in \cref{tab: rnaseq_diffsplicing}), based around two major strategies: i) isoform-based and ii) count-based methods, which is further subdivided into exon-based and event-based. Isoform-based methods aim to reconstruct the transcripts from sequencing reads and estimate relative abundance in each sample, followed by statistical testing to identify transcripts with significant expression differences across experimental groups\cite{Mehmood2020}. Conversely count-based methods dissect genes into counting units and document the number of reads falling within those units\cite{Mehmood2020}; exon-based methods assigns reads into exonic and junction regions, whereas event-based methods quantify transcripts by inclusion of individual splicing events with a percent splicing index (PSI\nomenclature{PSI}{Percent-Spliced In})) value for each event (i.e. proportion of associated isoforms that contain the AS event of interest). 

However, similar to transcript quantification, there is no clear consensus about the optimal tool or pipeline for such analysis. Benchmarking studies have similarly revealed that the choice of tools can directly impact the sensitivity and precision to detect differential isoforms, which is influenced by the number of replicates and the conditions heterogeneity\cite{Merino2019}. Exon-based methods (i.e.\textit{DEXSeq, edgeR, limma}) were found to overall perform better than other methods with superior precision and sensitivity, with \textit{edgeR} recommended for faster performance and reduced memory requirements\cite{Mehmood2020}. 


\begin{changemargin}{1.5cm}	
	%\captionsetup{width=30cm}
	\begin{landscape}
		\small %smaller font
		\setlength\tabcolsep{2pt} %reduced margin size in table
		\renewcommand{\arraystretch}{1}
	\begin{longtable}[c]{p{2.5cm}p{2cm}p{2cm}p{2.5cm}p{17cm}}
		\caption[Overview of Differential splicing analysis methods]%		
		{\textbf{Overview of Differential splicing analysis methods}. \newline AS - Alternative Splicing, AF - Alternative First, IR - Intron Retention, MX - Mutually Exclusive, SE - Skipped Exon. Table is adapted from Mehmood et al. (2020) \cite{Mehmood2020} and is by no means comprehensive. }
		\label{tab: rnaseq_diffsplicing}\\
		\toprule
		Approach &
		Method &
		Annotation &
		Designs &
		Model \\ \midrule
		\multirow{2}{*}{Isoform-based} &
		Cufflinks \newline /cuffdiff2 &
		Yes, \textit{de novo} &
		2 groups &
		\tabitem Following transcript assembly, transcript abundance estimated by maximising the likelihood score across all possible combinations of relative abundances of each associated isoform \newline 
		\tabitem Variability between replicates and uncertainty in abundance accounted with a beta negative binomial model \\ 
		
		&
		DiffSplice &
		\textit{Ab initio} &
		2 groups &
		\tabitem Reconstructs a graph of the transcriptome based on reads, from   which the abundance is estimated from alternative paths and alternative splicing modules are identified genomic regions where transcripts diverge \newline 
		\tabitem  Abundance of modules is compared using a non-parametric permutation test \\ \cmidrule(l){1-5} 
		\multirow{4}{*}{Exon-based} &
		DEXSeq &
		Yes &
		Complex &
		\tabitem Applies a generalised linear model to exon-level expression data to model differential usage of exons across experimental groups, assuming that read counts follow a negative binomial distribution \\ 		
		&
		edgeR &
		Yes &
		Complex &
		\tabitem Fits a negative binomial generalised log-linear model to exon-level expression data  to test differential exon usage by comparing the log-fold-change of an exon to that of the gene \\
		&
		JunctionSeq &
		Yes, \textit{de novo} &
		Complex &
		\tabitem Uses a similar statistical method as DEXSeq with added features to include novel exon junctions in differential exon usage \\	
		&
		limma &
		Yes &
		Complex &
		\tabitem Fits a linear model to exon-level expression data for differential exon usage between experimental groups \\ \cmidrule(l){1-5} 
		\multirow{4}{*}{Event-based} &
		dSpliceType &
		Yes &
		2 groups &
		\tabitem For each AS event type (SE, RI, MX, A3SS, A5SS), it calculates the read coverage signal for each base and the normalised  logarithmic ratios of PSI between groups. DS event are then identified using a parametric test on the PSI \\ 			
		& 
		MAJIQ &
		Yes, \textit{de novo} &
		2 groups &
		\tabitem Uses local splicing variations, which denote splits in a splice graph mapping to the edges of a reference exon to calculate PSI.   
		\tabitem Change in PSI is then quantified using Bayesian modelling and bootstrapping \\ 		

		&
		rMATS &
		Yes &
		2 groups, \newline paired samples &
		\tabitem Calculates PSI for each AS event after applying a hierarchical framework to account for within-sample uncertainty and between-sample variability.
		\tabitem Mean PSI across each AS event is then tested between experimental conditions using a likelihood ratio \\ 	
		&
		SUPPA2 &
		Yes &
		2 groups, \newline paired samples &
		\tabitem Determines transcript abundance using RSEM to estiminate PSI   for each AS event \\* \bottomrule 
		\end{longtable}
\end{landscape}
\end{changemargin}
%Anvar2018

\subsection{TappAS: Integrated framework for Differential Analysis of Long-reads}\label{ch3_tappas_explained}
After trialling various methods, \textit{tappAS}\cite{DeLaFuente2020} (v1.0.0) was chosen as a framework for the differential expression and splicing analysis of long reads across biological conditions (i.e. AD vs non-AD) (\textbf{Chapters 5 and 6}). To date, it is the only tool that allows integration of isoform-level, long-read-derived annotations with public databases to comprehensively understand the functional implications of alternative splicing. Accessible as a user-friendly Java application, it provides the flexibility to use expression derived from short-reads or long-reads, and supports complex design experiments: i) case-control, ii) time-course single series and iii) time-course multiple series. Developed by the same authors as \textit{SQANTI}\cite{Tardaguila2018}, it was recommended as an extension to the Iso-Seq bioinformatics pipeline for the functional annotations of isoforms.    

The following sections detail specific analyses from \textit{tappAS} in investigating differential expression and splicing changes associated with progressive tau pathology in rTg4510 mice at a global (\textbf{Chapter 5}) and targeted level (\textbf{Chapter 6}). All details are summarised from Lorena de la Fuente et.al (2020)\cite{DeLaFuente2020}.

\vspace{2cm}
\begingroup
\parindent=0em
\etocsettocstyle{\rule{\linewidth}{\tocrulewidth}\vskip0.5\baselineskip}{\rule{\linewidth}{\tocrulewidth}}
\etocsetnexttocdepth{5}
\localtableofcontents 
\endgroup


\begin{figure}[htp]
	\begin{center}
		\includegraphics[page=21,trim={0cm 5cm 0 0cm},clip, scale = 0.8]{Figures/ProjectDevelopment_Figures.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth,singlelinecheck=off}
	\caption[Overview of \textit{tappAS} for differential analysis of long-reads]%
	{\textbf{Overview of \textit{tappAS} for differential analysis of long-reads}: Shown is \textbf{A)} the \textit{tappAS} project creation workflow, which requires three input files: a transcript-level expression matrix, an experimental design file and a transcript-level functional annotation file. The expression matrix can be obtained from mapping RNA-Seq data to a long-read-defined transcriptome (\cref{fig:isoform_quant_strategy}\textbf{B}) or FL read count directly from long-read sequencing (\cref{fig:isoform_quant_strategy}\textbf{C}). \textbf{B)} Overview of \textit{tappAS} modules for functional isoform annotation and implications of alternative splicing. Figures and legends are adapted from Fuente et al. (2020)\cite{DeLaFuente2020}.
	}
	\label{fig:tappAS_overview}
\end{figure}

\clearpage
\subsubsection{Functional annotations of long-read derived isoforms}
\textit{tappAS} requires three inputs (\cref{fig:tappAS_overview}\textbf{A}) :
\begin{enumerate}
	\item An experimental design file to enable comparisons between two or more groups and/or over a time-course 
	\item A transcript-level functional annotation file, which is generated post \textit{SQANTI} using \textit{IsoAnnot} (https://isoannot.tappas.org), as a "scaffold" for transcript-level annotations. For the purpose of this thesis, the annotation file was provided as a conglomerate, long-read defined transcriptome of all the samples merged. The annotations incorporate feature elements from public annotations at both the transcript and protein level. 
	\item A transcript level expression matrix, which can either be derived directly from the FL long-read transcript counts, or from mapping and transcript quantification of short-reads to the long-read defined transcriptome using \textit{Kallisto}\cite{Bray2016}. Raw transcript counts were tabulated per sample.  	 
\end{enumerate}


\subsubsection{Isoform pre-filtering and normalisation}
% Discussion of expression 
%The choice to use quantification directly from long-reads or derived indirectly from short-reads is dependent on the read coverage. While there will be no assembly ambiguity in using the abundance counts from long-reads, the coverage from the whole transcriptome approach would be insufficient for meaningful quantitative analyses - as shown in the coverage with ERCC (see Figure \cref{fig:isoseq_whole_ercc}) with lowly-expressed transcripts not being detected. Conversely, this is unlikely to be a limiting factor for the targeted transcriptome approach given the saturation coverage of target genes with additional sequencing of off-target, highly-expressed genes (see Figure \cref{fig:isoseq_targeted_rate}). Consequently, the expression matrix input for the whole transcriptome analyses was derived from the short-read alignment, and from the long-read abundance for the targeted transcriptome analyses. 
Lowly-expressed transcripts with a sum of expression value less than 1 CPM (counts per million\nomenclature{CPM}{Counts per million}) or a large variance (>100 Coefficient of Variation) across all the samples were removed to reduce noise. The raw transcript counts were then normalised using TMM normalisation \cite{Robinson2010} to account for differences in library size (sequencing depth) and sample RNA library composition. Of note, TMM assumes that the majority of the transcripts are not differentially expressed. Gene abundance was then deduced from the sum of normalised counts of associated isoforms, after removing transcripts with low or highly-varied expression values. 
 

\iffalse
\begin{changemargin}{1cm}	
	%\captionsetup{width=30cm}
	\begin{landscape}
		\small %smaller font
		\setlength\tabcolsep{2pt} %reduced margin size in table
		\renewcommand{\arraystretch}{1}
		\begin{longtable}[c]{p{2cm}p{5cm}p{5cm}p{13.5cm}}
		\caption[Overview of Differential splicing analysis methods]%		
		{\textbf{Overview of Differential splicing analysis methods}. \newline }
		\label{tab: tappas_publicannotations}\\
				\toprule
		Level                    & Elements                     & Public Annotation     & Functional Implications    \\ \midrule
		\multirow{4}{*}{Transcript} & UTR elements \& uORFs & UTRscan      & mRNA subcellular localisation, translation efficiency and stability           \\
		& repeat \& low complexity regions & repeatMakser & chromatin organisation by serving as boundaries for heterochromatin   domains \\
		& miRNA binding sites          & mirWalk2.0 ,mirRbase  & mRNA decay and translation \\
		& RNA binding protein sites    & CLIP data from CLIPdb &                            \\
		\hdashline[0.5pt/5pt]
		\multirow{6}{*}{Protein} & Pfam domains                 & InterProScan          &                            \\
		& Transmembrane regions        & TMHMM                 &                            \\
		& Signal peptides              & SignalP               &                            \\
		& Coiled-coil region           & COILS                 &                            \\
		& Nuclear Localisation Signals & cNLS mapper           &                            \\
		& Disordered regions           & MobiDB Lite           &                            \\ \bottomrule
		\end{longtable}
	\end{landscape}
\end{changemargin}
\fi


\subsubsection{Functional Diversity Analysis}
The functional diversity of all multi-isoform genes were catalogued by performing pairwise comparisons between isoforms of the same gene, testing for either the i) presence or absence of the annotated feature, or ii) the genomic position of the annotate feature; a feature is classified as varying if the feature's coordinates differ by >9bp between isoforms of the same gene. Of note, many of the protein-defined features can be explored using both approaches given that these elements can be affected through complete skipping (resulting in absence) or partial disruption (resulting in varied genomic position).

%\subsubsection{Differential Feature Inclusion Analysis}

\subsubsection{Differential Gene and Isoform Expression Analysis}
\textit{maSigPro} performs a two-step regression strategy to first define a negative binomial general linearised models\cite{Nueda2014} for each gene or transcript, accounting for both condition and longitudinal effects, and identify differentially expressed genes. A stepwise regression is then applied to identify the conditions for which the differentially expressed genes have statistically significant profiles, determined by a user-defined threshold for the R\textsuperscript{2} of the regression model; the R\textsuperscript{2} defines the proportion of deviance that was explained by the linear regression model ("goodness of fit"), whereby a recommended threshold of 0.5 was used to identify differentially expressed genes with meaningful biological implications\cite{Conesa2006}. P-values were adjusted for multiple testing, by controlling the false discovery rate (FDR\nomenclature{FDR}{False Discovery Rate}) with the Benjamin and Hochberg correction; FDR-adjusted P values < 0.05 were considered as significant.

Following the identification of statistically significant gene models, the specific conditions (phenotype or time-associated changes) for which the genes show statistically significant profile changes (the significant variables) were identified by using an iterative backward stepwise approach \cite{Conesa2017}. The procedure therefore first started with all the variables imputed (different phenotype x age at different time points). At each iteration step, the P-value associated to each variable was determined and only the variables with a P-value < 0.05 were retained. 

\subsubsection{Differential Transcript Usage}
\label{ch:diu_method}
In addition to assessing expression changes across conditions through differential transcript expression analysis, the relative expression, and as such the usage, of these isoforms can also change (described in \cref{intro:dtu}). To identify genes that exhibit differential transcript usage, the proportion (of fraction) of the associated isoform was calculated using the following:

\begin{myequation}[!h]
	\label{eqn:tappas_IF}
	\begin{align}
		IF_{cig} = \frac{\bar{E}_{cig}}{\sum_{i=1}^{n}\bar{E}_{cig}}
	\end{align}
	where:
	\begin{conditions*}
		\hspace{3mm}\conj{E}\textsubscript{cig} & mean normalised expression for isoform \textit{i} associated to gene \textit{g} under condition \textit{c}\\
		\hspace{3mm}n  & total number of isoforms associated with gene \textit{g}
	\end{conditions*}
	%\captionsetup{width=0.95\textwidth}
	%\caption[Calculation of isoform fraction for differential isoform usage analysis]%
	%{\textbf{Calculation of isoform fraction for differential isoform usage analysis}. Equation is adopted from \textit{tappAS}}    
\end{myequation}

% Need more information from paper on how DIU was performed
Despite abundant evidence of widespread isoform diversity \cite{Wang2008}, most protein-coding genes have been reported to typically express a few dominant isoforms \cite{Gonzalez-Porta2013, Ezkurdia2015}, while the remaining are very lowly expressed and unlikely to be main contributors to the proteome \cite{Gonzalez-Porta2013}. As such, minor isoforms were filtered to avoid finding genes associated with differential transcript usage due to "flat" behaviour of these minor isoforms \cite{DeLaFuente2020} (relatively small non-negligible expression changes of minor isoforms in the opposing direction of the predominant isoforms). \textit{tappAS} provides two strategies to filter lowly-expressed isoforms: an isoform is only retained if its proportion relative to other isoforms is greater than the pre-specified threshold (default: proportion $>$ 10\%) in at least one sample, or alternatively if its proportion relative to the major isoform is below a pre-specified threshold (default: FC = 2). A major isoform is defined as the isoform with the highest expression across all the conditions, with the remaining isoforms annotated as minor. Implemented as an additional filtering step after \textit{tappAS} and recommended in other bioinformatic tools\cite{Vitting-Seerup2017}, lowly expressed genes were also filtered as there would be less confidence in isoform fraction used for determining genes with significant differential isoform usage.   