\documentclass[../Main/Knit.tex]{subfiles}

\section{Alzheimer's Disease}

Alzheimer’s disease (AD\nomenclature{AD}{Alzheimer's disease}) is a devastating neurodegenerative disorder, clinically characterised by progressive memory loss, cognitive decline, and behavioural impairment. The most common form of dementia, it is estimated to affect XXX worldwide with numbers expecting to increase to X by 2050, ensuing both a heavy economic and social burden amounting to £XXX each year. Despite international efforts to better understand the disorder for drug discovery and development, there are currently no cure and existing medication only act to reduce symptoms.

\subsection{Pathology}
The symptoms of AD are underpinned by both morphological and molecular changes in the brain, initially in the temporal lobes (hippocampus and entorhinal cortex) and later in the frontal lobes. Conversely, the occipital lobes, motor cortex, and the cerebellum are relatively resistant to neuronal degeneration even in advanced stages of AD.

Neuroimaging scans and post-mortem brain analysis from patients reveal significant brain atrophy caused by neuronal and synaptic loss. Further microscopic examination reveal accumulation of beta-amyloid (Abeta) in amyloid plaques and aggregation of tau in neurofibrillary tangles, which are now believed to manifest years before presentation of clinical symptoms and diagnosis. In addition to these neuropathological changes, there is increasing evidence for the causative role of the innate immune system. Despite the well characterisation of these neuropathological hallmarks, the exact biological mechanisms driving AD onset and pathogenesis are still widely unknown.


\subsection{The two hallmarks: plaques and tangles}
Amyloid plaques are extracellular deposits of amyloid-beta, short fragments produced from sequential cleavage of APP (amyloid precursor protein\nomenclature{APP}{Amyloid Precursor Protein}), a transmembrane protein involved in synapse formation and stability, by beta- and gamma-secretase (BACE\nomenclature{BACE}{Beta-secretase}). It is thought that in AD, the processing of APP is altered resulting in imbalanced ratio of longer (and more aggregating) and shorter forms of Abeta, with an increased propensity to form plaques that disrupt synaptic transmission and cause neuronal apoptosis. 

Neurofibrillary tangles (NFT\nomenclature{NFT}{Neurofibrillary tangles}) are dense intracellular aggregates of misfolded and hyperphosphorylated tau, which is a microtubule-associated protein involved in microtubule maintenance and stability. It is thought that in AD, the increased phosphorylation of tau induces detachment from microtubule with an increased propensity to form tangles of paired helical filaments that disrupt microtubule function and subsequent axonal growth and transport. The degree/amount of neurofibrillary tangle formation is further found to be closely associated to the severity of AD, allowing AD classification into 6 stages (BRAAK stages) that are defined by the spread of NFTs. 


\subsection{Genetic Component}
AD is commonly known to affect people who are aged 65 and above (termed late-onset Alzheimer’s disease, LOAD\nomenclature{LOAD}{Late Onset Alzheimer's Disease}), with younger patients accounting for 5\% of total AD cases (termed early-onset Alzheimer’s disease, EOAD\nomenclature{EOAD}{Early Onset Alzheimer's Disease}). While LOAD is complex with a heterogeneous genetic composition and a heritability of 50-80\%, EOAD is almost completely genetically determined with EOAD patients presenting a clear familial autosomal dominant pattern of inheritance (Familial Alzheimer’s disease, FAD\nomenclature{FAD}{Familial's Alzheimer's Disease}) (Jarmolowicz et al. 2015); to date, more than 160 highly-penetrant, causative mutation have been identified in EOAD, all located within three genes involved in amyloid plaque formation: APP, PSEN1 and PSEN2 (presenilin 1 and 2, which are components of BACE\nomenclature{PSEN1}{Presenilin 1}\nomenclature{PSEN2}{Presenilin 2}) (Chai, 2007).

Despite challenges to identity causative mutations in LOAD, being a complex disorder with a heterogeneous etiology, the emergence of genome-wide association studies (GWAS\nomenclature{GWAS}{Genome-wide association studies}) and subsequent meta-analyses has facilitated the identification of multiple genetic loci that are associated with an increased risk of developing LOAD. These genetic loci are typically changes or variants of single DNA base-pair (single-nucleotide polymorphisms – SNPs\nomenclature{SNP}{Single Nucleotide Polymorphism}) that are more commonly found in individuals with LOAD than without. 

To date, the most recent GWAS meta-analysis of 74,000 AD individuals identified over XX significant LOAD risk loci, many of which were annotated to the non-coding cis regulatory regions of gene (Lambert et al., 2013). At least 42 genes/loci have been associated with LOAD at genome-wide significance in at least one GWAS \citep{Verheijen2018}. These genes included BIN1 (bridging integrator 1\nomenclature{BIN1}{Bridging Integrator }), CLU (clusterin)\nomenclature{CLU}{Clusterin}, CR1 (complement receptor 1\nomenclature{CR1}{Complement Receptor 1}), PICALM (phosphatidylinositol binding clathrin assembly protein\nomenclature{PICALM}{Phosphatidylinositol Binding Clathrin Assembly Protein}), with the most significant genetic locus annotated to APOE (apolipoprotein E\nomenclature{APOE}{Apolipoprotein E}); inheritance of both APOE allele increases the risk of AD development by X\%. Common biological pathways emerging from these GWAS studies are immune response, lipid metabolism, endocytosis, and cell adhesion molecule (CAM) pathways (\citep{Verheijen2018}). 

Collectively, these common but low penetrant variants, with the exception of APOE, contribute modestly to the risk of developing AD, highlighting the polygenic nature of AD. The mechanisms behind these variants currently remain poorly understood, however they typically fall into three main biological pathways that may play an important role: the immune system and inflammatory responses, cholesterol and lipid metabolism, and endosomal vesicle recycling. Comprehensive case-control examination of genes proximal to these LOAD-associated variants have further revealed significant differential changes in gene expression and splicing (Humphries et al. 2015), implicating the role of transcriptomic dysregulation in AD pathogenesis. The very fact that most variants lie within the introns rather than exons suggest that it is the fine tune balance of gene expression and regulation that is at play, emphasising the importance epigenomic and transcriptomic studies. 

\subsection{Mouse Models}
Molecular changes in both genes and regulatory regions are highly conserved between human AD and mouse model neurodegeneration,

\subsection{Currently available mouse models in AD}

\pagebreak
\section{Gene expression and regulation}

Common observation from gene expression analysis is that genes typically express multiple isoforms, and the greater the number of annotated isoforms, the greater the number of expressed isoforms (with a plateau of 12 isoforms) (\cite{Djebali2012}). However, as most studies are performed on bulk-tissues, it is unclear whether this is a consequence of multiple isoforms in one single cell or from multiple isoforms from multiple single cells. Perhaps assumed but the expression of alternative isoform is also not consistent, with usually a dominant isoform (\cite{Djebali2012})

MicroRNA \nomenclature{miRNA}{micro RNA}), 22 nucleotides long, involved in regulation of gene expression through various ways, including promotion of transcript degradation and inhibition of translation machinery. This is typically achieved by the contact of miRNA with the 3'UTR of mRNA. It is estimated that up to XX\% of genes are regulated by miRNAs, and has been found to multiple roles in immune functions.

\pagebreak
\section{Transcriptional profiling}
Transcriptome profiling by the identification of full landscape of transcribed elements is critical to elucidate the functional relationship between the genomic loci and molecular mechanisms that drive development and diseases. Transcriptome profiling of disease-relevant tissue has enabled discovery of pathogenic coding and non-coding splicing variants in rare diseases, that would have otherwise been missed by exome and whole-genome sequencing in Mendialian disease diagnosis (\cite{Cummings}, \cite{Kremer2017})
	
With Mendialian diseases such as Duchenne muscular dystrophy, pathogenic variants that result in aberrant splicing (exon inclusion, exon skipping, exon extension, intronic splice gain, exonic splice gain) can have significant downstream impacts (i.e. loss of function). A genetic variant can result in aberrant splicing in the following ways (\cite{Cummings}):
\begin{itemize}
	\item variant at the splicing donor or acceptor site resulting in a masked splicing site and downstream alternative site used for splicing, thus exonic extension
	\item variant at the splicing donor or acceptor site resulting in masked splicing site, exon skipping
	\item variant within an intron (cryptic splice site), resulting in a strong splicing site and thus intronic splice gain
	
\end{itemize}


Transcriptome diversity is highly regulated by various mechanisms: 
\begin{itemize}
	\item Alternative transcription initiation (ATI) \nomenclature{ATI}{Alternative Transcription Initiation}) 
	\item Alternative cleavage and alternative polyadenylation (APA) \nomenclature{APA}{Alternative Poly-Adenylation})
	\item Alternative splicing (AS) \nomenclature{AS}{Alternative Splicing})
\end{itemize}

Alternative splicing and polyadenylation is a widespread phenomenon that facilitates generation of multiple distinct mRNA transcripts or isoforms from one gene, which are subsequently translated to different protein isoforms with unique, and potentially, antagonistic functions (E. T. Wang et al., 2008). AS further regulates gene expression through various mechanisms: non-sense mediated decay, miRNA-mediated mRNA degradation, altered translational efficiency of isoforms. In contrast, alternative polyadenylation regulates RNA transportation, localization, stability, and translation by generating splice isoforms with different cleavage sites.

Alternative splicing is essential in shaping transcriptome and proteome diversity - over 95\% of 22,000 protein-coding multi-exonic human genes are estimated to undergo alternative splicing, with up to 70\% containing multiple polyadenylation sites and 30\% having multiple first exons due to alternative transcription start sites. Each gene is estimated to have on average six transcript isoforms \cite{Dunham2012}, and this figure is likely to increase with more transcriptomic studies. It occurs most prevalently in the brain implicating its role in neuronal development and maintenance (Pan et al., 2008) (Mazin et al., 2014) (Raj, Blencowe, 2015). It is predicted that a single cell, with a transcription of 600,000 molecules, will have generated 5 - 15 conservative isoforms per gene, and 2-4 exon cassette isoforms (\citep{Karlsson2017}) (a single oligodendrocyte contained ~2000 conservative transcripts associated with 700 genes, and 1000 unique isoforms). 

Isoforms can differ at the 5' (alternative transcript start sites - TSSs), exons (alternative splicing) and 3' end (alternative transcription termination sites - TTSs)). Exon splicing can be further divided into alternative splice sites (alternative 5'-splice site, alternative 3'-splice site), exon skipping and intron retention. 
AS events can be classified into five different types: 
\begin{itemize}
	\item Intron retention \nomenclature{IR}{Intron Retention}, defined by the presence of an exon which overlaps with the intron of another transcript within the same gene. IR can introduce stop codons, subsequently prompting non-sense mediated decay but can also change open reading frame \nomenclature{ORF}{Open Reading Frame}, generating functionally different variant 
	\item Skipped exon \nomenclature{SE}{Skipped Exon}, defined by the presence a missed exon which is completely overlapped with an intron of another transcript 
	\item Alternative 5' splice site \nomenclature{A5SS}{Alternative 5' Splice Site}
	\item Alternative 3' splice site \nomenclature{A3SS}{Alternative 3' Splice Site}
	\item Mutually Exclusive Exon 
\end{itemize} In addition to above five common categories, many other complex types, such as alternative position, i.e., alternative 3' and 5' site (Wang and Brendel, 2006), AS and transcriptional initiation (ASTI) (Nagasaki et al., 2006) alternative first exons (Chen et al., 2007), and composite patterns (Wang and Rio, 2018), can occur."

\textbf{Nonsense mediated decay (NMD)} \nomenclature{NMD}{Nonsense Mediated Decay} products are alternatively spliced isoforms that are not translated into proteins, by containing an early stop codon. A premature termination-translation codon highly supportive of NMD is defined by a stop codon within at least 50-55 base pairs upstream of splice junctions. 

\textbf{Fusion Transcripts} are a consequence of trans-splicing event of merging two separately encoded pre-mRNA into one transcript

\textbf{Long non-coding RNA \nomenclature{lncRNA}{Long non-coding RNA}} are polyadenylated RNA with more than 200 nucleotides. 
	
\textbf{Natural Antisense Transcripts \nomenclature{NATs}{Natural Antisense Transcripts}} 

\textbf{3'Polyadenylation} Polyadenylation of 3'end of mRNA regulates mRNA stability and translation efficiency. Studies using long-read sequencing of human transcriptome have revealed differences in poly(A) length distribution between genes, and even between isoforms of the same gene  with protein-coding isoforms having shorter poly-A tails than intron-retaining isoforms (\cite{Workman2019a}. This is line with studies showing that hyperadenylation targets intron-retaining transcripts for degradation (\cite{Bresson2015})

\textbf{Allele-specific expression} Preferential transcription of RNA from the paternal or maternal copy, which can be assessed using long-read sequencing from coverage of heterozygous SNP. 

\subsection{Short-read RNA-sequencing}
Transcriptomic profiling of AD in human and mouse models (determination of changes in splicing patterns) have been traditionally performed using exon microarrays and more recently, RNA-Sequencing (RNA-Seq\nomenclature{RNA-Seq}{RNA-Sequencing}) (Table X).
Multiple methods for transcriptome profiling in the past: 
\begin{itemize}
	\item One of the first methods of transcriptome profiling is to use multiple expressed sequence tags (EST) \nomenclature{EST}{Expressed Sequence Tags}), short oligonucleotide tags, that can be sequenced - Serial Analysis of Gene Expression \nomenclature{SAGE}{Serial Analysis of Gene Expression}).
	\item Hybridisation of cDNA to oligonucleotides on an array (microarray) i.e Affymetrix's GeneChips, also allowing examination of individual exons
	\item Quantitative PCR for validation of expression data 
\end{itemize}

Through massively-parallel sequencing of amplified DNA templates in a “sequence-by-synthesis” fashion to generate short-reads (Figure X) rather than relying on hybridization of target and probe, RNA-Seq allows deep surveying of the entire transcriptome, with transcript identification and quantification, and interrogation of alternative splicing events by discovery of splice variants and polymorphisms. With greater signal-to-noise ratio and higher nucleotide-level resolution, has been effective in identifying AS events such as exon skipping and intron retention, with the establishment of its role in diseases (E. T. Wang et al., 2008). 

Typically, several millions of 25-150bp (typically <700bp) reads are generated and aligned to genome to identify transcribed sequences. Major advances, including generation of reads that retain information on transcript orientation, allowing input of low yield or quality, have revolutionised the field. Also, now possible to sequence transcriptome \textit{de novo}, allowing characterisation of novel organisms. 

However, despite its power to identify and quantify gene expression (transcriptional profiling at a gene level), RNA-Seq is severely limited in assembling and reconstructing transcripts due to the reliance of short-reads that are only able to span a small part of the transcript rather than the full length (Figure X) \cite{Gordon2015}\cite{Wang2016}; short reads have an average length of 100-500bp, whereas transcripts are on average 2-3kb - 50\% of human transcripts are > 2.5Kb, with a range from 186bp to 109kb (Piovesan, Caracausi, Antonaros, Pelleri, \& Vitale, 2016) \cite{Sharon2013}. In particular, there are three transcriptional features that are difficult to characterise with short reads \cite{Kuo2017}:
\begin{enumerate}
	\item Transcript start sites (TSS) \nomenclature{TSS}{Transcription Start Sites} and Transcript termination sites (TTS) \nomenclature{TTS}{Transcription Termination Sites},for which any interior multiple TSS and TTS sites within a transcribed locus would be undetected due to overlapping exons and splicing junctions, and low coverage  
	\item Exon chaining given that short-reads typically only span one splice junction. Thus, while short-reads may able to accurately identify the exons present, the exact sequence and linking of the exons are predicted by short-read assemblers with challenges (Figure \ref{fig:kuo_splicing}).  
	\item Transcriptional Noise, particularly of reads in intronic regions that are falsely identified as intron retention, or of reads in intergenic regions that are erroneously classified as fusion gene. 
\end{enumerate}
It is therefore unclear which combination of exons are spliced in, and whether alternative (distant) exons pairs are included in mutually exclusive or independent fashion (i.e. whether events are coordinated though some distant alternative exons have shown to be correlated included (\cite{Fagnani2007})Furthermore, short-read RNA-sequencing fails to capture the connectivity of exons and informs whether the alternative processive events are coordinated (coordination is defined by two or more alternative RNA processing events are dependent of each other and the probability of this occurrence is greater than the observation of the sole event). --> Molecular co-association of distant human alternative exons



\begin{figure}[h]
	\centering
	\vspace{20pt}
	\includegraphics[width=0.8\linewidth, height=0.2\textheight]{Pictures/kuo_splicing_picture.png}
	\captionsetup{width=0.95\textwidth}
	\caption[Challenges of using short-reads for transcript assembly]%
	{\textbf{Challenges of using short-reads for transcript assembly}: Example of a transcript model that is impossible to resolve using short-reads (yellow). Figure and caption taken from \cite{Kuo2017}}
	\label{fig:kuo_splicing}
\end{figure}	

Various bioinformatic packages have been developed to assemble these short reads into transcripts, by probabilistically assigning and mapping reads to isoforms and exon-exon boundary or XXX, to identify and estimate transcript abundance (Figure X) (Trapnell et al., 2010)(Kingsford, Schatz, \& Pop, 2010)(Au et al., 2013). This, however, requires complex computational analysis and has resulted in conflicting outcomes and limited success, compounded by the fact that alternative transcripts often have significant overlaps and only a minor proportion of reads span splicing junctions. These tools further rely heavily on reference annotation libraries (RefSeq/Ensembl) or predefined splicing events, which may be inaccurate or incomplete; resulting in prediction of transcripts that do not exist (false positives) or fails to detect true transcripts (false negatives) particularly with genes that have large number of variants (Au et al., 2013). Pre-defined models are particularly limiting when comparing splicing profiles between different conditions, such as control versus transgenic mice, as any splicing changes observed are likely to be AD-specific. While there are tools that are de novo, these typically generate different and often conflicting results [Table X]. 

Attempts to overcome challenges with transcriptome assembly included generation of “synthetic long reads”, by tagging full-length complementary DNAs with unique molecular identifiers (UMIs) before cluster amplification and sequencing on Illumina (Tilgner et al., 2015). With the presence of UMIs, transcript isoforms can be reconstructed for up to 4Kb for isoform discovery and expression analysis (Stark, Grzelak, \& Hadfield, 2019). [However…]
RNA-Seq is thus impaired to profile the transcriptome at an isoform-level, investigate cis-acting mechanisms with transcripts, and characterise the functional aspects of isoform diversity (Tardaguila et al., 2018)(Hayer et al., 2015).

\subsection{Long-read sequencing approaches}
The limitations with RNA-Seq were addressed with the emergence of long-read, third-generation sequencing approaches, which generated longer reads that were able to span the full-length transcript. Rather than massively-parallel sequencing of templates in “wash-and-scan” fashion that resulted in de-phasing and subsequently shorter reads, both platforms allowed real-time sequencing of templates in an uninterrupted and processive manner. Two technologies currently dominate this space: Single Molecule Real Time (SMRT\nomenclature{SMRT}{Single Molecule Real Time}) from Pacific Biosciences (PacBio \nomenclature{PacBio}{Pacific Biosciences}) and protein nanopore sequencing technology from Oxford Nanopore Technologies (ONT \nomenclature{ONT}{Oxford Nanopore Technologies}). The performance and cost specifications of these two platforms are outlined in Table X. Other long read sequencing methods and protocols, synthetic long read (SLR \nomenclature{SLR}{Synthetic Long Read}) (\cite{Tilgner2015}) or sparse isoform sequencing (spISO-seq \nomenclature{spISO-seq}{Sparse Isoform Sequencing}) (\cite{Tilgner2018}), however these require more complex workflows. 

The consequent generation of longer reads, ranging from 300 – 20,000 bases provided unprecedented ability to sequence entire or new entire lengths of transcripts from 5’ end to polyA tail, relinquishing the need for transcriptome assembly and resolving splicing junctions. Allowing greater accuracy at transcript identification, an increasing number of studies have used such technologies to characterise isoform diversity and splicing with unprecedented success (Table X). Generally in comparison with RNA-Seq, Iso-Seq encapsulates longer transcripts, identifies novel gene locus, and correction of gene model. "Long transcript reads provide better support and higher accuracy in splice junctions than short reads, when these reads are aligned back to the genome. Thus gene models predicted from long reads yield more accurate exon/intron structure and can merge two or more misannotated adjacent genes."


\subsection{Hybrid approach of short and long read sequencing}
Despite the ability of long-read sequencing (particularly, Iso-Seq\nomenclature{Iso-Seq}{Isoform Sequencing}) to discover large number of novel and longer transcripts and identify complex splicing events such as alternative adenylation, there are inherent biases to sequencing the more highly-expressed and relatively shorter transcripts. Consequently, while the new chemistry has improved the error rate and increased throughput, the coverage is still insufficient for accurate transcript quantification and sensitive differential transcript analysis based on long reads alone (Koren et al., 2012). Furthermore, there is currently no consensus to validate or functionally characterise these transcripts (B. Wang, Kumar, Olson, \& Ware, 2019). The current standard for such application is thus a hybrid approach of aligning the short-reads to the long-reads to improve alignment and assemblage, and for downstream isoform quantification. 

\subsection{Isoform quantification} 
Isoform-specific expression can be deduced from short-reads alone using statistical models if the gene is well annotated (i.e. all isoforms are known) based on i) reads aligning to contiguous genomic segment (exonic reads) and ii) reads aligning to two contiguous segments with a single gap of 60-400bp (junction reads)(Jiang and Wong, 2009)).

Various bioinformatic tools and computational models have been developed to quantify isoform quantification from RNA-Seq data. There are currently two main methods:
\begin{enumerate}
\item Inclusion level, calculated for a regulated exon by aligning reads either to candidate alternative exons and its junctions (inclusion reads), or to flanking exons and subsequently skipping the candidate alternative exon (skipping/exclusion reads) (Chen et al. 2012)
\item Percent-Spliced-In (PSI\nomenclature{PSI}{Percent-Spliced In}), calculated by proportion of isoforms that include the exon (Venables et al. 2008)(Katz et al. 2010). If the PSI value is calculated for a particular splicing event, it can be considered equivalent to the inclusion level. 
\end{enumerate}
Isoform quantification can either be expressed as a global measure of expression, which provides a global gene expression ranking in one sample (measured by RPKM: Reads of a transcript sequence per Millions mapped read\nomenclature{RPKM}{Reads of a transcript sequence per Millions}), or as a relative measure of expression, which is normalized per gene locus and comparable across conditions (measured by inclusion level or PSI value). 

Isoform abundance calculated by aligning short-reads to transcriptome is preferential to alignment with reference annotation library (RefSeq/GENCODE) in narrowing down the isoforms expressed and thus subsequently enabling more reliable abundance quantification. Reference annotation library is constructed on all data from the same species, and inclusion of annotated but not truly expressed isoforms can increase variability of abundance estimates. Finally, if the reference library is incomplete, then truly expressed isoforms would be completely missed and RNA-Seq reads would be incorrectly assigned to annotated isoform (\cite Au2013)


\textbf{Differential splicing analysis }\\
When analyzing splicing patterns between multiple conditions, changes in isoform abundance can be defined in two ways: 
\begin{enumerate}
	\item Differential Isoform Expression (DIE\nomenclature{DIE}{Differential Isoform Expression}): changes in absolute expression of an isoform, evaluated using count matrixes 
	\item Differential Splicing (DS\nomenclature{DS}{Differential Splicing}): changes in relative expression of an isoform from the same gene, resulting in a change in isoform proportion and is evaluated using changes in gene exon usage	
\end{enumerate}

Figure X shows an example of a change in DIE but no change in DS: A two-fold increase of both isoforms from the same gene results in a change in absolute but not relative expression to one another. A change in DIE but not in DS may indicate a transcription-related mechanism. If a change in DS is observed, a change in DIE of one of the isoforms would also be observed. A change in multiple isoforms would also be observed, as long as the change is not in the same direction (upregulated/downregulated) with the same magnitude. Any changes in DS/relative abundance of isoforms indicate a splicing-related mechanism. 

In addition to exploring differential splicing in terms of isoform abundance, which typically involves an exon-based approach that focuses on differential exon usage (i.e. DEXSeq), a splicing based approach can also be taken. This involves analyzing individual splicing events (exon skipping, alternative donor and acceptor) for systematic changes between conditions. rMATS, SUPP2, LeafCutter and Majiq are such tools that identify and quantify splicing events using junction reads. 


\section{Aims and Objectives}
\begin{enumerate}
	\item Whole transcriptome analysis of AD post-mortem brain tissues as reference dataset, shed light on differential isoform expression
	\item Particular interest on 19 loci identified from meta-analysis of GWAS studies on AD (Lambert et al. 2013) Targeted transcriptome analysis  
	\item Classification of AS events, which most commonly observed/dominant? Isoforms derived from transcriptional regulation (alternative promoters) vs post-transcriptional regulation? 
	\item Impact of AS events on protein domains. Non-sense mediated decay? 
	\item Integration with other (epi)genetic analysis on same samples, i.e. DNA methylation, lysine acetylation, gene expression 
	\item Protein analysis? Integration with any publicly available mass-spec datasets 
\end{enumerate}

Gene expression and mRNA isoforms vary widely across tissues (\cite{Wang2008}), thus sequencing the disease-relevant tissue (in this case entorhinal cortex) is important for understanding the pathology of AD. However, it is consequently important to note that other tissues may have to be considered to fully grasp the whole picture of AD development. 

While human post-mortem brain tissues remain to be the gold standard for transcriptomic studies, important to highlight that post-mortem interval and storage conditions of brain material highly influence transcriptome stability, particularly affecting alternative splicing. Furthermore changes in gene/transcript expression can be due to differences in cellular composition (i.e. neuronal loss/reactive gliosis) rather than indicative of disease-associated transcriptional regulation. 

\section{Future Directions}
At time of writing, there have been other major advances in the field that would unfortunately not be explored. This include, single cell transcriptomics and direct RNA-Sequencing: analysis of mRNA expression at the resolution of individual, "single", cells, allowing representation of cell-to-cell variation rather than taking the stochastic average from bulk measurements, and thereby resolving heterogeneity. This is currently achieved by the capture and analysis of single cells using a microfluidic or droplet-based technology. Importance of single cell approaches highlighted in \cite{Karlsson2017} with few isoforms shared between cells (7\% of all detected isoforms shared between all cell-types, though this increased to 60\% for exon-cassette isoforms). 

Single-cell studies have highlighted the difference in transcriptome diversity at a single cell level, with small overlap of isoforms between cells (\cite Karlsson2017). Previous methods on quantifying transcripts at a single cell level have relied on RNA-fluorescence in-situ Hybridisation (RNA-FISH), which is limited in terms of throughput and characterisation of complex splicing events (\cite Byrne2017)

While the methods I have adopted for long-read sequencing in this thesis allows interrogation of full-length transcripts, this is reliant on the generation and amplification of cDNA from mRNA, which can produce artefacts (template switching), introduce bias (distortion of relative cDNA abundance) and lose RNA modifications. In 2018, ONT showed that it was able to sequence RNA directly using the minION by adding poly(T) adapters directly to the mRNA, with a translocase that was able to bind and process RNA efficiently	 \cite{Garalde2018}, achieving coverage and accuracy comparable to that with ONT-cDNA method. 

