\documentclass[../Main/Knit.tex]{subfiles}

\section{Alzheimer's Disease}

Alzheimer’s disease (AD\nomenclature{AD}{Alzheimer's disease}) is a devastating neurodegenerative disorder, clinically characterised by progressive memory loss, cognitive decline, and behavioural impairment. The most common form of dementia, it is estimated to affect XXX worldwide with numbers expecting to increase to X by 2050, ensuing both a heavy economic and social burden amounting to £XXX each year. Despite international efforts to better understand the disorder for drug discovery and development, there is currently no cure and existing medication only act to ameliorate symptoms.

\subsection{Pathology}
The symptoms of AD are underpinned by both morphological and molecular changes in the brain. Neuroimaging scans and post-mortem brain analysis from patients reveal significant brain atrophy caused by neuronal and synaptic loss. Further microscopic examination reveal accumulation of beta-amyloid (A$\beta$) in amyloid plaques and aggregation of tau in neurofibrillary tangles (NFT\nomenclature{NFT}{Neurofibrillary tangles}), two key hallmarks of AD, which are now believed to manifest years before presentation of clinical symptoms and diagnosis (\cref{fig:AD_intro}). These neuropathological changes are accompanied with heightened neuroinflammation through abnormal activation and distribution of microglia (most abundant brain resident immune cells) and astocytes (glial cells with multiple roles in supporting neuronal function and metabolism). 

The progression of these neuropathological changes have been well mapped, particularly the spread of NFTs under the Braak staging\cite{H1991},  with pathology initially apparent in the temporal lobes (hippocampus and entorhinal cortex) and later in the frontal lobes (\cref{fig:AD_development}). Conversely, the occipital lobes, motor cortex, and the cerebellum are relatively resistant to neuronal degeneration even in advanced stages of AD.

% Protein aggregates a common feature of neurodegenerative diseases; tie in AD with other dementias
Of note, it is important to emphasise that aside from A$\beta$ deposition and NFT formation, 15-20\% of AD patients are also presented with  Lewy body (LB) pathology \cite{C1995,L2003} - the defining pathological hallmark of Parkinson's disease (PD)\cite{Wakabayashi2007}\nomenclature{PD}{Parkinson's Disease} and dementia with Lewy bodies (DLB)\cite{Spillantini1997}\nomenclature{DLB}{Dementia with Lewy Bodies} and is chararacterised by abnormal aggregation of $\alpha$-synuclein into intraneuronal cytoplasmic inclusion bodies. Up to 75\% of AD patients further present neuronal cytoplasmic inclusions comprising of aggregates of TDP-43 \cite{King2010,McAleese2017,Arai2009} (Transactive response DNA binding protein-43), the defining hallmarks of frontotemporal dementia (FTD\nomenclature{FTD}{Frontotemporal dementia}) and amyotrophic laternal sclerosis\cite{Pesiridis2009} (ALS\nomenclature{ALS}{amyotrophic laternal sclerosis}).

\begin{figure}[!htp]
	\centering
	\includegraphics[page=1,trim={0 7cm 1cm 12cm},clip, scale = 0.8]{Introduction_Figures.pdf}
	\captionsetup{width=0.95\textwidth,singlelinecheck=off}
	\caption[Two key hallmarks of AD Neuropathology: amyloid plaques and neurofibrillary tangles]%
	{\textbf{Two key hallmarks of AD Neuropathology: amyloid plaques and neurofibrillary tangles}: Schema comparing a normal healthy brain and a brain with advanced AD. AD pathology is well characterised by the presence of extracellular amyloid plaques and intracellular neurofibrillary tangles, accompanied by significant neuronal loss and subsequent shrinkage of the neocortex and hippocampus. Figure is taken from Palmer (2015)\cite{AlanM.Palmer2015}
	}
	\label{fig:AD_intro}
\end{figure} 

\begin{figure}[!htp]
	\centering
	\includegraphics[page=1,trim={0 19cm 0cm 0cm},clip, scale = 0.8]{Introduction_Figures.pdf}
	\captionsetup{width=0.95\textwidth,singlelinecheck=off}
	\caption[Progression of amyloid plaques and neurofibrillary tangles with AD development]%
	{\textbf{Progression of amyloid plaques and neurofibrillary tangles with AD development}: Progression of \textbf{a)} amyloid plaques consisting of A$\beta$ measured according to Thal Phasing\cite{DR2002}, and \textbf{b)} neurofibrillary tangles composed of hyperphosphorylated tau by Braak staging\cite{H1991}. Figure is taken from Masters et al.(2015)\cite{Masters2015}. 
		\\
		\\ 
		The deposition of A$\beta$ (Figure a) can be mapped Thal Phasing from a) the neocortex, to b) allocortical regions comprising of the entorhinal cortex and hippocampus, the striatum, and finally to iv) subcortical regions\cite{DR2002}. 
		\\
		\\
		In a similar pattern, the progressive spread of NFTs can be classified under the six stages of Braak (Figure b) from i,ii) the transentorhinal regions such as the entorhinal cortex, to the iii) hippocampus, iv) adjoining neocortex and finally to vi,v) other neocortical regions\cite{H1991}. 	
	}
	\label{fig:AD_development}
\end{figure}


\subsection{Genetics Component}
Although AD is commonly known to affect people aged 65 and above (referred as Late-Onset Alzheimer’s disease, LOAD\nomenclature{LOAD}{Late Onset Alzheimer's Disease}), 5\% of AD cases arise in much younger patients (termed Early-Onset Alzheimer’s disease, EOAD\nomenclature{EOAD}{Early Onset Alzheimer's Disease}) who typically present a clear familial autosomal dominant pattern of inheritance (Familial Alzheimer’s disease, FAD\nomenclature{FAD}{Familial's Alzheimer's Disease})\cite{Jarmolowicz2015}. To date, more than 160 highly-penetrant, causative mutation have been identified in EOAD, all located within three genes involved in amyloid plaque formation: \textit{APP} (amyloid precursor protein\nomenclature{APP}{Amyloid Precursor Protein}), \textit{PSEN1} and \textit{PSEN2} (presenilin 1 and 2\nomenclature{PSEN1}{Presenilin 1}\nomenclature{PSEN2}{Presenilin 2}) \cite{LM2010,Chai2007}. While the clinical manifestations and presentation of neurological hallmarks are similar between EOAD and LOAD, patients with EOAD performed significantly worse in cognitive abilities not involved with memory (such as executive functions, language and visuoconstructional abilities)\cite{Joubert2016}.

Unlike FAD, LOAD does not follow a clear Mendelian inheritance pattern but is rather complex with a heterogeneous genetic composition. Nonetheless, twin studies have reported a relatively high heritability rate of 50-80\% \cite{Gatz2006}, indicating that there is still a large genetic predisposition for developing AD. Indeed over recent years, the emergence of genome-wide association studies (GWAS\nomenclature{GWAS}{Genome-wide association studies}) and subsequent meta-analyses \cite{Bellenguez2020,Naj2020,Kunkle2019,Jansen2019,Lambert2013,Naj2011,Hollingworth2011,Harold2009,Lambert2009,Bertram2008} have facilitated the identification of multiple genetic loci associated with an increased risk of developing LOAD. These genetic loci are typically changes or variants of single DNA base-pair (single-nucleotide polymorphisms – SNPs\nomenclature{SNP}{Single Nucleotide Polymorphism}) that are more commonly found in individuals with LOAD than without. 

The strongest risk factor for LOAD is the $\epsilon$4 allele of the \textit{APOE} gene \cite{Lambert2013}, which encodes for the cholesterol transporter apolipoprotein E (ApoE) - a key regulator of lipid homeostasis by mediating lipid transport between astrocytes and neurons, a process critical for synaptic function and maintenance\cite{DH2001}. While the $\epsilon$4 allele is estimated to occur in \textasciitilde 15\% of the general population, it is observed in 40\% of patients with AD\cite{Farrer1997,alzgene2010}. Harbouring one APOE$\epsilon$4 allele further increases the risk of developing LOAD by three-four times, while harbouring two $\epsilon$4 alleles increases the risk by 15 times \cite{Farrer1997}. Conversely, the $\epsilon$2 allele confers a neuroprotective effect against AD \cite{Nagy1995,EH1994}. 

The most recent GWAS meta-analysis of 111,326 AD cases and 677,336 control individuals identified 42 significant LOAD risk loci \cite{Bellenguez2020}, many of which were annotated to the non-coding cis regulatory regions of gene (Lambert et al., 2013). These genes included BIN1 (bridging integrator 1\nomenclature{BIN1}{Bridging Integrator }), CLU (clusterin)\nomenclature{CLU}{Clusterin}, CR1 (complement receptor 1\nomenclature{CR1}{Complement Receptor 1}), PICALM (phosphatidylinositol binding clathrin assembly protein\nomenclature{PICALM}{Phosphatidylinositol Binding Clathrin Assembly Protein}), with the most significant genetic locus annotated to APOE (apolipoprotein E\nomenclature{APOE}{Apolipoprotein E}); inheritance of both APOE allele increases the risk of AD development by X\%. Many of these candidate genes fell into established biological pathways\cite{Verheijen2018}: amyloid and tau metabolism, immune response, lipid metabolism, endocytosis, and cell adhesion molecule (CAM) pathways. 

Collectively, these common but low penetrant variants, with the exception of APOE, contribute modestly to the risk of developing AD, highlighting the polygenic nature of AD. The mechanisms behind these variants currently remain poorly understood, however they typically fall into three main biological pathways that may play an important role: the immune system and inflammatory responses, cholesterol and lipid metabolism, and endosomal vesicle recycling. Comprehensive case-control examination of genes proximal to these LOAD-associated variants have further revealed significant differential changes in gene expression and splicing (Humphries et al. 2015), implicating the role of transcriptomic dysregulation in AD pathogenesis. The very fact that most variants lie within the introns rather than exons suggest that it is the fine tune balance of gene expression and regulation that is at play, emphasising the importance epigenomic and transcriptomic studies. 

% AD-associated genes are specifically enriched in active enhancers of macrophages and microglia (https://www.nature.com/articles/s41467-021-21823-y)
%https://academic.oup.com/hmg/article/29/17/2899/5892988
%https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-021-00890-2
%https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-018-0551-4 - Recently there has been a growing interest in understanding the transcriptomic changes attributed to AD [18,19,20,21,22,23,24,25], 
%https://www.nature.com/articles/s41586-018-0023-4#Sec5
%https://www.nature.com/articles/s41586-019-0924-x#disqus_thread
%https://www.nature.com/articles/s41586-019-1195-2
%https://www.biorxiv.org/content/10.1101/694281v2
%https://pubmed.ncbi.nlm.nih.gov/31503409/
%https://science.sciencemag.org/content/369/6509/eaaz8528
%https://www.nature.com/articles/s41588-020-00721-x?utm_source=ng_etoc&utm_medium=email&utm_campaign=toc_41588_52_11&utm_content=20201031&sap-outbound-id=0374E51F3312B92ED9509B8823EEE3DF0AE64014

\clearpage
\subsection{Hypothesis surrounding AD aetiology}
Despite the well characterisation of AD neuropathology, the exact biological mechanisms driving AD onset and pathogenesis are still widely unknown. To date, there are two key hypothesis proposed for the cause of AD: i) amyloid cascade hypothesis, ii) tau tangle hypothesis. Results from GWAS, however, implicated other pathways that could be involved: immune response, lipid metabolism, endocytosis, and cell-adhesion molecule (CAM) pathways (synaptic signalling), which are entangled with the two key hypothesis. 

\boldheader{Amyloid cascade hypothesis} 
The amyloid cascade hypothesis posits that the extracellular accumulation of A$\beta$ is the key driver of AD pathogenesis, which initiates a pathological cascade of NFT, cell loss, vascular damage \cite{Hardy1992}. A$\beta$ are short peptides (39-43 amino acids) \cite{J1987} produced from the amyloidogenic cleavage of APP (a transmembrane protein involved in synapse formation and stability) by $\beta$-secretase (BACE - β-site APP-cleaving enzyme 1\nomenclature{BACE}{Beta-secretase}) and $\gamma$-secretase (a complex protein consisting of PSEN1 and PSEN2) (\cref{fig:APP_Processing}). Due to cleavage at various sites, $\gamma$-secretase produces A$\beta$ of varying lengths, with 90\% secreted as A$\beta$\textsubscript{40} and the remaining 10\% as A$\beta$\textsubscript{42}\cite{Asami-Odaka1995}.

In AD, the processing of APP is altered with the vast majority of causative \textit{APP} mutations (typically found at or near the cleavage sites),\textit{PSEN1} and \textit{PSEN2} mutations favouring the production of the longer and more self-aggregating A$\beta$42 \cite{Li2019,D1996,JT1993}, thereby promoting the formation of insoluble fibrils and plaques characteristic of AD\cite{JT1993}. 

%https://www.nature.com/articles/nrn2620

%Although interestingly, the neuroprotective ApoE$\epsilon$2 allele while associated with intact cognition is also associated with AD pathology in the oldest old population\cite{DJ2009}, suggesting that an A$\beta$-independent mechanism may be at play.   

%"TREM2 mediated phagocytosis is critical for Aβ and neuronal debris clearance in AD (Kleinberger et al., 2014; Xiang et al., 2016; Yeh et al., 2016). Specifically, TREM2 expression is important for microglia to physically associate with Aβ plaques (Ulrich et al., 2014; Wang et al., 2016; Yuan et al., 2016; Jay et al., 2015, 2017a,b)." 

%"CD33 is elevated in the AD brain in microglia and infiltrating macrophages and is thought to modulate microglial activation and Aβ clearance (Griciuc et al., 2013; Walker et al., 2015).  In fact, knock-out of CD33 in AD mouse models results in reduced Aβ plaque burden (Griciuc et al., 2013). 

\boldheader{Tau tangle hypothesis} 
%https://pubmed.ncbi.nlm.nih.gov/33848474/
The tau hypothesis posits that the phosphorylation and aggregation of tau to form NFTs are the primary drivers of AD\cite{KS1986}, which is also the defining feature of more than 20 other neurodegenerative disorders known collectively as tauopathies\cite{Orr2017}. Tau, encoded by \textit{MAPT}, is a microtubule-associated protein involved in microtubule maintenance and stability. It is thought that in AD, the increased phosphorylation of tau induces detachment from microtubule and promote aggregation of paired helical filaments that disrupt microtubule function and subsequent axonal growth and transport. The severity of NFTs has shown to correlate better with cognitive decline and progression of AD than amyloid plaques \cite{Serrano-Pozo2016,Giannakopoulos2003,PV1992} with its spread traced through Braak staging \cite{H1991}(\cref{fig:AD_development}\textbf{b}). 

%Regional variation in MAPT mRNA and protein expression was observed with a 2-fold higher level in the neocortex than the cerebellum, any may explain regional vulnerability of brain regions to tau pathology\cite{Trabzuni2012}. 

\vspace{1cm}
\begin{figure}[!htp]
	\centering
	\includegraphics[page=2,trim={0 9cm 0cm 15cmcm},clip, scale = 0.8]{Introduction_Figures.pdf}
	\captionsetup{width=0.95\textwidth,singlelinecheck=off}
	\caption[Sequential cleavage of APP into A$\beta$ by $\beta$-secretase and $\gamma$ secretase]%
	{\textbf{Sequential cleavage of APP into A$\beta$ by $\beta$-secretase and $\gamma$ secretase}: Schema depicting sequential cleavage of APP, a transmembrane protein, either through the \textbf{a)} non-amyloidogenic pathway or \textbf{b)} the amyloidogenic pathway.
	\\
	\\
	In the non-amyloidogenic pathway, APP is cleaved by ADAM protein family (primarily, ADAM10, also known as $\alpha$-secretases) followed by $\beta$-secretase. Conversely in the amyloidogenic pathway, APP is sequentially cleaved by by $\beta$-secretase and $\gamma$-secretase, which can produce A$\beta$ of varying lengths. Monomeric A$\beta$ molecules, particularly, A$\beta$42, have increased propensity to oligomerise and aggregate to form the fibrils and plaques that are characteristic of AD. Figure is taken from Acker et. al (2019)\cite{Acker2019}. 
	}
	\label{fig:APP_Processing}
\end{figure}
    
\boldheader{Endocytosis} 
Endocytic processing (internalisation of substrates into the cell) is directly implicated in AD due to distinct cellular localisation of secretases involved in the amyloidogenic processing of APP\cite{Acker2019}. Contrary to the non-amyloidogenic pathway that predominantly occurs at the plasma membrane\cite{Sisodia1992}, amyloidogenic processing of APP takes place in the endosome and is spatially regulated: BACE1 and PSEN1/$\gamma$ complex are localised at the plasma membrane and thus must first undergo endocytosis before assemblage with PSEN2/$\gamma$ secretase at the endosome (\cref{fig:APP_Trafficking}). Increasing evidence suggest that regulation of this endocytic pathway is altered in AD, creating an intracellular pool of A$\beta$ peptides \cite{Peric2015}; pathological significance of these intraneuronal peptides have been corroborated in AD mouse models with their appearance coinciding with cognitive deterioration in AD mouse models \cite{Tomiyama2010,Knobloch2007,Billings2005} and with stronger association to neuronal loss than A$\beta$ plaques\cite{Christensen2008}. Indeed, several risk genes emerging from recent GWAS are directly involved in endocytic regulation of APP processing, including: i) \textit{Bin1}, ii) \textit{Picalm}, and iii) \textit{Sorl1}, which encodes for SORLA (encoded by \textit{SORL1}), an APP-binding receptor involved in APP trafficking away from the late endosome for amyloidogenic processing and facitating A$\beta$ degradation in lysosome \cite{Schmidt2016,Dumanis2015}.   	


\begin{figure}[!htp]
	\centering
	\includegraphics[page=6,trim={0 8cm 0cm 0cm},clip, scale = 0.8]{Introduction_Figures.pdf}
	\captionsetup{width=0.95\textwidth,singlelinecheck=off}
	\caption[Spatial regulation of APP trafficking and processing]%
	{\textbf{Spatial regulation of APP processing}: Schema depicting APP trafficking and processing through the non-amyloidogenic (\cref{fig:APP_Processing}\textbf{a}), which predominantly occurs at the plasma membrane (boxed green), and amyloidogenic pathway (\cref{fig:APP_Processing}\textbf{b}), which preferentially occurs in the endosome (boxed red). APP processing through the amyloidgenic pathway is spatially regulated by the localisation and distinct internalisation of assembled PSEN1/$\gamma$ complex and BACE1 at the plasma membrane (boxed purple) and PSEN2/$\gamma$ in the endosome. Figure is adapted from Acker et. al (2019)\cite{Acker2019}  
	}
	\label{fig:APP_Trafficking}
\end{figure}

 

\boldheader{Immune Response}
Profound neuroinflammation, an inflammatory response within the CNS primarily orchestrated by the activation of microglia (microgliosis) and astrocytes (astrogliosis) with heightened release of cytokines, has been widely implicated in AD development pathology \cite{Cisbani2021,Griciuc2021}. While the role of the immune response is poorly understood in AD, it is widely accepted that an imbalance of the innate immune response is at play\cite{Frost2019} (\cref{fig:microglia_AD}\textbf{a}). Reactive microglia has been found surrounding amyloid plaques\cite{PL1987}, suggesting that plaque-associated microglia have a compromised phagocytic ability to remove A$\beta$\cite{Mawuenyega2010} - a complex process that involves recogition of toxic species (in this case, detrimental protein aggregates) by receptors, such as TREM2, CD33 and CR1 (GWAS AD risk genes) (\cref{fig:microglia_AD}\textbf{b}). Furthermore, reactive microglia can also release pro-inflammatory neurotoxic cytokines that can trigger neuronal apoptosis\cite{Qin2002,Wang2015b} and upregulate β-secretase\cite{Chen2012}, resulting in enhanced A$\beta$ propagation (\cref{fig:microglia_AD}\textbf{b}).  

Multiple recent transcriptomic profiling studies of single cells in AD human post-mortem brain tissues\cite{Mathys2019,Nott2019,Thrupp2020,Olah2020,Leng2021,Young2021} and mouse models\cite{Keren-Shaul2017,Mathys2017} have revealed subpopulations of microglia and astrocytes that have an altered molecular expression signature associated with disease progression; distinct AD-associated microglia cell states were detected with initial observed microglial proliferation accompanied with release of pro-inflammatory cytokiens and later with increased expression of interferon-response genes\cite{Mathys2017}. Similar microglial cell states (termed disease-associated microglia - DAM\cite{Keren-Shaul2017}, and activated response microglia - ARM\cite{Frigerio2019}) have been characteised in mouse models with upegulated expression in innate immune response and inteferon pathways. Moreover, these AD-associated microglia subpopulations were enriched with altered expressions of \textit{Trem2} and \textit{Cd33} \cite{Mathys2019,Frigerio2019}. A subset of AD-associated astrocytes, likely to represent reactive astrocytes, has also been characterised with upgregulated expression of \textit{Gfap} and \textit{Cd44}, and downregulation of genes associated with homeostasis\cite{Leng2021}. Notably, the largest proteomic study, to date, has identified that the protein co-expression module most robustly associated with AD and FTD was associated in astrocyte and microglial-associated proteins, and was significantly enriched in AD GWAS genes and protective markers of anti-inflammatory disease-associated microglia \cite{Johnson2020}.    

%Role of adaptive immune response with recent identification of a subpopulation of immune cells, CD8+ T effector memory CD45RA+ (TEMRA), associated with AD pathology and exhibited stronger antigenic stimulation (increase in cytokine signalling) \cite{Gate2020}. In addition to the stronger-associated AD risk variants, previous studies have identified additional rare coding variants in genes involved in the immune response\cite{R2017,Bis2018} and transcriptional regulation\cite{Bis2018}.

\begin{figure}[!htp]
	\centering
	\includegraphics[page=8,trim={0 8cm 0cm 0cm},clip, scale = 0.8]{Introduction_Figures.pdf}
	\captionsetup{width=0.95\textwidth,singlelinecheck=off}
	\caption[Role of microglia in AD development and pathology]%
	{\textbf{Role of microglia in AD development and pathology}: A simplified schema illustrating the \textbf{a)} multifaceted roles of microglia in AD, ranging from a protective to a detrimental role by the respective secretion of anti- and pro-inflammatory cytokines, and \textbf{b)} microglia's dual response to A$\beta$ plaques, either through A$\beta$ clearance or release of pro-inflammatory cytokines. Recent emergence of single RNA-sequencing studies have revealed significant heterogeneity in microglia isolated from AD post-mortem brain tissues, highlighting the complex role that microglia plays in AD development and pathology. Under physiological conditions, the microglia is ramified. PRR - Pattern recongition receptors, such as TREM2 and CD33, are found on the cell surface of microglia and are involved in recognising toxic species for phagocytosis. Both figures are adapted from Leng et al. (2021)\cite{Leng2021a}  
	}
	\label{fig:microglia_AD}
\end{figure}


\boldheader{Lipid Metabolism}
Identification of \textit{Apoe} $\epsilon$4 variant as the strongest genetic variant of LOAD established a link between lipid metabolism and AD. Increasing evidence further postulates that A$\beta$ clearance is regulated in an APOE isoform-dependent manner\cite{Castellano2011} with APOE4 having low binding affinity to A$\beta$ and thus, being the least efficient at A$\beta$ clearance\cite{RM2012} - carriers of $\epsilon$4 allele have more pervasive amyloid plaques than non-carriers\cite{DE1993,E2009}. APOE lipidation status, mediated by ABCA1\cite{R2010}, has also been reported to impact A$\beta$ aggregation, with APOE4 being poorly lipidated compared to APOE2 and APOE3, resulting in increased propensity to aggregate\cite{DM2006}. 

More broadly, lipid metabolism and homeostasis are linked to AD development in that APP, $\beta$- and $\gamma$-secretase are transmembrane proteins (\cref{fig:APP_Trafficking}); lipid membrane constitution and organisation would therefore have an impact on the APP trafficking and secretase activities \cite{DiPaolo2011}. \textit{ABCA7}, by regulating the lipid composition at bilayer, can indirectly influece the activity and expression of $\beta$-secretase \cite{Sierksma2020,Sakae2016}.  

%Link between Trem2 and Apoe, whereby lack of Trem2 resulted in plaque-associated APOE protein only in APOE4 mice but not APOE3 mice (https://pubmed.ncbi.nlm.nih.gov/32703241/); E4 expressing microglia had less active trascriptional response and reduced A$\beta$ uptake compared to E3-expressing microglia.  (https://www.nature.com/articles/s41467-021-23762-0)

CLU 
%https://science.sciencemag.org/content/370/6512/61?rss=1
%https://www.nature.com/articles/s43587-021-00073-z
%https://www.nature.com/articles/s41467-021-23762-0

\boldheader{Synpatic signalling} 

\pagebreak
\section{Transcriptomic profiling}
Transcriptomic landscape profiling of disease-relevant tissues, in taking a snapshot of the present state of the cells, is key to elucidating the functional relationship between the genetic variants identified from GWAS and the molecular mechanisms that drive disease development and pathology in a time- and tissue-dependent manner \cite{Verheijen2018}.  %https://pubmed.ncbi.nlm.nih.gov/31626773/
	
\subsection{Alternative Splicing}
Alternative splicing is a transcriptional regulatory mechanism that produces distinct transcripts (isoforms) from a single gene, which are subsequently translated to different protein isoforms with unique, and potentially, antagonistic functions\cite{Wang2008}. A widespread phenomen with over 95\% of human genes estimated to be influenced \cite{Pan2008}, it occurs most prevalently in the brain, where it impacts upon neuronal development and maintenance\cite{Pan2008, Mazin2014, Raj2015}. There is a growing recognition of the key role of aberrant mis-splicing in neurodegenerative diseases \cite{Gandal2018,RL2019}, including schizophrenia and autism. 

\boldheader{Mechanism}
Splicing involves the removal of non-coding sequences (introns) from mRNA precursors and ligation of coding sequences (exons), resulting in isoforms with different exonic structure (\cref{fig:AS_events}). This relies on a concerted and regulated assembly of the spliceosome (a multimegaton, dynamic ribonucleoprotein complex) by its recognition and stepwise-binding to sequence elements within the pre-mRNA (cis-elements), and a group of regulating splicing factor proteins (trans-elements) (\cref{fig:AS_mechanism}). Recent studies suggest that this process occurs co-transcriptionally, such that the intron can be identified and removed as soon as it is synthesised by the RNA polymerase. 
%There are two types of spliceosome - major and minor - both of which involve the activity of five uridine-rich small nuclear ribonucleoproteins (snRNP\nomenclature{snRNPs}{Small Nuclear Ribonucleoproteins}) and numerous non-snRNP proteins\cite{Will2011}. Using a similar mechanism but composed of different snRNPs, the minor spliceosome removes less than 1\% (0.4\%) of introns\cite{Turunen2013} and is thus referred to as "U12-dependent non-canonical splicing", as opposed to "U2-dependent canonical splicing" with major spliceosome.

Correct splicing first requires recognition of short sequence motifs upstream (5' splice site, 5'SS\nomenclature{5'SS}{5' Splice Site}, donor site) and downstream (3' splice site, 3'SS\nomenclature{3'SS}{3' Splice Site}, acceptor site) of the intron/exon boundary, followed by sequential assembly of the spliceosome components and intron excision\cite{Herzel2017}. The 5'SS is typically defined by a conserved 9-nucleotide sequence with a GU(T) dinucleotide, and the 3'SS by a polypyrimidine tract (PPT\nomenclature{PPT}{Polypyrimidine Tract}) followed by a conserved AG dinucleotide \cite{Will2011}. Almost all introns in human and mouse are flanked by the GT-AG splice site dinucleotides\cite{Sheth2006} (termed splice junctions), with other dinucleotide variations known to exist in very minute proportions: GC-AG and AT-AC comprises \textasciitilde0.9 and \textasciitilde0.09\% of human splice sites\cite{Parada2014}. An increasing number of disease are linked to abberant alternative splicing by pathogenic variants by interferring cis-acting elements (5'SS and 3'SS, resulting in exon skipping, exon inclusion, exon extension or exonic splice gain; intron resulting in intron gain) and action of trans-acting protein splicing factors.

%Alternative splicing is highly-regulated in a temporal and cell-specific manner by the binding and fin-tune balance of trans-acting factors to cis-acting elements. The sequence of these cis-acting elements within the exon (exonic splicing - ES) or intron (intronic splicing - IS) determines the binding affinity of the trans-acting factors to either enhance or suppress splicing. RNA structure also plays a role in blocking or allowing trans-acting factors to bind, and transcription factor assemblage through distal interactions. 
%Trans-acting factors include RNA-binding proteins such as SR protein (serine and arginine-rich proteins), heterogeneous nuclear ribonucleoproteins (hnRNPs, and other tissue-specific proteins and complementary microRNA (miRNAs); Cis-elements include exonic splicing enhancers and silencers ( ESEs, ESSs), intronic splicing enhancers and silencers (ISEs, ISSs). Localisation and function of trans-factors are further regulated by post-translational modifications, adding to the layer of complexity through activation/suppression from other proteins involves post-translational modifications i.e. kinases and phosphatases. 


\vspace{1cm}
\begin{figure}[!htp]
	\centering
	\includegraphics[page=9,trim={0 14cm 0cm 2cm},clip, scale = 0.7]{Introduction_Figures.pdf}
	\captionsetup{width=0.95\textwidth,singlelinecheck=off}
	\caption[Alternative Splicing Events]%
	{\textbf{Alternative Splicing Events}: Splicing involves removal of introns and ligation of exons either constitutively or alternatively to generate isoforms with different exonic structure: an exon can be entirely skipped (orange exon in SE) or two exons that are not spliced together (orange and blue exon in MX), an intronic sequence can be retained (light orange in IR), a different 5'SS or 3'SS can be used resulting in a novel splice junction (A5', A3'), and the usage of different first and last exons which can result in an alternative promoter and termination.  
	}
	\label{fig:AS_events}
\end{figure}


\begin{figure}[!htp]
	\centering
	\includegraphics[page=3,trim={0 14cm 8cm 2cm},clip, scale = 0.8]{Introduction_Figures.pdf}
	\captionsetup{width=0.95\textwidth,singlelinecheck=off}
	\caption[Splicing Mechanism: spliceosome assembly on nascent RNA]%
	{\textbf{Intron removal is catalysed by the assembly and complex rearrangement of spliceosome. a)}:  Consensus sequence of splice sites that demarcate the intron/exon boundary and are essential for recruitment of spliceosomal snRNPs. \textbf{b)} Co-transcriptional assembly of spliceosome with stepwise interaction of spliceosomal snRNPs, with the formation of 
	\begin{enumerate}
	\item E (Early) commitment complex with the identification and binding of U1 snRNP to the 5'SS and branchpoint binding protein (BPP\nomenclature{BPP}{Branchpoinnt Binding Protein} to BPS 
	\item A (Assembly) catalytically-active complex with association of U2 snRNP to the branch site following the dissociation of BPP. The "A" here denotes to the adenosine of BPS
	\item B pre-catalytic spliceosome complex with recruitment of U4,U5 and U6 snRNPs
	\item Bact pre-catalytic spliceosome complex after major conformational rearrangements within the spliceosome (RNA-protein and RNA-RNA interactions) followed by the release of U1 and U4 snRNP to expose the adenosine from BP to the 5'SS 
	\item B* catalytically-active complex with nucleophilic attack of adenosine on 5'SS (first step of transesterification) 
	\item C catalytically-active complex with further conformational changes in the U2 snRNA to C* complex, with nucleophilic attack of the 5'SS to 3'SS (second step of transesterification) 
	\item P (Post-spliceosome complex). The mRNA product is then released from remaining spliceosome (ILS), now bound to the intron lariat. The snRNPs can then disassociate and be recycled for next cycle of splicing.
	\\
	\end{enumerate} 
	Intron excision is primarily executed by two transesterification reactions.	BPS - Branch point sequence, CTD - carboxyl-terminal domain, ILS - Intron lariat spliceosome, PAS - poly(A) site, SS - Splice site, TSS - Transcription start site,TTS - Transcription termination site. Figure is taken from Herzel et al. 2017\cite{Herzel2017}.
}
	\label{fig:AS_mechanism}
\end{figure}

%\boldheader{Other regulatory mechanisms}
%Of note, splicing is only one of the main ways of gene regulation;pre-transcriptional regulation involving chemical modifications of DNA nucleotides (epigenetics), transcriptional regulation involving the binding of transcription factors and post-transcriptional regulation involving the processing of mRNA transcripts. Splicing works with these mechanisms together (?). 
%AS further regulates gene expression through various mechanisms: non-sense mediated decay, miRNA-mediated mRNA degradation, altered translational efficiency of isoforms. In contrast, alternative polyadenylation regulates RNA transportation, localization, stability, and translation by generating splice isoforms with different cleavage sites. Nonsense mediated decay (NMD) \nomenclature{NMD}{Nonsense Mediated Decay} products are alternatively spliced isoforms that are not translated into proteins, by containing an early stop codon. A premature termination-translation codon highly supportive of NMD is defined by a stop codon within at least 50-55 base pairs upstream of splice junctions.  up to 70\% containing multiple polyadenylation sites and 60\% with two or more promoters from alternative transcription start sites \cite{Carninci2006}. In addition to above five common categories, many other complex types, such as alternative position, i.e., alternative 3' and 5' site (Wang and Brendel, 2006), AS and transcriptional initiation (ASTI) (Nagasaki et al., 2006) alternative first exons (Chen et al., 2007), and composite patterns (Wang and Rio, 2018), can occur.
%Introduction of variable segments within otherwise identical mRNAs, with 80% of this variability observed within open reading frames (ORFs), subsequently contributing significantly to proteome diversity, and 20% falling within untranslated regions that regulates cis-binding effects. (G. S. Wang & Cooper, 2007) Subsequently, influencing mRNA stability, translation efficiency (including miRNA binding sites), and mRNA localisation. One-third of alternative splicing events introduce premature termination codons (PTCs), which cause mRNA degradation by nonsense-mediated decay (NMD) (Lewis, Green, & Brenner, 2003). Therefore, alternative splicing regulates temporal and spatial expression of functionally diverse isoforms, on–off regulation by NMD, or other post-transcriptional regulatory responses; also for cell identity, whereby number of tissue-specific isoforms highest in the brain. 

\newpage
\subsection{Role of Alternative Splicing in AD}
Dysregulation of splicing can have significant functional consequences in driving or contributing to disease progression and susceptibility, by disrupting protein isoform function (loss-of-function or gain-of-function) or generating an inbalanced isoform ratio (change in relative isoform expression). Studies on the role of alternative splicing in AD have largely focused on selected FAD gene, such as \textit{PSEN1, APP and MAPT}; mutation in intron 4 of PSEN1 produced aberrantly-spliced truncated PSEN1 isoforms that were found to increase A$\beta$42 \textit{in vitro}\cite{DeJonghe1999}. Perhaps more well known is the altered splicing of MAPT whereby exclusion or inclusion of exon 10 (E10) generate isoforms with either 3 (3R tau, E10-) or 4 (4R tau, E10+) microtubule-binding repeat domains, with the latter having a greater interaction with microtubules. Over 50 tauopathy-associated intronic mutations were found clustered around the 5'splice site of exon 10 in favouring inclusion of exon 10 \cite{DSouza1999, Ghetti2015}, resulting in increased production of 4Rtau and imbalanced 4R/3R ratio that contribute to tau aggregation \cite{Adams2010} - mimicking this imbalanced ratio in a tauopathy mouse model induced more seizures, and more phosphorylated,self-aggregating tau \cite{Schoch2016}.  
%tau-isoform mediated neurodegeneration   
% efforts to better understand gene expression changes in AD by sequencing transcriptome of affected brain regions


%(Trabzuni et al 2012)(Rockentein et al 1995)(Buee et al 2000)(Valenca et al. 2016) studies on AS

Co-aggregation of multiple spliceosomal components (snRNP associated with spliceosomal complex) with tau in NFTs in human post-mortem AD brain tissues implicates that the core splicing machiney is atltered with a global disruption of splicing in AD pathogenesis\cite{Bai2013}. More recently, the most characterised transcriptomc profiling of AD post-mortem brain tissues\cite{Raj2018} to date revealed abberant splicing as a hallmark of AD. Furtermore, the study identified AD-associated splice variants (splicing quantiative trait loci - sQTL) were enriched in transcriptionally active regions with overlap of AD-associated epigenetic variants (DNA methylation - mQTL, histone acetylation - hQTL), and AD-associated SNP. 
highlighting region-specific deterioration associated with AD\cite{Mills2013}
Evidence of upregulated levels of intron-retained transcripts in AD vs control, including in genes previous implicated in AD (Bin1,Picalm)\cite{Li2021}


\begin{changemargin}{1.5cm}
%\captionsetup{width=30cm}
\begin{landscape}
	\small %smaller font
	\setlength\tabcolsep{2pt} %reduced margin size in table
	\renewcommand{\arraystretch}{1}
	\begin{longtable}[c]{p{3cm}p{4cm}p{3cm}p{16cm}}
		\caption[Transcriptomic and Proteomic Studies in AD]%		
		{\textbf{Transcriptomic and proteomic studies in AD human post-mortem brain tissues in revealing missplicing as a widespead hallmark of AD}. \textsuperscript{a}RNA-Seq data from ROSMAP dataset\cite{DeJager2018}. AS - Alternative Splicing, NMD - Nonsense Mediated Decay, RBP - RNA binding protein, TSS - Transcription Start Site}
		\label{tab: AS_ADHuman_studies}\\
		
		\toprule
		\multicolumn{1}{c}{References} &
		\multicolumn{1}{c}{Samples and Tissue} &
		\multicolumn{1}{c}{Method} &
		\multicolumn{1}{c}{Implications} \\* \midrule
		\endfirsthead
		%
		\endhead
		%
		\bottomrule
		\endfoot
		%
		\endlastfoot
		%		
		\centering Twine et al. (2011)\cite{Twine2011} &
		\centering 3 AD, 33 Controls\newline Total, frontal \& temporal lobe &
		\centering RNA-Seq &
		\tabitem \textit{Apoe} downregulated in AD temporal lobe and 3 isoforms with different TSS detected. Differential isoform expression detected: ENST00000252486 and ENST00000446996 from TSSA were downregulated (3.09-fold) whereas ENST00000425718 from TSSB was upregulated (26.5-fold) \newline
		\tabitem \textit{Ank1} downregulated in AD total brain \\
		\hdashline[0.5pt/5pt]
		
		\centering Mills et al. (2013)\cite{Mills2013} &
		\centering 5 AD, 5 Controls \newline Parietal lobe &
		\centering RNA-Seq &
		\tabitem Differentially expressed genes enriched in lipid metabolism (\textit{ACOT1}, \textit{ACOT2} and  \textit{DBI/ACBP} upregulated, \textit{TECR} downregulated) \newline
		\tabitem Differential isoform expression observed in \textit{DBI/ACBP}; non-coding DBI-009 upregulated while protein-coding DBI-003 is downregulated. \\
		\hdashline[0.5pt/5pt]
		
		\centering Bai et al. \newline(2013)\cite{Bai2013} &
		\centering 18 AD, 17 Controls \newline Cortex &
		\centering Mass Spectrometry, RNA-Seq &
		\tabitem Extranuclear aggregation of 36 proteins, including spliceosomal components (U1 snRNP, U1-70K) \newline
		\tabitem Accumulation of unspliced RNA molecules in AD, with reduced splicing efficiency (increased ratio of pre- and mature RNA) in AD-associated genes (\textit{BACE1, BIN1, CLU, GFAP, PICALM, PSEN1, SORL1})  \\
		\hdashline[0.5pt/5pt]
	
		\centering  Lai et al. \newline(2014)\cite{Lai2014} &
		\centering 8 AD, 8 Controls\newline Superior Temporal Gyrus &
		\centering Microarray &
		\tabitem 22 genes identified with differential AS event (characterised by differential exon usage) \newline  \tabitem \textit{GNAL} transcript variant 5 downregulated in AD whereas transcript variant 1 showed no change \newline  \tabitem \textit{MAP4} transcript variant 3 downregulated in AD whereas transcript variant 1 was upregulated\\
		\hdashline[0.5pt/5pt]

		\centering Mills et al. (2014)\cite{Mills2014} &
		\centering 14 AD, 16 Controls \newline Superior Temporal Gyrus &
		\centering RT-qPCR &
		No difference in total \textit{APOE}, \textit{APOE-005}, or \textit{APOE-001} between AD superior temporal gyrus and controls, contrary to Twine et al. (2011)\cite{Twine2011} \\
		
		\centering Humphries et al.(2015)\cite{Humphries2015} &
		\centering 10 AD, 10 Controls, \newline Temporal lobe &
		\centering RNA-Seq &
		9 genes differentially expressed (\textit{ABCA7, CR1, MS4A14, MS46E, PTK2B}), 5 of which had differential splicing (\textit{ABCA7, TMEM259, EPHA1, MS4A6A, MS4A6E}) defined by overall exon distribution differences\\
		\hdashline[0.5pt/5pt]
				
		\centering Magistri et al.(2015)\cite{Magistri2015} &
		\centering 4 AD,4 Controls \newline Hippocampus &
		\centering RNA-Seq &
		\tabitem Downregulation of \textit{TAC1} and upregulation of \textit{SERPINE1} \newline 
		\tabitem Pathway analysis indicate dysregulation in neural communication and A$\beta$ clearance \\		
		\hdashline[0.5pt/5pt]
		
		\centering Alkallas et al. (2017)\cite{Alkallas2017} &
		\centering 6 AD, 5 Controls \newline Dorsolateral Cortex &
		\centering RNA-Seq &
		RBFOX1 (RBP) downregulated in AD; hence, reduced stability and abundance of RBFOX-regulated transcripts encoding for synaptic tramission proteins, contributing to loss of synaptic function	\\
		\hdashline[0.5pt/5pt]

		\centering Annese et al. (2018)\cite{Annese2018} &
		\centering 6 AD, 6 Controls \newline Hippocampus, Temporal and Frontal lobe &
		\centering RNA-Seq &
		\tabitem 2,122 differentially expressed genes, including upregulation of \textit{TESPA1, CPLX3 SERPINA5, SERPINA1} and dowregulation of \textit{NEUROD6, NEUROD1, LOC400891, CAMK1D} \newline
		\tabitem Deregulated micro-RNA (miR-132/212) and general decrease in RNA editing in AD \\
		\hdashline[0.5pt/5pt]
				
		\centering Raj et al. \newline (2018)\cite{Raj2018} &
		\centering 268 AD, 182 Controls \newline Dorsolateral prefrontal cortex &
		\centering RNA-Seq &
		\tabitem 84 differentially spliced (differential intron usage) genes, 11 of which also differentially expressed (\textit{PFKP, NDRG, APP, PICALM, CLU}) \newline
		\tabitem sQTLs enriched in RBPs, including \textit{PTBP1, ELAVL1} and mutliple hnRNP \newline
		\tabitem TWAS identified 21 genes with differential intron usage associated with AD, including \textit{CR1,PTK2B, CLU, AP2A1, AP2A2, MAP1B} \\
		\hdashline[0.5pt/5pt]
		
		\centering Johnson et al. \newline (2018)\cite{Johnson2018} &
		\centering 20 AD, 13 Controls \newline Dorsolateral prefrontal cortex &
		\centering Mass-Spectrometry &
		\tabitem More alternative exon-exon junction peptides mapped to \textit{MAPT, BIN1, PTK2B, FERMT2} in AD. \newline 
		\tabitem Higher RBPs protein levels in AD, with enrichment in modules correlated with tau pathology \\
		\hdashline[0.5pt/5pt]	
				
		\centering Han et al. (2019) \cite{Han2019} &
		\centering 24 AD, 50 Controls \newline Hippocampus &
		\centering RNA-Seq &
		3 AD-associated exon skipping events in \textit{RELN} \& \textit{NOS1}, resulting in truncated protein with loss of functional domains. Identified SNP adjacent to \textit{RELN's} skipped exon, within splicing regulatory element. \\
		\hdashline[0.5pt/5pt]
			
		\centering Adusumalli et al. (2019) \cite{Adusumalli2019} &
		\centering 42AD, 38 Controls\cite{Bai2013} \newline Frontal cortex &
		\centering Mass Spectrometry &
		\tabitem 1,136 differential intron-retention events at 781 genes (including \textit{BIN1, MAPT}), enriched in mRNA export and splicing, and had signficantly different protein level between AD and controls.
		\tabitem Differentially retained introns have higher GC content, indicating DNA methlation changes may contribute to differential IR \\
		\hdashline[0.5pt/5pt]
		
		
		\centering Garcia-Escudero et al. (2021) \cite{Garcia-Escudero2021} &
		\centering 32 AD (Braak I-VI), 10 Control &
		\centering qPCR, Western Blot &
		Identified novel human-specific truncated Tau isoform with intron 12 retention, which is downregulated in AD and is less prone to aggregate compared to other tau isoforms \\
		\hdashline[0.5pt/5pt]
		
		\centering Fan et al. (2021) \cite{Fan2021} &
		\centering 210 AD, 191 Controls\textsuperscript{a} &
		\centering RNA-Seq &
		\tabitem 2 isoform modules with 38 isoforms upregulated in AD, of which 33 has not been reported as AD-related (including \textit{ANLN, DOCK5, ERBB3, SEPT8, UGT8}). 67 genes identified with differentially expressed isoforms in different modules  \\
		\hdashline[0.5pt/5pt]
		
		\centering Yang et al. 2021 \cite{Yang2021} &
		\centering 1074 AD, 608 Controls \newline 9 brain regions&
		\centering RNA-Seq & 
		\tabitem 1,530 differential ES events in 1,103 genes enriched in endocytosis, RAS and ARF signalling; 2,415 differential ES events in 1,701 genes, associated with tau progression, enriched in axon guidance \newline 
		\tabitem \textit{MBP} exon 5 skipping; \textit{ASPH} exon 15 skipping \& exon 5 and 8 skipping in cerebellum \newline  
		\tabitem 15,556 RBP-associated ES events in 113 RBP (\textit{CHL1} exon 25, \textit{ASPH} exon 5); \textit{RBM3, RBPMS2, AZGP1, RPS16} differentially expressed in STG; SNP in \textit{ABCA7} donor site associated with exon2 skipping \newline 
		\tabitem 70\% of genes predicted LOF due to ES, with most significant loss attributed to peptidy-serine phopshorylation; 86 AD genes with partial function loss were enriched in neuronal development	\\
		
		\centering Li et al. 2021 \cite{Li2021} &
		\centering 84 AD, 80 Controls \newline Temporal cortex &
		\centering RNA-Seq, Mass Spectrometry &
		\tabitem Higher intron-retained levels in AD (including \textit{BACE1, BIN1, PICALM}) but no differential gene expression, suggesting a compensatory mechanism \newline
		\tabitem HMBOX1, a transcription factor involved in innate immune response, had strongest associated differentially expressed intron level with tau pathology \newline
		\tabitem Increased IR was associated with reduced protein expression, possibly due to NMD \\* \bottomrule
	\end{longtable}
\end{landscape}
\end{changemargin}

%Studies using long-read sequencing of human transcriptome have revealed differences in poly(A) length distribution between genes, and even between isoforms of the same gene  with protein-coding isoforms having shorter poly-A tails than intron-retaining isoforms (\cite{Workman2019a}. This is line with studies showing that hyperadenylation targets intron-retaining transcripts for degradation (\cite{Bresson2015})



\subsection{Limitations of short-read RNA sequencing}
\label{rnaseq_intro}
Transcriptomic profiling of AD to study gene and transcript expression changes has been traditionally performed using exon microarrays and more recently, RNA-Sequencing (RNA-Seq\nomenclature{RNA-Seq}{RNA-Sequencing}) (summarised in \cref{tab: AS_ADHuman_studies}), which involves high throughput parallel sequencing of amplified DNA templates in a “sequence-by-synthesis” fashion. Through larger sample sizes and significant advances in bioinformatic tools, RNA-Seq has allowed a more comprehesive annotation of the transcriptome and interrogation of alternative splicing events, particularly exon skipping and intron retention. 

Despite the power of RNA-Seq to identify and quantify expression at a gene level, efforts to characterise isoform diversity and perform transcript-based analysis are constrained by the fact that RNA-Seq approaches generate short reads that cannot span full-length transcripts. Reads generated by RNA-Seq typically have an average length ranging between 100 - 700bp (depending on the sequencing platform), whereas transcripts are on average 2-3kb - 50\% of human transcripts are >2.5Kb\cite{Sharon2013} and range from 60bp to 103kb \cite{Piovesan2016,Sharon2013} - the longest known human processed transcript to date is Titin with 363 exons and spanning 106kb\cite{Bang2001}. Thus, while short-reads are sufficient for gene-based analyses in accurately identifying exons with the associated gene, RNA-Seq fails to capture the connectivity of exons for transcript assembly\cite{Gordon2015}\cite{Wang2016}. 

Various bioinformatic tools have been developed to overcome this challenge of transcript reconstruction by probabilistically assigning short reads to isoforms and exon-exon boundary \cite{Trapnell2010, Kingsford2010, Au2013}. However, this has required complex computational analysis, often resulting in conflicting outcomes and limited success, compounded by the fact that isoforms often have significant overlaps and only a minor proportion of reads span splicing junctions. These tools further rely heavily on reference genome annotation or predefined splicing events, which can be inaccurate and incomplete, result in prediction of transcripts that do not exist (false positives) or failure to detect true transcripts (false negatives), particularly for genes with many isoforms\cite{Au2013}. Pre-defined transcript models are particularly limiting when comparing splicing profiles between different conditions, such as AD vs control, as any splicing changes observed are likely to be AD-specific and novel. 

% Figure for RNA-Seq parallel sequencing?
%(Tardaguila et al., 2018)(Hayer et al., 2015). - check papers 
%redraw figure with panel a) and b)
\begin{figure}[htp]
	\centering
	\includegraphics[page=10,trim={0 22cm 0 0},clip, scale = 0.8]{Introduction_Figures.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Challenges of using short-reads for transcript assembly]%
	{\textbf{Challenges of using short-reads for transcript assembly}: Example of a transcript model that is impossible to resolve using short-reads (yellow). Figure and caption taken from \cite{Kuo2017}}
	\label{fig:rna_seq_limitations}
\end{figure}	

%Attempts to overcome challenges with transcriptome assembly included generation of “synthetic long reads”, by tagging full-length complementary DNAs with unique molecular identifiers (UMIs) before cluster amplification and sequencing on Illumina (Tilgner et al., 2015). With the presence of UMIs, transcript isoforms can be reconstructed for up to 4Kb for isoform discovery and expression analysis (Stark, Grzelak, \& Hadfield, 2019). [However…]

\clearpage
\subsection{Opportunities with long-read RNA sequencing}
Recent advances in long-read sequencing technology addressed the issues with RNA-Seq by the ability to generate significantly longer reads that were able to span the full-length transcript. Rather than sequencing of templates in a “wash-and-scan” fashion that resulted in de-phasing and subsequently shorter reads, long-read sequencing technology capitalised on real-time sequencing of templates in an uninterrupted and processive manner. Pacifib Biosciences's (PacBio) Single Molecule Real Time (SMRT\nomenclature{SMRT}{Single Molecule Real Time}) and Oxford Nanopore Technologies (ONT \nomenclature{ONT}{Oxford Nanopore Technologies}) nanopore sequencing currently dominate this space, with both platforms generating reads >10kb (\textasciitilde15kb for PacBio and >30kb for ONT). Other long-read sequencing methods and protocols, such as synthetic long read\cite{Tilgner2015} and sparse isoform sequencing\cite{Tilgner2018}, however these require more complex workflows.

The consequent generation of longer reads, ranging from 300 – 20,000 bases provided unprecedented ability to sequence entire or new entire lengths of transcripts from 5’ end to polyA tail, relinquishing the need for transcriptome assembly and resolving splicing junctions. Allowing greater accuracy at transcript identification, an increasing number of studies have used such technologies to characterise isoform diversity and splicing with unprecedented success (Table X). Generally in comparison with RNA-Seq, Iso-Seq encapsulates longer transcripts, identifies novel gene locus, and correction of gene model. "Long transcript reads provide better support and higher accuracy in splice junctions than short reads, when these reads are aligned back to the genome. Thus gene models predicted from long reads yield more accurate exon/intron structure and can merge two or more misannotated adjacent genes."

%https://www.nature.com/articles/s41467-019-11049-4
%https://www.proquest.com/docview/2459613726?pq-origsite=gscholar&fromopenview=true
%https://academic.oup.com/brain/article/144/4/1082/6247188?login=true
%https://link.springer.com/article/10.1186/s13059-019-1856-3
%https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7237973/

%https://genome.cshlp.org/content/28/2/231.full
%https://www.nature.com/articles/nbt.4259



Applications of these novel bioinformatic approaches to study alternative splicing in AD human post-mortem brains revealed over 2000 genes with differential transcript usage, including \textit{APP} and \textit{BIN1} \cite{Marques-Coelho2021}.

\clearpage
\section{Modelling AD: Mouse Models}
While profiling human post-mortem brain tissues remains the gold standard for studying AD pathogenesis, there are various confounding secondary factors (environmental exposures such as diet, medication) and technical difficulties (agonal state impacting RNA quality) to consider. In comparison, use of mouse models can be tightly controlled by their living conditions, their age and pathology to focus and follow specific pathology progression. As such, mouse models act as a valuable reductionist tool to dissect and understand the processes that drive AD onset and progression, identify biomarkers and validate novel targets\cite{Hall2012}. 

To study different aspects of pathology, a number of AD mouse models have been developed with mutations that either result in amyloidopathy (formation of A$\beta$ plaques) or tauopathy (NFT) (\cref{tab:mouse_models}). Amloidopathy is typically developed through insertion and overexpression of human \textit{APP}, either alone or in combination with \textit{PSEN1}, whereas tauopathy is recapitulated by overexpressing human \textit{MAPT} with FTD-associated mutations (given no causative \textit{MAPT} mutations have been identified in AD)

\vspace{1cm}
\begin{table}[!h]
	\centering
	\setlength\tabcolsep{3.5pt}
	\captionsetup{width=1\textwidth}
	\caption[Representative AD Mouse Models]%
	{\textbf{Representative AD mouse models}. Tabulated is a list of the most widely used AD mouse models developed from overexpression of one or more genes associated with FAD. Table is adapted from Hall et al. (2012) \cite{Hall2012} and is by no means comprehensive. mo - months}
	\label{tab:mouse_models}
	\begin{threeparttable}
		\begin{tabular}{@{}lllcccc@{}}
			\toprule
			\multicolumn{2}{c}{Mouse Models} &
			\multicolumn{1}{c}{Mutations} &
			\begin{tabular}[c]{@{}c@{}}Plaques\\  (mo)\end{tabular} &
			\begin{tabular}[c]{@{}c@{}}Tangles\\   (mo)\end{tabular} &
			\begin{tabular}[c]{@{}c@{}}Neuronal \\ Loss (mo)\end{tabular} &
			\begin{tabular}[c]{@{}c@{}}Cognitive \\ Deficit (mo)\end{tabular} \\ \midrule
			\multirow{4}{*}{hAPP}     & PDAPP                 & Ind\tnote{b}                        & 6  & x  & x     & 6                \\
			& Tg2576                & Swe\tnote{a}             & 11 & x  & x     & \textgreater{}12 \\
			& J20                   & Swe\tnote{a}, Ind\tnote{b} & 6  & x  & x     & 4                \\
			& APP23                 & Swe\tnote{a}               & 6  & x  & 14-18 & 3                \\
			\multirow{3}{*}{hAPP/PS1} & PS/APP & Swe\tnote{a}, M146L\tnote{e}                 & 6  & x  & 22    & 3                \\
			& APP/PS1         & Swe\tnote{a}, PSEN1dE9              & 6  & x  & x     & 6                \\
			& 5xFAD                 & Swe\tnote{a}, Lon\tnote{a}, Flo\tnote{c}, M146L\tnote{e}, L28V\tnote{e} & 2  & x  & 9     & 4                \\
			\multirow{4}{*}{hTau}     & hTau.P301S            & MAPT P301S                 & x  & 4  & 3     & 3                \\
			& 3xTg                  & Swe\tnote{a}, MAPT P301L, M146V     & 6  & 12 & -    & 4                \\
			& rTg4510               & MAPT P301L                 & x  & 4  & 6     & 3                \\
			& htau                  & Wildtype                   & x  & 9  & 10    & 6                \\ \cmidrule(l){1-7} 
		\end{tabular}
		\begin{tablenotes}
			\footnotesize
			\item[a] Swedish APP mutation K670N/M671L
			\item[b] Indiana APP mutation V717F
			\item[c] London APP mutation V717I
			\item[d] Florida APP mutation I716V 
			\item[e] Human PSEN1 mutations 
		\end{tablenotes}
	\end{threeparttable}
\end{table}

It is important to note that there are currently no mouse model that encapsulates all the defining features of AD and one of the major criticisms of current AD mouse models us how representative they are of sporadic, late-onset AD. Moreover, insertion of transgenes have been found to disrupt endogenous mouse genes that may sigificantly contribute to the neurodegenerative phenotype observed in these mice. 

Since my PhD, there have been efforts to generate mouse models that more closely resemble LOAD with incorporation of AD-associated variants, such as $\epsilon$ variant of \textit{APOE} and R47H \textit{Trem2} variant \cite{apoe4trem2_mousemodel,Lewandowski2020}. 

%rtg4510: https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0106050

\subsection{Transcriptomic alterations in AD mouse models}

\begin{changemargin}{1.5cm}
	\begin{landscape}
		\small %smaller font
		\setlength\tabcolsep{2pt} %reduced margin size in table
		\renewcommand{\arraystretch}{1.2}
		\begin{longtable}[c]{p{3cm}p{4cm}p{3cm}p{16cm}}
			\captionsetup{width=1.6\textwidth}
			\caption{Transcriptomic and proteomic studies in AD human post-mortem brain tissues and mouse models in revealing missplicing as a widespead hallmark of AD. *RNA-Seq data from ROSMAP dataset\cite{DeJager2018}. AS - Alternative Splicing, RBP - RNA binding protein, TSS - Transcription Start Site}\\
			\toprule
			\multicolumn{1}{c}{References} &
			\multicolumn{1}{c}{Samples and Tissue} &
			\multicolumn{1}{c}{Method} &
			\multicolumn{1}{c}{Implications} \\* \midrule
			\endfirsthead
			%
			\endhead
			%
			\bottomrule
			\endfoot
			%
			\endlastfoot
			%		
	
			\centering Maziuk2018 et al. (2018)\cite{Maziuk2018} &
			\centering rTg4510 mouse model (8 TG, 8 WT) (2 - 8 months) \newline Frontal cortex&
			\centering Immunohistochemistry &
			Majority of (65\%) of RNA binding proteins showed decreased tau association, with the exception of EWSR1, TAF15 and hnRNPA0 that form soluble aggregates with progressive tau pathology. RBPs colocalised with phosorylated-tau but not mature NFT in rTg4510 \\
			
			\centering Apicco et al. (2019) \cite{Apicco2019} &
			\centering PS19 Mouse Model (3 TG, 3 WT) \newline Cortex &
			\centering RNA-Seq &
			Reduced expression of transcripts encoding RNA binding protein in PS19 tau mouse model, with genes involved in synaptic function (Snap25,Camk2bGria2) subjected to disrupted alternative splicing. Modulating RBP aggregation by reducing TIA1 (RNA-binding protein) partially corrected splicing dysregulation associated with tauopathy \\* \bottomrule
		\end{longtable}
	\end{landscape}
\end{changemargin}

\newpage
\section{Aims and Objectives}
\begin{enumerate}
	\item Whole transcriptome analysis of AD post-mortem brain tissues as reference dataset, shed light on differential isoform expression
	\item Particular interest on 19 loci identified from meta-analysis of GWAS studies on AD (Lambert et al. 2013) Targeted transcriptome analysis  
	\item Classification of AS events, which most commonly observed/dominant? Isoforms derived from transcriptional regulation (alternative promoters) vs post-transcriptional regulation? 
	\item Impact of AS events on protein domains. Non-sense mediated decay? 
	\item Integration with other (epi)genetic analysis on same samples, i.e. DNA methylation, lysine acetylation, gene expression 
	\item Protein analysis? Integration with any publicly available mass-spec datasets 
\end{enumerate}
%By identifying and characterising such alternative splicing events, there is hope of deepening current understanding of the biological role of transform isoform diversity and in the case of AD, identify novel pathways involved in pathogenesis of disease as potential target treatments. 

\begin{figure}[htb]
	\begin{center}
		\includegraphics[page=3,trim={2cm 35cm 0 2cm},clip,scale = 0.55]{Introduction_Figures.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Mouse samples for Whole and Targeted Transcriptome Iso-Seq]%
	{\textbf{Mouse samples for Whole and Targeted Transcriptome Iso-Seq}.}
	\label{fig:samples}
\end{figure} 

Gene expression and mRNA isoforms vary widely across tissues (\cite{Wang2008}), thus sequencing the disease-relevant tissue (in this case entorhinal cortex) is important for understanding the pathology of AD. However, it is consequently important to note that other tissues may have to be considered to fully grasp the whole picture of AD development. 

While human post-mortem brain tissues remain to be the gold standard for transcriptomic studies, important to highlight that post-mortem interval and storage conditions of brain material highly influence transcriptome stability, particularly affecting alternative splicing. Furthermore changes in gene/transcript expression can be due to differences in cellular composition (i.e. neuronal loss/reactive gliosis) rather than indicative of disease-associated transcriptional regulation. 

\section{Future Directions}
At the time of writing, there have been other major advances in the field have unfortunately not be explored. This include, single cell transcriptomics and Direct RNA-Sequencing. As most studies are performed on bulk-tissues, it is unclear whether this is a consequence of multiple isoforms in one single cell or from multiple isoforms from multiple single cells. Analysis of mRNA expression at the resolution of individual, "single", cells, allowing representation of cell-to-cell variation rather than taking the stochastic average from bulk measurements, and thereby resolving heterogeneity. This is currently achieved by the capture and analysis of single cells using a microfluidic or droplet-based technology. Importance of single cell approaches highlighted in \cite{Karlsson2017} with few isoforms shared between cells (7\% of all detected isoforms shared between all cell-types, though this increased to 60\% for exon-cassette isoforms). 

To date, Direct RNA sequencing of the native RNA molecules rather than the cDNA is only possible on the ONT. This approach offers several benefits over standard cDNA sequencing, in i) eliminating the risk of generating library artefacts from reverse transcription and PCR, ii) removing length bias toward shorter abundant fragments that could skew the population of transcripts, and finally iii) elucidation of RNA epigenetic modifications. 

%https://www.nature.com/collections/sxnwgntqsk?utm_source=sn&utm_medium=referral&utm_content=RMarketing&utm_campaign=SRBM_USG_YM01_GL_LSGR_Gene_LP
%https://www.biorxiv.org/content/10.1101/2020.12.07.414060v1
%https://www.nature.com/articles/s41467-020-18976-7
%https://www.nature.com/articles/s41593-019-0539-4
Single-cell studies have highlighted the difference in transcriptome diversity at a single cell level, with small overlap of isoforms between cells (\cite{Karlsson2017}). Previous methods on quantifying transcripts at a single cell level have relied on RNA-fluorescence in-situ Hybridisation (RNA-FISH), which is limited in terms of throughput and characterisation of complex splicing events (\cite{Byrne2017})

While the methods I have adopted for long-read sequencing in this thesis allows interrogation of full-length transcripts, this is reliant on the generation and amplification of cDNA from mRNA, which can produce artefacts (template switching), introduce bias (distortion of relative cDNA abundance) and lose RNA modifications. In 2018, ONT showed that it was able to sequence RNA directly using the minION by adding poly(T) adapters directly to the mRNA, with a translocase that was able to bind and process RNA efficiently	 \cite{Garalde2018}, achieving coverage and accuracy comparable to that with ONT-cDNA method. 

% https://www.nature.com/articles/s41598-021-85801-6
% https://www.nature.com/articles/s41588-020-00773-z importance of linking genomics with proteomics
% https://www.nature.com/articles/s41591-020-0815-6 - more proteomics
% https://www.nature.com/articles/s41593-020-00764-7

%Epigenetics: https://www.nature.com/articles/s41588-020-0696-0