\documentclass[../Main/Knit.tex]{subfiles}
\label{targetedmousetranscriptome}

%https://www.nature.com/articles/s41467-020-17009-7#Sec14

\section{Introduction}
One current limitation of whole transcriptome sequencing is the low coverage/sequencing depth achieved per gene due to the distribution of reads across the whole transcriptome. Consequently, while whole transcriptome sequencing allows identification of novel genes (genes not previously annotated to the genome) and novel isoforms, it may not detect isoforms particularly those of low expression resulting in many false negatives. This can be circumvented by the use of target capture, which enriches a selective panel of genes that are then only sequenced. Multiple samples can further be pooled and sequenced together by barcoding samples at cDNA synthesis, which simplifies laboratory workflow and minimises associated sequencing costs.

%[Other methods of Targeted Sequencing i.e. CRISPR]

%Advantages of targeted transcriptomics vs whole transcriptomics; resources in Targeted Transcriptomics paper; Literature Review 
%List of relevance of AD genes


\section{Methods}

\subsection{Samples}
Extracted RNA from mouse entorhinal cortex of wildtype and transgenic rTg4510 mice was sequenced on the PacBio's Sequel (n = 24, Table \ref{tab:mouse_samples_sequenced}), a subset of which were also sequenced on the Oxford Nanopore's MinION (n = 18, Table \ref{tab:mouse_samples_sequenced}). Three biological replicates were selected at each age (2, 4, 6 and 8 months) across wildtype and transgenic mice, multiplexed with barcodes (listed in \cref{tab:barcode_primers}) and sequenced as three batches.

\subsection{Library prepartion and sequencing}
Following the Iso-Seq lab protocol (as described in \cref{chap:isoseq_labpipeline}), 200ng RNA from each sample was primed for first strand cDNA synthesis (\cref{section:ch2_cDNA_synthesis_explanation}) with specific oligo-dT barcodes and amplified using PCR with 14 cycles (\cref{fig:isoseq_targeted_pccresults}, \cref{section:ch2_PCR_explanation}). Following purification with 0.4X and 1X AMPure PB beads, the two fractions were then recombined at equimolar quantities, and samples were subsequently combined at equimolar quantities according to each batch. Enrichment for target genes with IDT hybridisation capture was then performed for each batch (described in \cref{section:ch2_targetcapture_explanation}) using custom-designed probes (summarised in \cref{tab:mouse_probes}). Following successful target capture and library preparation (depicted in \cref{fig:isoseq_targeted_libresults}), sequencing was performed for each batch on the PacBio Sequel using a 1M SMRT cell. RNA from the same samples (n = 24) was also prepared with TruSeq Stranded mRNA Sample Prep Kit (Illumina) and subjected to 125bp paired-end sequencing using a HiSeq2500 (Illumina), and used as junction support of the long reads. 

\subsection{SMRT sequencing QC and data processing}
Processing of raw reads were performed using the Iso-Seq bioinformatics pipeline (outlined in \cref{section:isoseq_bioinformatics}), and is similar to the whole transcriptomics data processing with the exception of demultiplexing samples at \textit{Lima} with barcodes. Briefly, CCS reads were generated for each batch and demultiplexed for each sample. Full-length reads from each sample were then merged and collapsed to unique isoforms with \textit{Cupcake}, which were mapped to mouse reference genome (mm10) using \textit{Minimap2} and annotated with \textit{SQANTI3}. Partial isoforms as a consequence of 5'degradation were filtered out using \textit{TAMA}'s script (tama\_remove\_fragment\_models.py) with default parameters. Full-length Iso-Seq read counts from each individual sample were extracted from \textit{Cupcake's} read\_stat.txt file.

\begin{landscape}
\begin{table}[]
		\resizebox{1.5\textwidth}{!}{%
	\begin{tabular}{@{}cccccccccc@{}}
		\toprule
		\multicolumn{6}{c}{\multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}Sample   \\ demographics\end{tabular}}} &
		\multicolumn{4}{c}{Sequencing Platform} \\ \cmidrule(l){7-10} 
		\multicolumn{6}{c}{}                      & \multicolumn{2}{c}{PacBio Iso-Seq} & \multicolumn{2}{c}{Oxford Nanopore} \\ \midrule
		Sample &
		Phenotype &
		Age (Months) &
		RIN &
		\begin{tabular}[c]{@{}c@{}}Concentration\\ (ng/ul)\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Batch \\ (Barcodes)\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Whole \\ Transcriptome\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Targeted\\  Transcriptome\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Whole \\ Transcriptome\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Targeted \\ Transcriptome\end{tabular} \\ \midrule
		K19 & WT & 4 & 8.8 & 236  & 1 (PB\_BC\_1) &                 & X               &                  &                  \\
		K23 & WT & 8 & 9.1 & 143  & 1 (PB\_BC\_2) & X               & X               &                  &                  \\
		K21 & WT & 6 & 9   & 138  & 1 (PB\_BC\_3) &                 & X               &                  &                  \\
		K18 & TG & 2 & 8.8 & 136  & 1 (PB\_BC\_4) & X               & X               & X                &                  \\
		K20 & TG & 4 & 9.1 & 80.4 & 1 (PB\_BC\_5) &                 & X               &                  &                  \\
		K17 & WT & 2 & 9.2 & 77.1 & 1 (PB\_BC\_6) & X               & X               &                  &                  \\
		S19 & WT & 4 & 9.1 & 84.9 & 2 (PB\_BC\_1) &                 & X               &                  & X                \\
		K24 & TG & 8 & 9.2 & 65.4 & 2 (PB\_BC\_2) & X               & X               &                  & X                \\
		L22 & TG & 8 & 8.7 & 68.6 & 2 (PB\_BC\_3) & X               & X               &                  & X                \\
		M21 & WT & 2 & 9.2 & 72.3 & 2 (PB\_BC\_4) & X               & X               & X                & X                \\
		O18 & TG & 2 & 8.9 & 115  & 2 (PB\_BC\_5) & X               & X               &                  & X                \\
		O23 & WT & 8 & 9   & 91.8 & 2 (PB\_BC\_6) & X               & X               &                  & X                \\
		O22 & TG & 6 & 9.1 & 83.5 & 2 (PB\_BC\_7) &                 & X               &                  & X                \\
		P19 & WT & 6 & 8.9 & 92.2 & 2 (PB\_BC\_8) &                 & X               &                  & X                \\
		T20 & TG & 6 & 9   & 68.7 & 2 (PB\_BC\_9) &                 & X               &                  & X                \\
		Q20 & TG & 8 & 8.6 & 99.7 & 3 (PB\_BC\_1) & X               & X               &                  & X                \\
		Q21 & WT & 2 & 9.2 & 83.3 & 3 (PB\_BC\_2) & X               & X               &                  & X                \\
		S18 & TG & 2 & 8.9 & 115  & 3 (PB\_BC\_3) & X               & X               &                  & X                \\
		S23 & WT & 8 & 9.1 & 95.5 & 3 (PB\_BC\_4) & X               & X               &                  & X                \\
		Q18 & TG & 6 & 8.8 & 87.2 & 3 (PB\_BC\_5) &                 & X               &                  & X                \\
		Q17 & WT & 6 & 8.7 & 85.8 & 3 (PB\_BC\_6) &                 & X               &                  & X                \\
		L18 & TG & 4 & 8.8 & 145  & 3 (PB\_BC\_7) &                 & X               &                  & X                \\
		Q23 & WT & 4 & 9   & 70.8 & 3 (PB\_BC\_8) &                 & X               &                  & X                \\
		T18 & TG & 4 & 9   & 85   & 3 (PB\_BC\_9) &                 & X               &                  & X                \\ \bottomrule
	\end{tabular}%
}
\captionsetup{width=1.5\textwidth}
\caption[Mouse rTg4510 samples sequenced using whole and targeted transcriptome approach with PacBio Iso-Seq and ONT nanopore sequencing]%
{Mouse rTg4510 samples sequenced using whole and targeted transcriptome approach with PacBio Iso-Seq and ONT nanopore sequencing}
\label{tab:mouse_samples_sequenced}
\end{table}
\end{landscape}


\begin{table}[ht]
	\begin{tabular}{@{}cccccc@{}}
		\toprule
		Target &
		\begin{tabular}[c]{@{}c@{}}Number \\ of \\ Probes\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Genome \\ Co-ordinates\end{tabular} &
		Strand &
		\begin{tabular}[c]{@{}c@{}}Full\\  Region\\  (bp)\end{tabular} &
		\begin{tabular}[c]{@{}c@{}}Exons inc UTR \\ (bp)\end{tabular} \\ \midrule
		ABCA1  & 56         & chr  4 : 53030670 -   53160014    & - & 129,107 & 10,260 \\
		ABCA7  & 47         & chr  10 : 79997615 -   80015572   & + & 17,958  & 6,594  \\
		ANK1   & 52         & chr  8 : 22974836 -   23150497    & + & 175,662 & 9,018  \\
		APOE   & 5          & chr  7 : 19696125 -   19699285    & - & 2,923   & 1,251  \\
		APP    & 20         & chr  16 : 84954317 -   85173826   & - & 219,272 & 3,357  \\
		BIN1   & 20         & chr  18 : 32377217 -   32435740   & + & 58,524  & 2,455  \\
		CD33   & 9          & chr  7 : 43528610 -   43533290    & - & 5,716   & 2,571  \\
		CLU    & 9          & chr  14 : 65968483 -   65981545   & + & 13,063  & 1,808  \\
		FUS    & 16         & chr  7 : 127967479 -   127982032  & + & 14,554  & 1,845  \\
		FYN    & 18         & chr  10 : 39369799 -   39565381   & + & 195,583 & 3,692  \\
		MAPT   & 23         & chr  11 : 104231436 -   104332096 & + & 100,661 & 5,387  \\
		PICALM & 24         & chr  7 : 90130232 -   90209447    & + & 79,216  & 4,174  \\
		PTK2B  & 32         & chr  14 : 66153138 -   66281171   & - & 127,796 & 4,034  \\
		RHBDF2 & 21         & chr  11 : 116598082 -   116627138 & - & 28,855  & 3,934  \\
		SNCA   & 7          & chr  6 : 60731454 -   60829974    & - & 98,283  & 1,463  \\
		SORL1  & 48         & chr  9 : 41968370 -   42124408    & - & 155,801 & 6,938  \\
		TARDBP & 15         & chr  4 : 148612263 -   148627115  & - & 14,615  & 7,454  \\
		TREM2  & 5          & chr  17 : 48346401 -   48352276   & + & 5,876   & 1,146  \\
		TRPA1  & 28         & chr  1 : 14872529 -   14918981    & - & 46,215  & 4,263  \\
		VGF    & 9          & chr  5 : 137030295 -   137033351  & + & 3,057   & 2,553  \\
		& Total: 464 &                                   &   &         &        \\ \bottomrule
	\end{tabular}
\caption[Mouse probes for enrichment of AD-associated target genes]%
{\textbf{Mouse probes for enrichment of AD-associated target genes}. For target enrichment and subsequent sequencing, probes were designed and curated to 20 AD-associated genes (as detailed in \cref{section:ch2_targetcapture_explanation})}
\label{tab:mouse_probes}
\end{table}


\begin{figure}[htp]
	\centering
	\vspace{20pt}
	\includegraphics[page=1,trim={0 16cm 0 4cm},clip,scale = 0.45]{Figures/LabResults.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Iso-Seq Targeted Transcriptome - cDNA amplification and purification]%
	{\textbf{The first stage between the targeted and whole transcriptome sequencing is the same with samples typically amplified using 14 cycles followed by enrichment of high molecular weight cDNA in Fraction 2: a)} Like whole transcriptome sequencing, samples were amplified using 14 cycles (Figure \ref{fig:isoseq_whole_pccresults}) whereby cycles below generated insufficent cDNA and cycles above showed signs of over-amplification. The samples shown here (K19, K23, K21, K18, K20, K17) were multiplexed and sequenced in Batch 1 (see Table \ref{tab:mouse_samples_sequenced}. Ladder (L) shown is 100bp DNA ladder. \textbf{b)} Similar to whole transcriptome sequencing, ampilfied cDNA was further divided into two fractions (denoted here as F1 and F2) and purified with 1X (F1) and 0.4X (F2) Ampure beads. As shown in the bioanalyzer gel, there was an enrichment of higher-molecular weight cDNA in Fraction 2 compared to Fraction 1 across all the samples (with the exception of Sample K21 with loss of Fraction 2). Green and purple line represent the lower marker at 50bp and the upper marker at 17kb respectively. F1 - Fraction 1 containing cDNA purified with 1X Ampure beads; F2 - Fraction 2 containing cDNA purified with 0.4X Ampure beads.}
	\label{fig:isoseq_targeted_pccresults}
\end{figure}


\begin{figure}[!htp]
	\centering
	\vspace{20pt}
	\includegraphics[page=2,trim={0 14cm 3cm 1cm},clip,scale = 0.45]{Figures/LabResults.pdf}
	\captionsetup{width=0.95\textwidth}
	\caption[Iso-Seq Targeted Transcriptome - Target Capture and library preparation]%
	{\textbf{Successful target capture and library preparation across all batches, as shown by enrichment of transcripts with specific lengths:} \textbf{a)} and \textbf{c)} are bioanalyzer electropherogram traces of Batch 1 (n = 6) and Batch 2 (n = 9) respectively after enrichment of cDNA with selective IDT probes (Section \ref{section:ch2_targetcapture_explanation}). \textbf{b)}, \textbf{d)} and \textbf{f)} are bioanalyzer electropherogram traces of Batch 1, 2 and 3 respectively after library preparation (denoted here as "Final", Section \ref{section:ch2_smrtbelltemplate_explanation}. \textbf{e)} An overlay of Batch 2 after target capture and library preparation. 
	\\
	\\
	As can be seen across all figures, target capture appears to be successful with detected peaks, reflecting enrichment of target transcripts with specific lengths, which differs from the broad peaks that are evident in whole transcriptome sequencing (Figure \ref{fig:isoseq_whole_bioresults}). Library preparation with ligation of SMRT bell templates retained these targeted transcripts with good peak overlay, as seen in figure e). The difference in peak height (i.e. cDNA quantity) between target capture and library preparation is due to a difference in input cDNA concentration when running Bioanalyzer - input cDNA after library preparation was diluted with a 1:5 dilution factor to maximise amount of cDNA available for sequencing, whereas input cDNA after target capture was not diluted.}  
	\label{fig:isoseq_targeted_libresults}
\end{figure}


\subsection{Transcriptome Annotation and Iso-Seq Quantification}
Isoforms were annotated with \textit{SQANTI3} in combination with mouse reference gene annotations (mouse GENCODE, vm22), FANTOM5 CAGE peaks and \textit{STAR}-aligned RNA-Seq junctions, and were classified as either FSM, ISM, NIC, NNC, antisense, fusion, and intergenic (described in \cref{section: sqanti_annotations}). ISM isoforms were assumed as partial products of longer FSM isoforms, and associated read counts were aggregated.   

\newpage
\section{Results}
\subsection{Run performance and sequencing metrics}
Following library preparation and SMRT sequencing, a total of XXGb (s.d = XXGb) were obtained (Table \ref{tab:targeted_mouse_run_output}). Of note, 6 samples were first trialled and multiplexed in Batch 1 to determine the yield output and coverage depth - PacBio recommends starting with 4 - 8 samples for multiplexing. Having noticed that an average yield output (24Gb) with a high off-target sequencing, implicating saturation of target genes with 6 samples, the number was increased to 9 samples in Batch 2 and Batch 3. Despite more samples, the sequencing run for Batch 2 and 3, performed by Exeter's Seqeuncing Service, had a poor loading rate (38.1\% P1 of Batch 3 vs 71\% of Batch 1) and low subsequent yield. The samples were also potentially degraded after having been stored in -20\textdegree C for over 6 months due to Covid-19 lockdown. 

The yield difference between the first and last two batches was evident in the number of CCS reads (total = 996K; Batch 1 = 469K, Batch 2 = 306K, Batch 3 = 2221K Figure \ref{fig:isoseq_targeted_run_output}a) and FLNC reads (total = 930K; Batch 1 = 399K, Batch 2 = 275K, Batch 3 = 256K, Figure \ref{fig:isoseq_targeted_run_output}a) generated, after applying the bioinformatics Iso-Seq pipeline (same as the whole transcriptome approach with the exception of removing barcodes rather than general primers). However, calculation of the on-target rate suggested that while Batch 2 and 3 had lower output yield, the coverage of target genes was significantly greater than Batch 1 due to the increased sample size (mean rate in Batch 1 = 34.5\%; mean rate in Batch 2 = 46.2\%; mean rate in Batch 3: 42.9\%, Figure \ref{fig:isoseq_targeted_rate}). The on-target rate is defined as the proportion of mapped transcripts with sequences overlapping at least one target probe. 
%Off-target 

In addition to batch variability, the number of full-length transcripts obtained per sample varied within each batch (Figure \ref{fig:isoseq_targeted_run_output}b). This variability was not associated with RIN (corr = 0.147, P = 0.492, Spearman's rank) and is unlikely to be due to library preparation, given that samples were pooled in equal molarity during target capture. However, there was no significant difference in the number of full-length transcripts between WT and TG across the batched runs (Wilcoxon rank sum test, W = 73, P = 0.977, Figure \ref{fig:isoseq_targeted_run_output}c). 

\begin{landscape}
	\begin{table}[]
		\resizebox{1.5\textwidth}{!}{%
			\begin{tabular}{@{}cccccccccccccccccccl@{}}
				\toprule
				\multirow{3}{*}{Sample} &
				\multirow{3}{*}{\begin{tabular}[c]{@{}c@{}}Total \\ Bases \\ (GB)\end{tabular}} &
				\multirow{3}{*}{\begin{tabular}[c]{@{}c@{}}Polymerase\\ Reads\end{tabular}} &
				\multicolumn{6}{c}{Read   Length} &
				\multicolumn{3}{c}{Productivity} &
				\multicolumn{4}{c}{Control} &
				\multirow{3}{*}{\begin{tabular}[c]{@{}c@{}}Local \\ Base \\ Rate\end{tabular}} &
				\multicolumn{2}{c}{Template} &
				\multicolumn{1}{c}{\multirow{3}{*}{Notes}} \\ \cmidrule(lr){4-16} \cmidrule(lr){18-19}
				&
				&
				&
				\multicolumn{2}{c}{Polymerase} &
				\multicolumn{2}{c}{Subread} &
				\multicolumn{2}{c}{Insert} &
				\multirow{2}{*}{P0} &
				\multirow{2}{*}{P1} &
				\multirow{2}{*}{P2} &
				\multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}Total   \\ Reads\end{tabular}} &
				\multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}Read \\ Length\\  Mean\end{tabular}} &
				\multicolumn{2}{c}{Concordance} &
				&
				\multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}Adapter   \\ Dimer \\ (0-10bp)\end{tabular}} &
				\multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}Short \\ Insert \\ (11-100bp)\end{tabular}} &
				\multicolumn{1}{c}{} \\ \cmidrule(lr){4-9} \cmidrule(lr){15-16}
				&
				&
				&
				Mean &
				N50 &
				Mean &
				N50 &
				Mean &
				N50 &
				&
				&
				&
				&
				&
				Mean &
				Mode &
				&
				&
				&
				\multicolumn{1}{c}{} \\ \midrule
				Batch 1 &
				24.2 &
				712250 &
				34016 &
				70473 &
				1402 &
				1852 &
				3024 &
				3808 &
				\begin{tabular}[c]{@{}c@{}}4.62\% \\ (46613)\end{tabular} &
				\begin{tabular}[c]{@{}c@{}}71.58\%\\  (722026)\end{tabular} &
				\begin{tabular}[c]{@{}c@{}}24.76\% \\ (249707)\end{tabular} &
				9,690 &
				31,505 &
				0.84 &
				0.87 &
				2.31 &
				0 &
				0 &
				Sequenced in November 2019 \\
				Batch 2 &
				&
				&
				&
				&
				&
				&
				&
				&
				&
				&
				&
				&
				&
				&
				&
				&
				&
				&
				\multirow{2}{*}{\begin{tabular}[c]{@{}l@{}}Sequenced in July 2020\\ Samples were kept at -20 for over 9months. \\ Sequel broke down mid-run.  \\ Sequencing was prepared by Exeter's Sequencing Services\\ \\ \end{tabular}} \\
				Batch 3 &
				19.3 &
				383292 &
				50472 &
				100255 &
				1557 &
				2017 &
				3158 &
				3898 &
				\begin{tabular}[c]{@{}c@{}}18.68\% \\ (189549)\end{tabular} &
				\begin{tabular}[c]{@{}c@{}}38.11\%\\  (386743)\end{tabular} &
				\begin{tabular}[c]{@{}c@{}}43.56\% \\ (442054)\end{tabular} &
				3,440 &
				52,533 &
				0.85 &
				0.87 &
				2.86 &
				0 &
				0 &
				\\ \bottomrule
			\end{tabular}%
		}
		\captionsetup{width=1.5\textwidth}
		\caption[Run Yield Output from Targeted Transcriptome Iso-Seq of Tg4510]%
		{Iso-Seq run yield for each batch of Tg4510 mouse samples sequenced using targeted transcriptome approach}
		\label{tab:targeted_mouse_run_output}
	\end{table}
\end{landscape}

\begin{figure}[!htp]
	\begin{center}
		\includegraphics[page=1,trim={0 1cm 0 0},clip,scale = 0.55]{Figures/TargetedTranscriptome.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Targeted Transcriptome Iso-seq run performance]%
	{\textbf{Despite batch variability in targeted transcriptome sequencing, no difference in the number of full-length transcripts was observed between wildtype and transgenic mice}. \textbf{a)} Samples (n = 24) were multiplexed and sequenced in three runs (Batch 1, 2 and 3) with varied performance, as indicated by the number of polymerase reads through to poly-A FLNC reads. In the bioinformatics pipeline, the samples were demultiplexed and individually processed after generation of CCS reads from each run. \textbf{b)} Sample variability within each batch was observed from the number of poly-A FLNC reads generated. However, \textbf{c)} no statistical difference was observed in the overall number of full-length transcripts detected between wildtype and transgenic. Full-length transcripts were collapsed from poly-A FLNC reads in Iso-Seq Cluster. CCS - Circular Consensus Sequence, FLNC - Full-Length Non-Concatemer, FL - Full-Length, WT - Wildtype, TG - Transgenic}
	\label{fig:isoseq_targeted_run_output}
\end{figure}

\begin{figure}[!htp]
	\begin{center}
		\includegraphics[page=2,trim={0 25cm 0 0},clip,scale = 0.55]{Figures/TargetedTranscriptome.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[On-Target rate in Transcriptome Iso-Seq runs]%
	{\textbf{Coverage of target genes was greater in Batch 2 and 3 than Batch 1 due to more samples multiplexed and sequenced}. Samples (n = 24) were multiplexed and sequenced in three runs (Batch 1 = 6 samples, Batch 2 = 9 samples, Batch 3 = 9 samples). Despite lower run yield output (\ref{tab:targeted_mouse_run_output}), Batch 2 and Batch 3 had a higher on-target rate, which refers to the proportion full-length transcripts associated with target genes. A difference in the on-target rate between wildtype and transgenic samples was observed in Batch 1, which is a likely reflection of the sample variability in sequencing (Figure \ref{fig:isoseq_targeted_run_output}b). WT - Wildtype, TG - Transgenic}
	\label{fig:isoseq_targeted_rate}
\end{figure}


%Following library preparation and nanopore sequencing (Chapter X), a total of 28.54M reads (39.68Gb) were generated across two flow cells and a total of 22.8M (80\%) reads were successfully basecalled using Guppy.

\clearpage
\subsection{Transcriptome annotation}
After filtering for technical artefacts (563 (1.69\%) isoforms were removed due to intrapriming, 314 (0.94\%) isoforms were removed due to RT switching, 1,267 (3.80\%) were removed due to likely partial degradation), a total of 4,780 isoforms were detected across 20 AD-associated target genes across all the samples (n = 24). Of these isoforms, an overwhelming majority were novel (n = 4601, 96.2\%) with no RNA-Seq support (n = 24 samples, total number of uniquely mapped reads = 360 million) at the junction (n = 4,033, 84.4\%). This is likely to be reflection of the low coverage of RNA-Seq reads per sample  (mean number of uniquely mapped reads = 15 million) to comprehensively span these novel junctions, rather than an indication of the invalidity of these isoforms given the stringent processing of the Iso-Seq bioinformatics pipeline. Nevertheless, following the recommendations from \textit{SQANTI2} and to ease comparison with the whole transcriptome approach (Chapter X), we took the more stringent approach to only include novel isoforms with RNA-Seq support. All downstream analyses and statistics reported in this chapter thereon were based on the subset of \textit{SQANTI2}-filtered isoforms (n = 747 isoforms, Figure \ref{fig:isoseq_targeted_finalnumberiso}).

% proportion of reads with human MAPT; validation 
% supported by RNA-Seq, CAGE 

\iffalse
% plot for technical artefacts
\begin{figure}[!htp]
	\begin{center}
		\includegraphics[page=5,trim={0 12cm 0 0},clip,scale = 0.55]{Figures/TargetedTranscriptome.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Classification of novel and known isoforms from Targeted Sequencing in mouse cortex]%
	{\textbf{Majority of the novel isoforms detected of the target genes has at least one novel donor or acceptor splice sites}. Shown is the number of isoforms detected per target gene, further classified into FSM (Full Splice Match), ISM (Incomplete Splice Match), NIC (Novel In Catalogue) and NNC (Novel Not in Catalogue).}
\end{figure}
\fi
   
\begin{figure}[!htp]
	\begin{center}
		\includegraphics[page=3,scale = 0.55]{Figures/TargetedTranscriptome.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Wide isoform diversity in AD-associated genes from Targeted Sequencing in mouse cortex]%
	{\textbf{Wide isoform diversity observed in AD-associated genes with many novel isoforms detected}. Shown is the number of isoforms detected per target gene, classified by novel and known, after sequential processing and filtering in the bioinformatics Iso-Seq pipeline. Novel isoforms refer to isoforms that are not known in current existing annotations.}
	\label{fig:isoseq_targeted_finalnumberiso}
\end{figure}

\begin{figure}[!htp]
	\begin{center}
		\includegraphics[page=4,scale = 0.55]{Figures/TargetedTranscriptome.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Classification of novel and known isoforms from Targeted Sequencing in mouse cortex]%
	{\textbf{Majority of the novel isoforms detected of the target genes has at least one novel donor or acceptor splice sites}. Shown is the number of isoforms detected per target gene, further classified into FSM (Full Splice Match), ISM (Incomplete Splice Match), NIC (Novel In Catalogue) and NNC (Novel Not in Catalogue).}
	\label{fig:isoseq_targeted_finalnumberiso_sub}
\end{figure}

\subsection{Comparison with whole transcriptome}


\begin{figure}[!htp]
	\begin{center}
		\includegraphics[page=6,scale = 0.55]{Figures/TargetedTranscriptome.pdf}
	\end{center}
	\captionsetup{width=0.95\textwidth}
	\caption[Classification of novel and known isoforms from Targeted Sequencing in mouse cortex]%
	{\textbf{Majority of the novel isoforms detected of the target genes has at least one novel donor or acceptor splice sites}. Shown is the number of isoforms detected per target gene, further classified into FSM (Full Splice Match), ISM (Incomplete Splice Match), NIC (Novel In Catalogue) and NNC (Novel Not in Catalogue).}
\end{figure}

% use same samples, compare whether off-target genes from targeted transcriptome are most abundant or whether based on similar sequence homology to target genes; output similar to targeted ppt  
% compare the AD genes
% threshold cut off 

\clearpage
\subsection{Alzheimer's Disease Target Genes}
%Alternative splicing events?

\boldheader{Abca1}
Identified two isoforms, one already known (ENSMUST00000030010.3) and a much lower expressed novel isoform with one novel splice site (NIC). 
RNA-Seq expression identified differential gene and isoform expression with increased expression of  known isoform with progressive tau pathology. A smaller general trend is observed with Iso-Seq data alone but insignificant.

\boldheader{Sorl1}
In addition to identifying known isoform (ENSMUST00000060989.8), identified 8 novel isoforms (3 NIC and 5 NNC) of which one was characterised with intron retention and two were predicted for NMD (PB.8560.84 with a different polyA motif and PB.8560.39). 7 DMPs observed in gene. Known isoform is the dominant isoform

\boldheader{Mapt}
7 known isoforms detected (ENSMUST00000106992.9, ENSMUST00000100347.10, ENSMUST00000138384.7, ENSMUST00000106988.7, ENSMUST00000126820.1, ENSMUST00000132245.7, ENSMUST00000106993.9) with ENSMUST00000106992.9 the dominant isoform. 
56 novel isoforms detected (22 NIC and 34 NNC), 4 with intron retention and 6 predicted for NMD.  
RNA-Seq expression and Iso-Seq expression show very different profile due to RNA-Seq reads also from sequencing transgene human \textit{MAPT} - conversely any full-length transcript containing human \textit{MAPT} transgene would have been filtered out during Iso-Seq bioinformatics pipeline, thus Iso-Seq reads would only capture mouse \textit{MAPT} isoforms. Unable to use RNA-Seq reads due to sequencing of human transgene; No differential mouse \textit{MAPT} gene or transcript expression. 
Antisense transcript to MAPT also detected.
%Of note: 2 ISMs have have semi high expression counts and may be multimapped; apply a threshold to ensure that must have more than x number of exons from the known but caveats?
%humanMAPT decreases but progressive tau pathology?

\boldheader{Bin1}
Identified 4 known isoforms (ENSMUST00000025239.8, ENSMUST00000091967.12, ENSMUST00000234496.1, ENSMUST00000234857.1) and 89 novel isoforms (34 NIC and 55 NNC). 11 of the novel isoforms were characterised with intron retention, of which all but one was predicted for NMD. 9 additional isoforms were predicted for NMD. ENSMUST00000025239.8 (PB.3915.1) was identified as the major dominant isoform using Iso-Seq where as a novel isoform (PB.3915.208,NIC) was identified as the dominant isoform using RNA-Seq reads. Differential transcript expression observed using Iso-Seq reads with increased expression of novel isoform (PB.3915.524, NIC, P = 3.66 x 10\textsuperscript{-5}, R-squared = 0.547), but this was not recapitulated with RNA-Seq reads. 1 DMP associated with gene. 
%Iso-Seq and RNA-Seq expression does not correspond in terms of major isoform, example of misalignment from RNA-Seq reads? Check the difference between PB.3915.208 and PB.3915.1 sequence 
%check whole targeted approach if novel isoform is detected?

\boldheader{Tardbp}
Identified 6 known isoforms (ENSMUST00000084125.9, ENSMUST00000186317.1, ENSMUST00000105702.8, ENSMUST00000165113.7, ENSMUST00000045180.13, ENSMUST00000172073.7) and 18 novel isoforms (15 NIC, 3 NNC), the majority of which were characterised with intron retention (n = 14) and also predicted for NMD (n = 8) and 3 additional for NMD. ENSMUST00000084125.9 was identified as the dominant isoform using both Iso-Seq and RNA-Seq as expression.

\boldheader{App}
Identified 3 known isoforms (ENSMUST00000227723.1 ENSMUST00000227990.1 ENSMUST00000227021.1) and 39 isoforms, the majority of which were detected using RNA-Seq expression. Differential transcript expression observed using Iso-Seq reads with increased expression of known isoform (ENSMUST00000227654.1, P = 1.75 x 10\textsuperscript{-8}, R-squared = 0.657), but this was not recapitulated with RNA-Seq reads. 

\boldheader{Abca7}
Identified 3 known isoforms (ENSMUST00000227723.1 ENSMUST00000227990.1 ENSMUST00000227021.1) and 39 isoforms, the majority of which were detected using RNA-Seq expression. Differential transcript expression observed using Iso-Seq reads with increased expression of known isoform (ENSMUST00000227654.1, P = 1.75 x 10\textsuperscript{-8}, R-squared = 0.657), but this was not recapitulated with RNA-Seq reads. 2 novel isoforms were identified as dominant isoforms (PB.756.3, NIC and PB.756.31, NNC) 
%ISM mapped to multiple transcripts 

\boldheader{Ptk2b}
Identified 5 known isoforms (ENSMUST00000136216.7, ENSMUST00000022622.13, ENSMUST00000089250.8
ENSMUST00000178730.7, ENSMUST00000111121.1) and 82 novel isoforms of which 6 were characterised with intron retention. Differential transcript expression observed using Iso-Seq reads with increased expression of known isoform (ENSMUST00000089250.8, P = 2.317 x 10\textsuperscript{-8}, R-squared = 0.634), but this was not recapitulated with RNA-Seq reads. 2 dominant isoforms, but were identified differently by Iso-Seq and RNA-Seq reads.
%redraw isoform expression

\boldheader{Ank1}
Identified 6 known isoforms (ENSMUST00000118733.7, ENSMUST00000173248.7, ENSMUST00000110688.8, ENSMUST00000141784.8, ENSMUST00000121075.7, ENSMUST00000033947.14), and 3 novel isoforms (NIC). Of note 4 of the known isoforms are ISM with 3'fragment (suggesting not RNA degradation) but equivalent FSM not available. Alignment of RNA-Seq reads to both whole and targeted transcriptome identified ENSMUST00000121802.8 to be the dominant isoform, however, this was not detected in targeted transcriptome. 
% Example of truncated isoforms in targeted transcriptome
% check whole transcriptome data, check mouse transcriptome data

\boldheader{Fyn}
Identified 3 known isoforms (ENSMUST00000136659.1, ENSMUST00000099967.9, ENSMUST00000063091.12) and 26 novel isoforms. General trend of downregulated gene expression associated with progressive tau pathology with both Iso-Seq and RNA-Seq reads as expression input, but not significant. Known isoform (ENSMUST00000099967.9, PB.605.38) and novel isoform (PB.605.118) was predominaly expressed.  

\boldheader{Clu}
Identified 1 known isoform (ENSMUST00000022616.13) and 62 novel isoforms. Differential gene and isoform expression was detected using RNA-Seq reads as expression, with upgregulated expression associated with progressive tau pathology. However, this increase expression was attributed to a novel isoform (G4103.43,PB.2634.299) rather than the known longer isoform
% kallisto misalignment?

\boldheader{Cd33}
Identified 1 known isoform (ENSMUST00000205503.1) and 11 novel isoforms, the majority of which were characterised with intron retention (n = 7). Differential gene expression and isoform expression was observed using both Iso-Seq and RNA-Seq reads, driven by upregulated expression of the known isoform (PB.7476.2), which was also detected in the whole transcriptome. 

\boldheader{Fus}
Identified 4 known isoforms (ENSMUST00000205261.1, ENSMUST00000128851.7, ENSMUST00000174196.7, ENSMUST00000106251.9) and 21 novel isoforms. The known isoform was identified as the major isoform using Iso-Seq reads, whereas the novel isoform was identified with RNA-Seq reads (G12097.12,PB.7829.97).
% kallisto misalignment? known isoform is detected in the whole transcriptome

\boldheader{Picalm}
Identified 8 known isoforms, and 27 novel isoforms. All isoforms, including the novel isoforms, are similarly expressed, although the known isoform (ENSMUST00000049537.8, PB.7635.31) is still the highest expressed using both Iso-Seq and RNA-Seq reads as expression.

\boldheader{Snca}
Differential gene expression identified using both Iso-Seq and RNA-Seq reads as expresion with TG showing reduced gene expression across progressive tau pathology. However, very different pattern of isoform constitution between Iso-Seq and RNA-Seq reads as expression; known isoform is the dominant major isoform with Iso-Seq reads, however, not obvious from RNA-Seq reads. 
% kallisto misalignment?

\boldheader{Apoe}
Identified 6 known isoforms, and 49 novel isoforms. Differential gene expression identified using both Iso-Seq and RNA-Seq reads as expresion with TG showing increased gene expression across progressive tau pathology. Iso-Seq attributed this increase to the known isoform (PB.7333.28, ENSMUST00000173739.7), whereas RNA-Seq attributed this increase to the novel isoform (G11287.50)

\boldheader{Trpa1}
Identified 1 known isoform and 1 novel isoform, very lowly expressed across all the samples and even with RNA-Seq reads as alignment. 

\boldheader{Rhbdf2}
Identified 1 known isoform and 2 novel isoforms. Slight increase in gene expression driven by known isoform (ENSMUST00000103029.9), however not significant. 

\boldheader{Trem2}
Identified 3 known isoforms and 12 novel isoforms. Observed significant differential gene expression and tranascript expression - increased gene expression associated with progresssive tau pathology, which is attributed to the known isoform (ENSMUST00000024791.14).
% Of note the TAMA merged refers to a whole transcriptome transcript that is slightly different to the one detected in the targeted transcriptome

\boldheader{Vgf}
Identified 2 known isoforms and 3 novel isoforms, Observed differential gene expression and transcript expression -slight decreased gene expression in TG mice. Very low detected using Iso-Seq reads with only known isoform (ENSMUST00000041543.8) and novel isoform passing the threshold for detection, conversely RNA-Seq reads identified two novel isoforms as the major isoform. 
% kallisto alignment
