\documentclass[../Main/Knit.tex]{subfiles}

\section{Oxford Nanopore: cDNA Sequencing}

\subsection{Introduction}
In 2014, Oxford Nanopore Technology (ONT) introduced another long-read sequencing technology akin to PacBio's SMRT, in the ability to also generate long reads that are able to resolve the exon structure of mRNA transcripts. However, rather than mimicking the natural DNA synthesis as is the focus in all major sequencing applications including the PacBio's SMRT, ONT's nanopore-based sequencing adopts an entirely different approach; the DNA sequence is inferred from fluctuations in a current applied across a membrane as it passes through a protein pore.

Inhibited mostly by the ability to deliver very high-molecular weight DNA to the pore, nanopore sequencing is able to generate much longer reads than SMRT sequencing (from 500bp to currently 2.3Mb (Payne et al. 2019)).  

Basecalling accuracy of reads have dramatically increased, with raw base-called error rate reduced from <1\% from SMRT sequencing and <5\% for nanopore sequencing. 

In comparison to SMRT sequencing, accuracy of nanopore reads is independent of DNA length, but reliant on achieving optimal translocation speed of DNA through pore, which often decreases in the late stages of sequencing, thereby negatively impacting quality. Contrary to SMRT sequencing, each DNA fragment is read only once: 
\begin{itemize}
	\item 1D: each strand of dsDNA fragment is read independently, and single pass accuracy is final accuracy of fragment 
	\item 1D\textsuperscript{2}: sequence complementary strand of dsDNA fragment immediately after, allowing determination of more accurate consensus strand, achieving a median consensus accuracy of 98\%. 
\end{itemize} 

Low complexity stretches, including homopolymers, are difficult to resolve with current pores (R9) and basecallers, as translocation of homopolymers do not change the sequence of nucleotides within pore, thereby resulting in a constant signal. Significant major advances in technology have been made over the past 3 years, with 4 pore version released in 2019 (R9.4, R9.4.1, R9.5.1 and R10.0) \cite{Amarasinghe2020}. 

\subsubsection{Mechanism}
In contrast to Pacbio's XXX by XXX Sequel, ONT's nanopore sequencing can be performed in a handheld MINion device (10 × 3 × 2 cm, 90 g), housing a flow cell at its centre where the DNA sample is loaded. (Also on mobile device: \cite{Samarakoon2020}) Each flow cell contains a sensor array, consisting of 512 channels, each with 4 cells that can in turn house one nanopore. However, while the current MinION contains a total of 2048 nanopores, only one of the four wells in each channel can be active at any time, as controlled by Application Specific Integrated Circuit (ASIC), allowing up to 512 independent DNA molecules to be sequenced simultaneously.

*1D vs 2D vs 1D2* 
*6-mer* 
*flow cell and the different nanopore?*

\subsubsection{ONT Kits}
 

