\documentclass[../Main/Knit.tex]{subfiles}
\newpage

\section{Oxford Nanopore: cDNA Sequencing}
%https://www.nature.com/articles/s41587-020-0731-9

\subsection{Introduction}
In 2014, Oxford Nanopore Technology (ONT) introduced another long-read sequencing technology akin to PacBio's SMRT with the ability to also generate long reads capable of resolving the exon structure of mRNA transcripts. However, rather than mimicking the natural DNA synthesis and measuring the incorporation events on the template strand (as is the focus in all major sequencing applications including the PacBio's SMRT) ONT's nanopore-based sequencing adopted an entirely novel approach - the DNA is directly inferred real-time from fluctuations in an electric current applied across a membrane as it passes through a protein pore. 

\subsubsection{Mechanism}
In contrast to Pacbio's XXX by XXX Sequel, ONT's nanopore sequencing can be performed in a handheld MinION device (10 × 3 × 2 cm, 90 g), housing a flow cell at its centre where the DNA sample is loaded. (Also on mobile device: \cite{Samarakoon2020}) Each flow cell contains a sensor array, consisting of 512 channels, each with 4 cells that can in turn house one nanopore (currently CsgG pore from \textit{E.coli} \cite{Goyal2014}), which is inserted in an electrical resistant membrane surrounded by electrolytes. As an electric field is applied across the membrane, the negatively-charged DNA is driven across the nanpopore and subsequently interrupts the current. The pattern to the current disruption is unique and sensitive to the different nucleotides, which allows the sequence to be determined. However, while the current MinION contains a total of 2048 nanopores, only one of the four wells in each channel can be active at any time, as controlled by Application Specific Integrated Circuit (ASIC), allowing up to 512 independent DNA molecules to be sequenced simultaneously.

*6-mer* 
*flow cell and the different nanopore?*
% https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1462-9
% https://community.nanoporetech.com/posts/flo-minxxx-x-rx-x-r
%The difference is in the nanopores present in each flow cell. The nanopore is essentially a biological protein; it is found naturally occurring in the environment. We perform a lot of engineering internally to optimise these nanopores for utilisation in our system, enabling greater throughput and accuracy. The R9.4.1 nanopore is the current broadly-used nanopore. As DNA moves through the nanopore, there's a pinch-point - a narrowing of the hole, which allows us to measure the current, and so interpret the translocation of the molecule in real time. In the R10 series, there is a longer barrel and two pinch points, enabling two such measurements. This provides benefits for the sequencing of homopolymers - stretches of the same base multiple times. These homopolymer stretches are difficult, not only for our platform, but for all sequencing plaforms. We have found that introducing this second pinch point provides better fidelity for homopolymer sequences than previously seen with the R9.4 series; we are starting to see very accurate sequencing of homopolymer stretches of about ten bases. R10.3 is the newest nanopore in this series, and provides the high accuracy of the R10 series together with increased throughput and capture.
%https://nanoporetech.com/sites/default/files/s3/Product_brochure_Final_July_2018.pdf
%https://community.nanoporetech.com/posts/flowcell-r9-5-and-1dsq-lib
%https://community.nanoporetech.com/posts/1d-2-cdna-library-prep-low
%https://link.springer.com/content/pdf/10.1007%2F978-1-4939-7834-2.pdf

\subsubsection{Performance and Run Quality Metric}
The ability of nanopore sequencing to directly read the DNA has both positive and negative implications. Inhibited mostly by the ability to deliver very high-molecular weight DNA to the pore, nanopore sequencing is able to generate much longer reads than SMRT sequencing (from 500bp to currently XXX), with theoretically no upper limit \cite{Loman2015}, and with no bias towards length or GC content \cite{Oikonomopoulos2016, Weirather2017}. This offers great potential in transcriptomics profiling and genome assembly. However without the circular feature of PacBio SMRT sequencing, DNA strand cannot be sequenced multiple times and one of the major limitations of nanopore sequencing is its lower read accuracy. Low complexity stretches, including homopolymers, are furthermore difficult to resolve as translocation of homopolymers do not change the sequence of nucleotides within pore, thereby resulting in a constant signal. 

However over recent years, major advances in both the basecalling algorithms, the chemistry and nanopore itself has drastically increased the initial accuracy from 60\% \cite{Jain2015} to 98.3\% (vR.9.4.1 and Bonito). This includes the chemistry (1D\textsuperscript{2}) to sequence both the template and the complementary strand immediately after, thereby attaining a more accurate consensus read that increases the accuracy of template reads (1D) alone by 5\%\cite{Rang2018}. Further to mimic the circular consensus approach, two methodologies (INC-seq \cite{Li2016c} and R2C2 \cite{Volden2018}) involving Rolling Circle Amplification and subsequent nanopore sequencing of circularised templates have been proposed with accuracy approaching 97.5\%. 
